<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Variant Call Format Legacy</title>
    <link href="/vendor/fontawesome/css/all.min.css" rel="stylesheet" />
    <!-- CSS only -->
    <link
      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
      rel="stylesheet"
      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
      crossorigin="anonymous"
    />
    <link
      rel="stylesheet"
      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
      integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
      crossorigin="anonymous"
    />
    <link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <link
      href="https://fonts.googleapis.com/css2?family=EB+Garamond:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&family=Source+Sans+Pro:ital,wght@0,300;0,400;0,600;0,700;1,300;1,400;1,600;1,700&display=swap"
      rel="stylesheet"
    />

    <!-- Get locas CSS files -->
    
    <link rel="stylesheet" type="text/css" href="/home/jacob/Work/Docsite-Revamp/css/styles.css" />
    
    <link rel="stylesheet" type="text/css" href="/home/jacob/Work/Docsite-Revamp/css/docs.css" />
    
    <link rel="stylesheet" type="text/css" href="/home/jacob/Work/Docsite-Revamp/css/encyclopedia.css" />
    
  </head>
  <body>
    <!---->
    <nav class="navbar navbar-expand-md navbar-light bg-light">
      <div class="container-fluid">
        <button
          class="navbar-toggler"
          type="button"
          data-bs-toggle="collapse"
          data-bs-target="#navbarNavDropdown"
          aria-controls="navbarNavDropdown"
          aria-expanded="false"
          aria-label="Toggle navigation"
        >
          <span class="navbar-toggler-icon"></span>
        </button>
        <!-- Collapsed navigation -->
        <div class="my-container">
          <!-- Main title -->
          <a class="navbar-brand float-start" href="/.">
            <img
              class="brand-image"
              src="/home/jacob/Work/Docsite-Revamp/img/nih-header-logo-color.png"
              alt="NIH National Cancer
            Institute GDC Documentation"
            />
            <span class="hidden"
              >NIH National Cancer Institute GDC Documentation</span
            >
          </a>
          <!-- Expander button -->
          <button
            type="button"
            class="navbar-toggle"
            data-toggle="collapse"
            data-target=".navbar-collapse"
          >
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>

          <!-- GDC Apps -->
          <div class="pull-right">
            <div class="gdc-app-list dropbtn">
              <div class="dropbtn">
                <a id="myToggle" class="dropbtn dropdown-toggle">
                  <span class="dropbtn"
                    ><i class="fas fa-th"></i> GDC Apps</span
                  >
                </a>
                <div id="myDropdown" class="dropdown-content container">
                  <div class="row gdc-app-row">
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                  </div>
                  <div class="row gdc-app-row">
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                  </div>
                  <div class="row gdc-app-row">
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                  </div>
                </div>
              </div>
              <div
                data-dropdown-menu=""
                class="dropdown-menu gdc-apps-menu-container"
                role="menu"
              >
                <div class="gdc-apps-menu-grid">
                  <div class="menu-column">
                    <a
                      href="https://portal.gdc.cancer.gov/"
                      title="Data Portal"
                      target="_blank"
                      class="menu-item"
                    >
                      <img
                        width="21px"
                        src="/img/GDC-App-data-portal-blue.svg"
                        class="icon icon-gdc-portal home"
                        alt="GDC Data Portal"
                      />
                      <div>Data Portal</div>
                    </a>
                    <a
                      href="https://gdc.cancer.gov/developers/gdc-application-programming-interface-api"
                      title="GDC API"
                      target="_blank"
                      class="menu-item"
                    >
                      <span class="icon icon-gdc-portal-api">
                        <span class="path1"></span><span class="path2"></span
                        ><span class="path3"></span><span class="path4"></span
                        ><span class="path5"></span><span class="path6"></span
                        ><span class="path7"></span><span class="path8"></span
                        ><span class="path9"></span><span class="path10"></span>
                      </span>
                      <div>API</div>
                    </a>
                    <a
                      href="https://docs.gdc.cancer.gov/"
                      title="GDC Docs"
                      target="_blank"
                      class="active menu-item"
                    >
                      <span class="icon icon-gdc-docs">
                        <span class="path1"></span><span class="path2"></span
                        ><span class="path3"></span><span class="path4"></span
                        ><span class="path5"></span><span class="path6"></span
                        ><span class="path7"></span><span class="path8"></span
                        ><span class="path9"></span><span class="path10"></span
                        ><span class="path11"></span><span class="path12"></span
                        ><span class="path13"></span
                        ><span class="path14"></span>
                      </span>
                      <div>Documentation</div>
                    </a>
                    <a
                      href="https://portal.gdc.cancer.gov/legacy-archive"
                      title="GDC Legacy Archive"
                      target="_blank"
                      class="menu-item"
                    >
                      <span class="icon icon-gdc-legacy-archive">
                        <span class="path1"></span><span class="path2"></span
                        ><span class="path3"></span><span class="path4"></span
                        ><span class="path5"></span><span class="path6"></span
                        ><span class="path7"></span><span class="path8"></span
                        ><span class="path9"></span><span class="path10"></span>
                      </span>
                      <div>Legacy Archive</div>
                    </a>
                  </div>
                  <div class="menu-column">
                    <a
                      href="https://gdc.cancer.gov/"
                      title="GDC Website"
                      target="_blank"
                      class="menu-item"
                    >
                      <img
                        width="21px"
                        src="/img/GDC-App-website-blue.svg"
                        class="icon icon-gdc-portal home"
                        alt="GDC Website"
                      />
                      <div>Website</div>
                    </a>
                    <a
                      href="https://gdc.cancer.gov/access-data/gdc-data-transfer-tool"
                      class="menu-item"
                      title="GDC Data Transfer Tool"
                      target="_blank"
                    >
                      <span class="icon icon-gdc-data-transer-tool">
                        <span class="path1"></span><span class="path2"></span
                        ><span class="path3"></span><span class="path4"></span
                        ><span class="path5"></span><span class="path6"></span
                        ><span class="path7"></span><span class="path8"></span>
                      </span>
                      <div>Data Transfer Tool</div>
                    </a>
                    <a
                      href="https://portal.gdc.cancer.gov/submission/"
                      title="GDC Submission Portal"
                      target="_blank"
                      class="menu-item"
                    >
                      <span class="icon icon-gdc-submission-portal">
                        <span class="path1"></span><span class="path2"></span
                        ><span class="path3"></span><span class="path4"></span
                        ><span class="path5"></span><span class="path6"></span
                        ><span class="path7"></span><span class="path8"></span
                        ><span class="path9"></span><span class="path10"></span>
                      </span>
                      <div>Data Submission Portal</div>
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="container-fluid">
            <div class="collapse navbar-collapse" id="navbarNavDropdown">
              <ul class="navbar-nav navbar-center me-auto">
                <li class="nav-item">
                  <a class="nav-link" href="/">Home</a>
                </li>
                <li class="nav-item dropdown">
                  <a
                    aria-current="page"
                    href="/output/dist/api-getting-started/"
                    class="nav-link dropdown-toggle"
                    id="navbarDropdownMenuLink"
                    role="button"
                    data-bs-toggle="dropdown"
                    aria-expanded="false"
                    >API
                  </a>

                  <ul
                    class="dropdown-menu"
                    aria-labelledby="navbarDropdownMenuLink"
                  >
                     
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Getting_Started.html">Getting Started</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Search_and_Retrieval.html">Search and Retrieval</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Downloading_Files.html">Downloading Files</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Data_Analysis.html">Data Analysis</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/BAM_Slicing.html">BAM Slicing</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Submission.html">Submission</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Python_Examples.html">Python Examples</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/GraphQL_Examples.html">GraphQL Examples</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/System_Information.html">System Information</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Additional_Examples.html">Additional Examples</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Appendix_A_Available_Fields.html">Appendix A: Available Fields</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Appendix_B_Key_Terms.html">Appendix B: Key Terms</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Appendix_C_Format_of_Submission_Requests_and_Responses.html">Appendix C: Format of Submission Queries and Responses</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Release_Notes/API_Release_Notes.html">Release Notes</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/PDF/index.html">fa-file-text Download PDF /API/PDF/API_UG.pdf</a>
                    </li>
                     
                  </ul>
                </li>
                <li class="nav-item dropdown">
                  <a
                    class="nav-link dropdown-toggle"
                    id="navbarDropdownMenuLink"
                    role="button"
                    data-bs-toggle="dropdown"
                    aria-expanded="false"
                    href="/output/dist/data-submission-portal-data-submission-process/"
                  >
                    Data Submission
                  </a>

                  <ul
                    class="dropdown-menu"
                    aria-labelledby="navbarDropdownMenuLink"
                  >
                     
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Checklist.html">Before Submitting Data to the GDC Portal</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Data_Submission_Overview.html">Data Submission Overview</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Data_Submission_Process.html">Data Submission Portal</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Data_Submission_Walkthrough.html">Data Upload Walkthrough</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Pre_Release_QC.html">Pre-Release Data Portal</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Best_Practices.html">Submission Best Practices</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Release_Notes/Data_Submission_Portal_Release_Notes.html">Release Notes</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/PDF/index.html">fa-file-text Download PDF /Data_Submission_Portal/PDF/Data_Submission_Portal_UG.pdf</a>
                    </li>
                     
                  </ul>
                </li>
                <li class="nav-item dropdown">
                  <a
                    class="nav-link dropdown-toggle"
                    href="/output/dist/data-transfer-tool-index/"
                    id="navbarDropdownMenuLink"
                    role="button"
                    data-bs-toggle="dropdown"
                    aria-expanded="false"
                  >
                    Data Transfer Tool
                  </a>

                  <ul
                    class="dropdown-menu"
                    aria-labelledby="navbarDropdownMenuLink"
                  >
                     
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Users_Guide/Getting_Started.html">Getting Started</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Users_Guide/Preparing_for_Data_Download_and_Upload.html">Preparing for Data Download and Upload</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Users_Guide/Data_Download_and_Upload.html">Data Transfer Tool Command Line Documentation</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Release_Notes/DTT_Release_Notes.html">Release Notes - Command Line</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Users_Guide/Data_Download_DTT_UI.html">Data Transfer Tool UI Documentation</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Release_Notes/DTT_UI_Release_Notes.html">Release Notes - UI</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Users_Guide/Appendix_B_TroubleShooting.html">Troubleshooting Guide</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/PDF/index.html">fa-file-text Download PDF /Data_Transfer_Tool/PDF/Data_Transfer_Tool_UG.pdf</a>
                    </li>
                     
                  </ul>
                </li>
                <li class="nav-item dropdown">
                  <a
                    href="/output/dist/data-dictionary-index/"
                    class="nav-link dropdown-toggle"
                    id="navbarDropdownMenuLink"
                    role="button"
                    data-bs-toggle="dropdown"
                    aria-expanded="false"
                  >
                    Data Dictionary
                  </a>

                  <ul
                    class="dropdown-menu"
                    aria-labelledby="navbarDropdownMenuLink"
                  >
                     
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Dictionary/index.html">About</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Dictionary/viewer.html">Viewer</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Dictionary/gdcmvs.html">Search</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.html">Release Notes</a>
                    </li>
                     
                  </ul>
                </li>
                <li class="nav-item dropdown">
                  <a
                    class="nav-link dropdown-toggle"
                    id="navbarDropdownMenuLink"
                    role="button"
                    data-bs-toggle="dropdown"
                    aria-expanded="false"
                    href="/output/dist/data-getting-started/"
                    >Data</a
                  >

                  <ul
                    class="dropdown-menu"
                    aria-labelledby="navbarDropdownMenuLink"
                  >
                     
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Introduction.html">Introduction</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Data_Model/GDC_Data_Model.html">GDC Data Model</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Data_Security/Data_Security.html">Data Security</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/File_Formats/MAF_Format.html">File Format: MAF</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/File_Formats/VCF_Format.html">File Format: VCF</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline.html">Bioinformatics Pipeline: DNA-Seq Analysis</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline.html">Bioinformatics Pipeline: mRNA Analysis</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/miRNA_Pipeline.html">Bioinformatics Pipeline: miRNA Analysis</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/CNV_Pipeline.html">Bioinformatics Pipeline: Copy Number Variation Analysis</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/Methylation_Pipeline.html">Bioinformatics Pipeline: Methylation Liftover Pipeline</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/RPPA_intro.html">Bioinformatics Pipeline: Protein Expression</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/Aligned_reads_summary_metrics.html">Aligned Reads Summary Metrics</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Release_Notes/Data_Release_Notes.html">Release Notes</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/PDF/index.html">fa-file-text Download PDF /Data/PDF/Data_UG.pdf</a>
                    </li>
                     
                  </ul>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/index.html">
                    Encyclopedia
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </nav>

    <div class="row">
      <div class="col-md-3">
        <div class="container-fluid sidebar-container">
          <div class="sidebar">
            <ul class="main">
              <li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Affymetrix_SNP_6.0.html">Affymetrix SNP 6.0</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Annotations.html">Annotations</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Annotations_TCGA.html">Annotations TCGA</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Aggregated_Somatic_Mutation.html">Aggregated Somatic Mutation</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Aliquot.html">Aliquot</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Biospecimen_Data.html">Biospecimen Data</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Case.html">Case</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Cancer_Genomics_Hub.html">Cancer Genomics Hub</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Center_for_Cancer_Genomics.html">Center for Cancer Genomics</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Clinical_Data.html">Clinical Data</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Clinical_Supplement.html">Clinical Supplement</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Controlled_Access.html">Controlled Access</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Data_Access_Policy.html">Data Access Policy</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Data_Submitter.html">Data Submitter</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/dbGaP.html">dbGaP</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Entity.html">Entity</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/GDC_API.html">GDC API</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/GDC_Data_Portal.html">GDC Data Portal</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/GDC_Data_Submission_Portal.html">GDC Data Submission Portal</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/GDC_Data_Transfer_Tool.html">GDC Data Transfer Tool</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/GDC_Documentation_Site.html">GDC Documentation Site</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/GDC_Web_Site.html">GDC Web Site</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Genomic_Data_Analysis_Network.html">Genomic Data Analysis Network (GDAN)</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Genomic_Data_Commons.html">Genomic Data Commons</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Harmonized_Data.html">Harmonized Data</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/HTSeq-Counts.html">HTSeq-Counts</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/HTSeq-FPKM-UQ.html">HTSeq-FPKM-UQ</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/HTSeq-FPKM.html">HTSeq-FPKM</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Latest_Data.html">Latest Data</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Manifest_File.html">Manifest File</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/MD5_Checksum.html">MD5 Checksum</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Mutation_Annotation_Format.html">Mutation Annotation Format</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Mutation_Annotation_Format_TCGAv2.html">Mutation Annotation Format Legacy</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Redaction.html">Redaction</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Release_Number.html">Release Number</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/REST_API.html">REST API</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/RNA-Seq.html">RNA-Seq</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Sequence_Trace_File.html">Sequence Trace File</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/SNP_Array-Based_Data.html">SNP Array-Based Data</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/TCGA_Barcode.html">TCGA Barcode</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/TCIA.html">TCIA</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/UUID.html">UUID</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Variant_Call_Format.html">Variant Call Format</a></li><li class="main"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/TCGA_VCF_1.1v2.html">Variant Call Format Legacy</a></li><ul><li><a href='#HEADER'>HEADER</a></li><li><a href='#SAMPLE=<ID=id,SampleName=sampleName,Individual=individual,Description="description",'>SAMPLE=<ID=id,SampleName=sampleName,Individual=individual,Description="description",</a></li><li><a href='#None'>None</a></li><li><a href='#Structural variants'>Structural variants</a></li><li><a href='#Complex rearrangements'>Complex rearrangements</a></li><li><a href='#RNA-Seq variants'>RNA-Seq variants</a></li><li><a href='#Including validation status in VCF file'>Including validation status in VCF file</a></li><li><a href='#None'>None</a></li><li><a href='#SAMPLE=<ID=NORMAL_454,SampleName=TCGA-06-0881-10A-01W,Individual=TCGA-06-0881,Description="Validation'>SAMPLE=<ID=NORMAL_454,SampleName=TCGA-06-0881-10A-01W,Individual=TCGA-06-0881,Description="Validation</a></li><li><a href='#None'>None</a></li><li><a href='#SAMPLE=<ID=TUMOR_454,SampleName=TCGA-06-0881-01A-01W,Individual=TCGA-06-0881,Description="Validation'>SAMPLE=<ID=TUMOR_454,SampleName=TCGA-06-0881-01A-01W,Individual=TCGA-06-0881,Description="Validation</a></li><li><a href='#SAMPLE=<ID=TUMOR_Sanger,SampleName=TCGA-06-0881-01A-01W,Individual=TCGA-06-0881,Description="Validation'>SAMPLE=<ID=TUMOR_Sanger,SampleName=TCGA-06-0881-01A-01W,Individual=TCGA-06-0881,Description="Validation</a></li><li><a href='#tcgaversion HEADER line (e.g., ##tcgaversion=1.1). The current acceptable'>tcgaversion HEADER line (e.g., ##tcgaversion=1.1). The current acceptable</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#key=value'>key=value</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#contig=<ID=20,length=62435964,assembly=B36,md5=f126cdf8a6e0c7f379d618ff66beb2da,species="Homo'>contig=<ID=20,length=62435964,assembly=B36,md5=f126cdf8a6e0c7f379d618ff66beb2da,species="Homo</a></li><li><a href='#assembly=url or filename'>assembly=url or filename</a></li><li><a href='#assembly=ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/sv/breakpoint_assemblies.fasta'>assembly=ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/sv/breakpoint_assemblies.fasta</a></li><li><a href='#assembly=breakpoint_assemblies.fasta'>assembly=breakpoint_assemblies.fasta</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#INDIVIDUAL.'>INDIVIDUAL.</a></li><li><a href='#Handling failed checks'>Handling failed checks</a></li><li><a href='#Test files'>Test files</a></li></ul><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/pages/Variant_Type.html">Variant Type</a></li>
            </ul>
          </div>
        </div>
      </div>
      <div class="col-md-9">
        <div class="container-fluid doc-container"><h1><i class="fas fa-book"></i>   TCGA Variant Call Format (VCF) 1.1 Specification</h1><hr/>
<p><strong>Document Information</strong>
This document is retained here for reference purposes and should not be considered the current standard.  </p>
<p><strong>Specification for TCGA Variant Call Format (VCF)</strong><br/>
Version 1.1</p>
<p>Please note that VCF files are treated as <strong>protected</strong> data and must be
submitted to the DCC only in <strong>Level 2</strong> archives.</p>
<h1><i class="fas fa-book"></i>   About TCGA VCF specification</h1><hr/>
<p>Variant Call Format (VCF) is a format for storing and reporting genomic sequence
variations. VCF files are modular where the annotations and genotype information
for a variant are separated from the call itself. As of May 2011, VCF version
4.1 (described
<a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">here</a>)
is the most recent release. GSCs will generate sequence variation data using
high-throughput sequencing technologies and resulting variations will be
submitted to DCC as VCF files. TCGA has adopted VCF 4.1 with certain
modifications to support supplemental information specific to the project.
Subsequent sections describe the format TCGA VCF files should follow and
validation steps that would have to be implemented at the DCC.</p>
<h1><i class="fas fa-book"></i>   Summary of current version changes</h1><hr/>
<p>Following is a summary of additions/modifications for this version and the corresponding validation rule
number is included in parentheses.</p>
<p><strong>UUID compliance</strong>: All TCGA data is currently in the process of being
converted to be UUID-compliant. Until the conversion is complete and all centers
are prepared to start submitting UUID-compliant data, some of the VCF files may
adhere to UUID-based specification whereas some may still have barcodes.
Non-UUID files will follow the specification described here but for
UUID-compliance, VCF files should satisfy the following criteria.</p>
<ol>
<li>
<p><strong>SampleUUID</strong> and <strong>SampleTCGABarcode</strong> are required tags in each
    ##SAMPLE declaration. Please note that <strong>SampleName</strong> will not be a
    required tag once submitting center has fully converted to UUIDs.</p>
<ol>
<li>Metadata represented by SampleTCGABarcode at the DCC should correspond
    to the UUID assigned to SampleUUID.</li>
</ol>
</li>
<li>
<p><strong>Individual</strong> is not a required tag in ##SAMPLE declaration.</p>
</li>
<li>
<p>If ##<strong>INDIVIDUAL</strong> is declared in the header, all SampleUUIDs in the
    header must correspond to the same participant, and the corresponding TCGA
    barcode for that participant should be assigned to ##INDIVIDUAL.</p>
</li>
<li>
<p>SampleName is a required tag in ##SAMPLE declaration. The value assigned
    to SampleName should be a valid <a href="https://docs.gdc.cancer.gov/Encyclopedia/pages/TCGA_Barcode/">aliquot
    barcode</a> / <a href="https://docs.gdc.cancer.gov/Encyclopedia/pages/UUID/">UUID</a>
    in the database. (#15b, #15h)</p>
</li>
<li>
<p>Header declarations for INFO and FORMAT fields should match the values
    defined in Tables 4 and 5 respectively. (#7a)</p>
</li>
<li>
<p>Following FORMAT fields are required for all variant records in a VCF file:
    (#10c)</p>
<ul>
<li>
<p>Genotype (<strong>GT</strong>)</p>
</li>
<li>
<p>Read depth (<strong>DP</strong>)</p>
</li>
<li>
<p>Reads supporting ALT (<strong>AD</strong> or <strong>DP4</strong>)</p>
</li>
<li>
<p>Average base quality for reads supporting alleles (<strong>BQ</strong>)</p>
</li>
<li>
<p>Somatic status of the variant (<strong>SS</strong>). SS can be 0, 1, 2, 3, 4 or 5
    depending on whether relative to normal the variant is wildtype,
    germline, somatic, LOH, post-transcriptional modification, or unknown
    respectively. (#23)</p>
</li>
</ul>
</li>
<li>
<p>Values for INFO field <strong>VLS</strong> (validation status relative to non-adjacent
    Normal) will be checked for validity. It can be 0, 1, 2, 3, 4, or 5 based on
    whether the mutation is wildtype, germline, somatic, LOH, post
    transcriptional modification, or unknown respectively. (#9c)</p>
</li>
<li>
<p>Validation of tags in PEDIGREE declaration has changed as follows: (#16)</p>
<ul>
<li>
<p>Name_0, Name_1, etc. do not have to be these literal strings but instead
    represent arbitrary strings.</p>
</li>
<li>
<p>The keys and values used in the <name_n=value_n> should be unique
    across assignments in any given PEDIGREE declaration.</name_n=value_n></p>
</li>
<li>
<p>Value assigned in <name_n=value_n> does not have to be defined as a
    SAMPLE in a genotype column or in the header.</name_n=value_n></p>
</li>
</ul>
</li>
</ol>
<h1><i class="fas fa-book"></i>   TCGA-specific customizations</h1><hr/>
<p>The VCF 4.1 specification has been customized to support TCGA-specific variant
information. While majority of the steps pertaining to the basic structure of
the file remain the same, checks for supplemental information fields have been
introduced. For example, TCGA VCF specification allows for additional fields to
represent data associated with complex rearrangements, RNA-Seq variants, and
sample-specific metadata.</p>
<p>All TCGA-specific additions and modifications in <a href="#validation-rules">validation
steps</a> are prefixed with a
<tcga-vcf> tag for convenient comparison with 1000Genomes VCF 4.1. The
following table summarizes TCGA-specific customizations that have been added to
the VCF 4.1 specification. The first column, "Customization type", indicates
whether a new validation step has been introduced or if an existing step has
been modified</tcga-vcf></p>
<p><strong>Table 1: TCGA-specific validation steps</strong></p>
<table class="table">
<thead>
<tr>
<th><strong>Customization type</strong></th>
<th><strong>Description</strong></th>
<th><strong>Validation step # in TCGA-VCF 1.1 spec</strong></th>
<th><strong>Corresponding validation step # in VCF 4.1 spec</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>New</td>
<td>Validate that file contains ##tcgaversion HEADER line. Its presence indicates that the file is TCGA VCF and the value assigned to the field contains format version number</td>
<td>---</td>
<td>---</td>
</tr>
<tr>
<td>New</td>
<td>Additional mandatory header lines (Please refer to <a href="#TCGAVariantCallFormat(VCF)1.1Specificat">Table 2</a>)</td>
<td>#1</td>
<td>#1</td>
</tr>
<tr>
<td>New</td>
<td>Validation of SAMPLE meta-information lines</td>
<td>#15</td>
<td>---</td>
</tr>
<tr>
<td>New</td>
<td>Validation of PEDIGREE meta-information lines</td>
<td>#16</td>
<td>---</td>
</tr>
<tr>
<td>Modification</td>
<td>Acceptable value set for CHROM has been modified</td>
<td>#18a,b</td>
<td>#16a</td>
</tr>
<tr>
<td>Modification</td>
<td>Acceptable value set for ALT has been modified</td>
<td>#19</td>
<td>#17</td>
</tr>
<tr>
<td>New</td>
<td>Validation for INFO sub-field "VT" has been added</td>
<td>#22</td>
<td>---</td>
</tr>
<tr>
<td>New</td>
<td>Validation for FORMAT sub-field "SS" has been added</td>
<td>#23</td>
<td>---</td>
</tr>
<tr>
<td>New</td>
<td>Validation for INFO/FORMAT sub-field "DP" has been added</td>
<td>#24</td>
<td>---</td>
</tr>
<tr>
<td>New</td>
<td>Validation for complex rearrangement records has been added</td>
<td>#25</td>
<td>---</td>
</tr>
<tr>
<td>New</td>
<td>Validation for RNA-Seq annotation fields has been added</td>
<td>#26</td>
<td>---</td>
</tr>
<tr>
<td>New</td>
<td>Mandatory FORMAT fields have been added</td>
<td>#10c</td>
<td>---</td>
</tr>
<tr>
<td>New</td>
<td>Check for consistent definitions for INFO and FORMAT fields</td>
<td>#7a</td>
<td>---</td>
</tr>
</tbody>
</table>
<h1><i class="fas fa-book"></i>   File format</h1><hr/>
<p>The following example (based on <a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">VCF version
4.1)</a>
shows different components of a TCGA VCF file. Any VCF file contains two main
sections. The HEADER section contains meta-information for variant records that
are reported as individual rows in the BODY of the VCF file. Both sections are
described below.</p>
<p><strong>Case-sensitivity</strong>: Please note that all fields and their associated
validation rules are case-sensitive (as given in the specification) unless noted
otherwise.</p>
<p><strong>Figure 1: Components of a sample TCGA VCF file</strong></p>
<table class="table">
<thead>
<tr>
<th><img alt="images" src="images/vcfExample_VCF.png"/></th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
</tr>
</tbody>
</table>
<h2 id="HEADER">HEADER</h2>
<p>The HEADER contains meta-information lines that provide supplemental information
about variants contained in BODY of the file. HEADER lines could be formatted in
the following two ways:</p>
<pre><code>##key=value

Example:

##fileformat=VCFv4.1

##fileDate=20090805
</code></pre>
<p>or</p>
<pre><code>##FIELDTYPE=&lt;key1=value1,key2=value2,...&gt;

Example:

##INFO=&lt;ID=AA,Number=1,Type=String,Description="Ancestral Allele"&gt;
</code></pre>
<p>Meta-information could be applicable either to all variant records in the file
(e.g., date of creation of file) or to individual variants (e.g., flag to
indicate whether a given variant exists in dbSNP).</p>
<h3>Generic meta-information</h3>
<p><strong>Format</strong>: <em>##key=value</em> OR <em>##FIELDTYPE=<key1=value1,key2=value2,...></key1=value1,key2=value2,...></em></p>
<p>The following table lists some of the reserved field names. Files can be
customized to contain additional meta-information fields as long as they are not
in conflict with reserved field names. The first field in Table 2 (fileformat)
is mandatory and lists the VCF version number of the file.</p>
<p><strong>Table 2: Examples of generic meta-information fields</strong></p>
<table class="table">
<thead>
<tr>
<th><strong>Field</strong></th>
<th><strong>Case-Sensitive</strong></th>
<th><strong>Description</strong></th>
<th><strong>Sample values</strong></th>
<th>Required (fields in red are TCGA-specific requirements)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Fileformat</td>
<td>No</td>
<td>Lists the VCF version number the file is based on; must be the first line in the file</td>
<td>##fileformat=VCFv4.1</td>
<td>Yes</td>
</tr>
<tr>
<td>fileDate</td>
<td>No</td>
<td>Date file was created; should be in yyyymmdd format</td>
<td>##fileDate=20090805</td>
<td>Yes</td>
</tr>
<tr>
<td>Tcgaversion</td>
<td>No</td>
<td>Indicates that the file follows TCGA-VCF specification. Format version number is assigned to the field.</td>
<td>##tcgaversion=1.1</td>
<td>Yes</td>
</tr>
<tr>
<td>Reference</td>
<td>No</td>
<td>Reference build used for variant calling and against which variant coordinates are shown</td>
<td>##reference=1000GenomesPilot-NCBI36</td>
<td>Yes</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>OR</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>##reference=&lt;ID=hg18,</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>Source=<file: 1000genomespilot-ncbi36.fasta="" references="" seq=""></file:></td>
<td></td>
</tr>
<tr>
<td>Assembly</td>
<td>No</td>
<td>External assembly file. The field can be assigned a file name if assembly file is included in the archive submitted to the DCC or it can be a URL pointing to the file location.</td>
<td>##assembly=<a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/sv/breakpoint_assemblies.fasta">ftp://ftp-trace.ncbi.nih.gov/</a></td>
<td>Yes</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td><a href="ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/sv/breakpoint_assemblies.fasta">1000genomes/ftp/release/sv/breakpoint_assemblies.fasta</a></td>
<td>(if a contig from an assembly file is being referred to in the VCF file, especially for breakends)</td>
</tr>
<tr>
<td>center</td>
<td>No</td>
<td>Name of the center where VCF file is generated. A comma-separated list can be provided if files from multiple centers are merged.</td>
<td>##center="Broad"</td>
<td>Yes</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>OR</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>##center="Broad,UCSC,BCM"</td>
<td></td>
</tr>
<tr>
<td>phasing</td>
<td>No</td>
<td>Indicates whether genotype calls are partially phased (phasing=partial) or unphased (phasing=none)</td>
<td>##phasing=none</td>
<td>Yes</td>
</tr>
<tr>
<td>geneAnno</td>
<td>No</td>
<td>URL of the gene annotation source e.g., Generic Annotation File (GAF)</td>
<td>##geneAnno=<a href="https://api.gdc.cancer.gov/legacy/data/95c3618c-bd9e-4df4-96e4-ef8d54710e51">https://gdc.cancer.gov/about-data/data-harmonization-and-generation/gdc-reference-files</a></td>
<td>Yes (if annotation tags like GENE, SID and RGN are used)</td>
</tr>
<tr>
<td>vcfProcessLog</td>
<td>No</td>
<td>Lists algorithm, version and settings used to generate variant calls in a VCF file. If multiple VCF files are processed to produce a single merged file, the field records attributes for individual VCF files and the programs used to merge the files along with the associated version, parameters and contact information of the person who produced the merged file.</td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>##vcfProcessLog=&lt;InputVCF=<file1.vcf>,                                                                                                          InputVCFSource=<varcaller1>,</varcaller1></file1.vcf></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td><strong>Note</strong>: If VCF file does not represent a set of merged files, <em>MergeSoftware</em>, <em>MergeParam</em>, <em>MergeVer</em> and <em>MergeContact</em> tags will not be applicable and can be omitted.</td>
<td>InputVCFVer=&lt;1.0&gt;,</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>InputVCFParam=<a1,c2></a1,c2></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td><strong>Note</strong>: If multiple parameters need to be declared in <em>InputVCFParam</em>, key=value pairs can be used to name these parameters. For example:</td>
<td>InputVCFgeneAnno=<anno1.gaf>&gt;</anno1.gaf></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td>InputVCFParam=<key1=value1,key2=value2></key1=value1,key2=value2></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td>If there are multiple files for which parameters have to be declared, following format can be used:</td>
<td>OR</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td>InputVCFParam=<key1_1=value1_1,key1_2=value1_2;key2_1=value2_1,key2_2=value2_2></key1_1=value1_1,key1_2=value1_2;key2_1=value2_1,key2_2=value2_2></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>##vcfProcessLog=&lt;InputVCF=<file1.vcf,file2.vcf,file3.vcf>,</file1.vcf,file2.vcf,file3.vcf></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>InputVCFSource=<varcaller1,varcaller2,varcaller3>,</varcaller1,varcaller2,varcaller3></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>InputVCFVer=&lt;1.0,2.1,2.0&gt;,</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>InputVCFParam=<a1,c2;a1,b1;a1,b1>,</a1,c2;a1,b1;a1,b1></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>InputVCFgeneAnno=<anno1.gaf,anno2.gaf,anno3.gaf>,</anno1.gaf,anno2.gaf,anno3.gaf></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>MergeSoftware=<sw1,sw2>,</sw1,sw2></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>MergeParam=<a1,a2;b1,b2>,</a1,a2;b1,b2></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>MergeVer=&lt;2.1,3.0&gt;,</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td>MergeContact=<a href="mailto:johndoe@xyz.edu">johndoe@xyz.edu</a>&gt;</td>
<td></td>
</tr>
<tr>
<td>INDIVIDUAL</td>
<td>No</td>
<td>Specifies the individual for which data is presented in the file</td>
<td>##INDIVIDUAL=TCGA-24-0980</td>
<td>No</td>
</tr>
</tbody>
</table>
<h3>INFO/FORMAT/FILTER meta-information</h3>
<p><strong>Format</strong>: <em>##FIELDTYPE=<key1=value1,key2=value2,...></key1=value1,key2=value2,...></em></p>
<p>INFO, FORMAT and FILTER (case-sensitive values) are optional fields that have to
be declared in the HEADER if they are being referred to in BODY of the file.
Different <em>keys</em> that can be used to define them are described in Table 3. All
three fields do not use the same set of keys. Please refer to individual field
definitions for further details.</p>
<p><strong>Important</strong>: TCGA VCF 1.1 requires all VCF files to follow consistent header
declarations for standard INFO and FORMAT sub-fields. Please refer to Tables 4
and 5 for details. If a sub-field exists in these tables and is used in a TCGA
VCF file, then all <key=value> pairs in the definition should match entries in
the corresponding table for the file to pass validation.</key=value></p>
<p><strong>Table 3: Description of keys used in INFO/FORMAT/FILTER meta-information
declarations</strong></p>
<table class="table">
<thead>
<tr>
<th><strong>Key</strong></th>
<th><strong>Case-sensitive</strong></th>
<th><strong>Description</strong></th>
<th><strong>Data Type (Possible values)</strong></th>
<th><strong>Additional Notes</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>ID</td>
<td>Yes</td>
<td>name of the field; also used in BODY of the file to assign values for individual variant records</td>
<td>String, no whitespaces, no comma</td>
<td>---</td>
</tr>
<tr>
<td>Number</td>
<td>Yes</td>
<td>specifies the number of values that can be associated with the corresponding field</td>
<td>Set</td>
<td>Any integer &gt;= 0 indicating number of values;</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td><em>(Integer &gt;= 0, "A", "G", ".")</em></td>
<td>"A", if the field has one value per alternate allele;</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td>"G", if the field has one value per genotype;</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td>".", if number of values varies, is unknown, or is unbounded</td>
</tr>
<tr>
<td>Type</td>
<td>Yes</td>
<td>indicates data type of the value associated with the field</td>
<td>Set</td>
<td>"Flag" type indicates that the field does not contain a value entry, and hence the <em>Number</em> should be 0 in this case. FORMAT fields cannot have a "Flag" <em>Type</em> assigned to them.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td><em>(Integer, Float, Flag, Character, String)</em></td>
<td></td>
</tr>
<tr>
<td>Description</td>
<td>Yes</td>
<td>provides a brief description of the field</td>
<td>String, surrounded by double-quotes, cannot itself contain a double-quote, cannot contain trailing whitespace at the end of string before closing quotes</td>
<td>---</td>
</tr>
</tbody>
</table>
<h4>INFO lines</h4>
<p><strong>Format</strong>: <em>##INFO=<key1=value1,key2=value2,...></key1=value1,key2=value2,...></em><br/>
<strong>Required keys</strong>: ID, Type, Number, Description</p>
<p>INFO fields are optional and contain additional annotations for a variant.
Certain INFO fields have already been created and exist as reserved fields in
the current VCF standard. Custom INFO fields can be added based on study
requirements as long as they do not use the reserved field names. If an INFO
field is declared in the header, it needs to be described further using the
following format:</p>
<pre><code>##INFO=&lt;ID=ID,Number=number,Type=type,Description=description&gt;

Example:

##INFO=&lt;ID=AA,Number=1,Type=String,Description="Ancestral Allele"&gt;

##INFO=&lt;ID=AF,Number=A,Type=Float,Description="Allele Frequ

ency"&gt;
</code></pre>
<h4>FORMAT lines</h4>
<p><strong>Format</strong>: <em>##FORMAT=<key1=value1,key2=value2,...></key1=value1,key2=value2,...></em><br/>
<strong>Required keys</strong>: ID, Type, Number, Description</p>
<p>FORMAT declaration lines are used when annotations need to be added for
individual genotypes associated with each sample in the file. FORMAT sub-fields
are declared precisely as the INFO sub-fields with the exception that a FORMAT
sub-field cannot be assigned a "Flag" <em>Type.</em></p>
<pre><code>##FORMAT=&lt;ID=ID,Number=number,Type=type,Description=description&gt;

Example:

##FORMAT=&lt;ID=GT,Number=1,Type=String,Description="Genotype"&gt;

##FORMAT=&lt;ID=GQ,Number=1,Type=Integer,Description="Genotype Quality"&gt;
</code></pre>
<p><strong>Important</strong>: TCGA VCF 1.1 requires the following FORMAT sub-fields to be
defined for all variant records. Therefore, these FORMAT lines are not optional
for TCGA VCF files and should be declared in the header. Please refer to Table 5
for definitions for these sub-fields.</p>
<ul>
<li>
<p>Genotype (<strong>GT</strong>)</p>
</li>
<li>
<p>Read depth (<strong>DP</strong>)</p>
</li>
<li>
<p>Reads supporting ALT (<strong>AD</strong> or <strong>DP4</strong>). Either AD or DP4 is required to be
    defined although DP4 is preferred.</p>
</li>
<li>
<p>Average base quality for reads supporting alleles (<strong>BQ</strong>)</p>
</li>
<li>
<p>Somatic status of the variant (<strong>SS</strong>). SS can be 0, 1, 2, 3, 4, or 5
    depending on whether relative to normal the variant is wildtype, germline,
    somatic, LOH, post-transcriptional modification, or unknown respectively.</p>
</li>
</ul>
<p>These should be considered as required fields so that they are included by
default unless there is an exceptional scenario where the information for a
field cannot be obtained. In such a case, "." can be used to indicate missing
value.</p>
<h4>FILTER lines</h4>
<p><strong>Format</strong>: <em>##FILTER=<key1=value1,key2=value2,...></key1=value1,key2=value2,...></em><br/>
<strong>Required keys</strong>: ID, Description</p>
<p>FILTER fields are defined to list filtering criteria used for generating variant
calls. Custom filters can be applied as long as a definition is provided in the
HEADER. FILTERs that have been applied to the data should be described as
follows. Please note that FILTER declarations do not include <em>Type</em> or <em>Number</em>
keys.</p>
<pre><code>##FILTER=&lt;ID=ID,Description=description&gt;

Example:

##FILTER=&lt;ID=q10,Description="Quality below 10"&gt;

##FILTER=&lt;ID=s50,Description="Less than 50% of samples have data"&gt;
</code></pre>
<h4>Consistent definitions for reserved INFO and FORMAT fields</h4>
<p>To ensure that all TCGA VCF files have consistent definitions for standard
fields and to avoid merging errors due to contradicting definitions, following
header declarations for common fields are proposed. The 'Source' column in the
tables below indicates whether the field is from 1000Genomes VCF or if it is
specific to TCGA-VCF. By adhering to these definitions, we can ensure that a
given field is interpreted the same way across all centers and that same
'Number', 'Type' and 'Description' values are used for these IDs.</p>
<h5>Table 4: INFO sub-field definitions</h5>
<table class="table">
<thead>
<tr>
<th><strong>Sub-field</strong></th>
<th><strong>Source</strong></th>
<th><strong>Formatted declaration</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>AA</td>
<td>VCF</td>
<td>##INFO=<id=aa,number=1,type=string,description="ancestral allele"=""></id=aa,number=1,type=string,description="ancestral></td>
</tr>
<tr>
<td>AC</td>
<td>VCF</td>
<td>##INFO=<id=ac,number=.,type=integer,description="allele allele,="" alt="" as="" count="" each="" for="" genotypes,="" in="" listed"="" order="" same="" the=""></id=ac,number=.,type=integer,description="allele></td>
</tr>
<tr>
<td>AF</td>
<td>VCF</td>
<td>##INFO=<id=af,number=.,type=float,description="allele allele,="" alt="" as="" data,="" each="" for="" frequency="" in="" listed"="" order="" primary="" same="" the=""></id=af,number=.,type=float,description="allele></td>
</tr>
<tr>
<td>AN</td>
<td>VCF</td>
<td>##INFO=<id=an,number=1,type=integer,description="total alleles="" called="" genotypes"="" in="" number="" of=""></id=an,number=1,type=integer,description="total></td>
</tr>
<tr>
<td>BQ</td>
<td>VCF</td>
<td>##INFO=<id=bq,number=1,type=integer,description="rms base="" quality"=""></id=bq,number=1,type=integer,description="rms></td>
</tr>
<tr>
<td>CIGAR</td>
<td>VCF</td>
<td>##INFO=<id=cigar,number=1,type=integer,description="cigar align="" allele="" allele"="" alternate="" an="" describing="" how="" reference="" string="" the="" to=""></id=cigar,number=1,type=integer,description="cigar></td>
</tr>
<tr>
<td>DB</td>
<td>VCF</td>
<td>##INFO=<id=db,number=0,type=flag,description="dbsnp membership"=""></id=db,number=0,type=flag,description="dbsnp></td>
</tr>
<tr>
<td>DP</td>
<td>VCF</td>
<td>##INFO=<id=dp,number=1,type=integer,description="total across="" depth="" samples"=""></id=dp,number=1,type=integer,description="total></td>
</tr>
<tr>
<td>END</td>
<td>VCF</td>
<td>##INFO=<id=end,number=1,type=integer,description="end described="" in="" of="" position="" record"="" the="" this="" variant=""></id=end,number=1,type=integer,description="end></td>
</tr>
<tr>
<td>H2</td>
<td>VCF</td>
<td>##INFO=<id=h2,number=0,type=flag,description="hapmap2 membership"=""></id=h2,number=0,type=flag,description="hapmap2></td>
</tr>
<tr>
<td>H3</td>
<td>VCF</td>
<td>##INFO=<id=h3,number=0,type=flag,description="hapmap3 membership"=""></id=h3,number=0,type=flag,description="hapmap3></td>
</tr>
<tr>
<td>MQ</td>
<td>VCF</td>
<td>##INFO=<id=mq,number=1,type=integer,description="rms mapping="" quality"=""></id=mq,number=1,type=integer,description="rms></td>
</tr>
<tr>
<td>MQ0</td>
<td>VCF</td>
<td>##INFO=<id=mq0,number=1,type=integer,description="total mapping="" quality="" reads"="" zero=""></id=mq0,number=1,type=integer,description="total></td>
</tr>
<tr>
<td>NS</td>
<td>VCF</td>
<td>##INFO=<id=ns,number=1,type=integer,description="number data"="" of="" samples="" with=""></id=ns,number=1,type=integer,description="number></td>
</tr>
<tr>
<td>SB</td>
<td>VCF</td>
<td>##INFO=<id=sb,number=1,type=float,description="strand bias"=""></id=sb,number=1,type=float,description="strand></td>
</tr>
<tr>
<td>SOMATIC</td>
<td>VCF</td>
<td>##INFO=<id=somatic,number=0,type=flag,description="indicates a="" if="" is="" mutation"="" record="" somatic=""></id=somatic,number=0,type=flag,description="indicates></td>
</tr>
<tr>
<td>VALIDATED</td>
<td>VCF</td>
<td>##INFO=<id=validated,number=0,type=flag,description="indicates been="" by="" experiment"="" follow-up="" has="" if="" validated="" variant=""></id=validated,number=0,type=flag,description="indicates></td>
</tr>
<tr>
<td>1000G</td>
<td>VCF</td>
<td>##INFO=<id=1000g,number=0,type=flag,description="indicates 1000genomes"="" in="" membership=""></id=1000g,number=0,type=flag,description="indicates></td>
</tr>
<tr>
<td>IMPRECISE</td>
<td>VCF</td>
<td>##INFO=<id=imprecise,number=0,type=flag,description="imprecise structural="" variation"=""></id=imprecise,number=0,type=flag,description="imprecise></td>
</tr>
<tr>
<td>NOVEL</td>
<td>VCF</td>
<td>##INFO=<id=novel,number=0,type=flag,description="indicates a="" novel="" structural="" variation"=""></id=novel,number=0,type=flag,description="indicates></td>
</tr>
<tr>
<td>SVTYPE</td>
<td>VCF</td>
<td>##INFO=<id=svtype,number=1,type=string,description="type of="" structural="" variant"=""></id=svtype,number=1,type=string,description="type></td>
</tr>
<tr>
<td>SVLEN</td>
<td>VCF</td>
<td>##INFO=<id=svlen,number=.,type=integer,description="difference alleles"="" alt="" and="" between="" in="" length="" ref=""></id=svlen,number=.,type=integer,description="difference></td>
</tr>
<tr>
<td>CIPOS</td>
<td>VCF</td>
<td>##INFO=<id=cipos,number=2,type=integer,description="confidence around="" for="" imprecise="" interval="" pos="" variants"=""></id=cipos,number=2,type=integer,description="confidence></td>
</tr>
<tr>
<td>CIEND</td>
<td>VCF</td>
<td>##INFO=<id=ciend,number=2,type=integer,description="confidence around="" end="" for="" imprecise="" interval="" variants"=""></id=ciend,number=2,type=integer,description="confidence></td>
</tr>
<tr>
<td>HOMLEN</td>
<td>VCF</td>
<td>##INFO=<id=homlen,number=.,type=integer,description="length at="" base="" breakpoints"="" event="" identical="" micro-homology="" of="" pair=""></id=homlen,number=.,type=integer,description="length></td>
</tr>
<tr>
<td>HOMSEQ</td>
<td>VCF</td>
<td>##INFO=<id=homseq,number=.,type=string,description="sequence at="" base="" breakpoints"="" event="" identical="" micro-homology="" of="" pair=""></id=homseq,number=.,type=string,description="sequence></td>
</tr>
<tr>
<td>BKPTID</td>
<td>VCF</td>
<td>##INFO=<id=bkptid,number=.,type=string,description="id allele="" alternate="" assembled="" assembly="" file"="" in="" of="" the=""></id=bkptid,number=.,type=string,description="id></td>
</tr>
<tr>
<td>MEINFO</td>
<td>VCF</td>
<td>##INFO=<id=meinfo,number=4,type=string,description="mobile element="" form="" info="" name,start,end,polarity"="" of="" the=""></id=meinfo,number=4,type=string,description="mobile></td>
</tr>
<tr>
<td>METRANS</td>
<td>VCF</td>
<td>##INFO=<id=metrans,number=4,type=string,description="mobile chr,start,end,polarity"="" element="" form="" info="" of="" the="" transduction=""></id=metrans,number=4,type=string,description="mobile></td>
</tr>
<tr>
<td>DGVID</td>
<td>VCF</td>
<td>##INFO=<id=dgvid,number=1,type=string,description="id database="" element="" genomic="" in="" of="" this="" variation"=""></id=dgvid,number=1,type=string,description="id></td>
</tr>
<tr>
<td>DBVARID</td>
<td>VCF</td>
<td>##INFO=<id=dbvarid,number=1,type=string,description="id dbvar"\]="" element="" in="" of="" this=""></id=dbvarid,number=1,type=string,description="id></td>
</tr>
<tr>
<td>DBRIPID</td>
<td>VCF</td>
<td>##INFO=<id=dbripid,number=1,type=string,description="id dbrip"="" element="" in="" of="" this=""></id=dbripid,number=1,type=string,description="id></td>
</tr>
<tr>
<td>MATEID</td>
<td>VCF</td>
<td>##INFO=<id=mateid,number=.,type=string,description="id breakends"="" mate="" of=""></id=mateid,number=.,type=string,description="id></td>
</tr>
<tr>
<td>PARID</td>
<td>VCF</td>
<td>##INFO=<id=parid,number=1,type=string,description="id breakend"="" of="" partner=""></id=parid,number=1,type=string,description="id></td>
</tr>
<tr>
<td>EVENT</td>
<td>VCF</td>
<td>##INFO=<id=event,number=1,type=string,description="id associated="" breakend"="" event="" of="" to=""></id=event,number=1,type=string,description="id></td>
</tr>
<tr>
<td>CILEN</td>
<td>VCF</td>
<td>##INFO=<id=cilen,number=2,type=integer,description="confidence around="" between="" breakends"="" inserted="" interval="" length="" material="" of="" the=""></id=cilen,number=2,type=integer,description="confidence></td>
</tr>
<tr>
<td>DPADJ</td>
<td>VCF</td>
<td>##INFO=<id=dpadj,number=.,type=integer,description="read adjacency"="" depth="" of=""></id=dpadj,number=.,type=integer,description="read></td>
</tr>
<tr>
<td>CN</td>
<td>VCF</td>
<td>##INFO=<id=cn,number=1,type=integer,description="copy breakend"="" containing="" number="" of="" segment=""></id=cn,number=1,type=integer,description="copy></td>
</tr>
<tr>
<td>CNADJ</td>
<td>VCF</td>
<td>##INFO=<id=cnadj,number=.,type=integer,description="copy adjacency"="" number="" of=""></id=cnadj,number=.,type=integer,description="copy></td>
</tr>
<tr>
<td>CICN</td>
<td>VCF</td>
<td>##INFO=<id=cicn,number=2,type=integer,description="confidence around="" copy="" for="" interval="" number="" segment"="" the=""></id=cicn,number=2,type=integer,description="confidence></td>
</tr>
<tr>
<td>CICNADJ</td>
<td>VCF</td>
<td>##INFO=<id=cicnadj,number=.,type=integer,description="confidence adjacency"="" around="" copy="" for="" interval="" number="" the=""></id=cicnadj,number=.,type=integer,description="confidence></td>
</tr>
<tr>
<td>VLS</td>
<td>TCGA-VCF</td>
<td>##INFO=<id=vls,number=1,type=integer,description="final modification,5='unknown"' non-adjacent="" normal,0="wildtype,1=germline,2=somatic,3=LOH,4=post" relative="" status="" to="" transcriptional="" validation=""></id=vls,number=1,type=integer,description="final></td>
</tr>
<tr>
<td>SID</td>
<td>TCGA-VCF</td>
<td>##INFO=<id=sid,number=.,type=string,description="unique annotation="" from="" gene="" identifier="" or="" source="" unknown"=""></id=sid,number=.,type=string,description="unique></td>
</tr>
<tr>
<td>GENE</td>
<td>TCGA-VCF</td>
<td>##INFO=<id=gene,number=.,type=string,description="hugo gene="" or="" symbol="" unknown"=""></id=gene,number=.,type=string,description="hugo></td>
</tr>
<tr>
<td>RGN</td>
<td>TCGA-VCF</td>
<td>##INFO=<id=rgn,number=.,type=string,description="region a="" gene"="" in="" nucleotide="" occurs="" relation="" to="" variant="" where=""></id=rgn,number=.,type=string,description="region></td>
</tr>
<tr>
<td>RE</td>
<td>TCGA-VCF</td>
<td>##INFO=<id=re,number=0,type=flag,description="position have="" known="" occur"="" rna-edits="" to=""></id=re,number=0,type=flag,description="position></td>
</tr>
<tr>
<td>VT</td>
<td>TCGA-VCF</td>
<td>##INFO=<id=vt,number=1,type=string,description="variant be="" can="" del"="" ins="" or="" snp,="" type,=""></id=vt,number=1,type=string,description="variant></td>
</tr>
</tbody>
</table>
<h5>Table 5: FORMAT sub-field definitions</h5>
<table class="table">
<thead>
<tr>
<th><strong>Sub-field</strong></th>
<th><strong>Source</strong></th>
<th><strong>Formatted declaration</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>GT</td>
<td>VCF</td>
<td>##FORMAT=<id=gt,number=1,type=string,description="genotype"></id=gt,number=1,type=string,description="genotype"></td>
</tr>
<tr>
<td>DP</td>
<td>VCF</td>
<td>##FORMAT=<id=dp,number=1,type=integer,description="read at="" depth="" in="" position="" sample"="" the="" this=""></id=dp,number=1,type=integer,description="read></td>
</tr>
<tr>
<td>FT</td>
<td>VCF</td>
<td>##FORMAT=<id=ft,number=1,type=string,description="sample filter"="" genotype=""></id=ft,number=1,type=string,description="sample></td>
</tr>
<tr>
<td>GL</td>
<td>VCF</td>
<td>##FORMAT=<id=gl,number=.,type=float,description="genotype likelihoods"=""></id=gl,number=.,type=float,description="genotype></td>
</tr>
<tr>
<td>PL</td>
<td>VCF</td>
<td>##FORMAT=<id=pl,number=3,type=integer,description="normalized, a="ref" aa,ab,bb="" and="" applicable="" b="alt;" biallelic"="" for="" genotypes="" if="" is="" likelihoods="" not="" phred-scaled="" site="" where=""></id=pl,number=3,type=integer,description="normalized,></td>
</tr>
<tr>
<td>GP</td>
<td>VCF</td>
<td>##FORMAT=<id=gp,number=.,type=float,description="phred-scaled genotype="" posterior="" probabilities"=""></id=gp,number=.,type=float,description="phred-scaled></td>
</tr>
<tr>
<td>GQ</td>
<td>VCF</td>
<td>##FORMAT=<id=gq,number=.,type=integer,description="conditional genotype="" phred-scaled="" quality"=""></id=gq,number=.,type=integer,description="conditional></td>
</tr>
<tr>
<td>HQ</td>
<td>VCF</td>
<td>##FORMAT=<id=hq,number=2,type=integer,description="haplotype comma="" phred="" qualities"="" qualities,="" separated="" two=""></id=hq,number=2,type=integer,description="haplotype></td>
</tr>
<tr>
<td>CN</td>
<td>VCF</td>
<td>##FORMAT=<id=cn,number=1,type=integer,description="copy events"="" for="" genotype="" imprecise="" number=""></id=cn,number=1,type=integer,description="copy></td>
</tr>
<tr>
<td>CNQ</td>
<td>VCF</td>
<td>##FORMAT=<id=cnq,number=1,type=float,description="copy events"="" for="" genotype="" imprecise="" number="" quality=""></id=cnq,number=1,type=float,description="copy></td>
</tr>
<tr>
<td>CNL</td>
<td>VCF</td>
<td>##FORMAT=<id=cnl,number=.,type=float,description="copy events"="" for="" genotype="" imprecise="" likelihood="" number=""></id=cnl,number=.,type=float,description="copy></td>
</tr>
<tr>
<td>MQ</td>
<td>VCF</td>
<td>##FORMAT=<id=mq,number=1,type=integer,description="phred ancestor"="" genome's="" is="" novel="" probability="" respect="" score="" style="" that="" the="" to="" variant="" with=""></id=mq,number=1,type=integer,description="phred></td>
</tr>
<tr>
<td>HAP</td>
<td>VCF</td>
<td>##FORMAT=<id=hap,number=1,type=integer,description="unique haplotype="" identifier"=""></id=hap,number=1,type=integer,description="unique></td>
</tr>
<tr>
<td>AHAP</td>
<td>VCF</td>
<td>##FORMAT=<id=ahap,number=1,type=integer,description="unique ancestral="" haplotype"="" identifier="" of=""></id=ahap,number=1,type=integer,description="unique></td>
</tr>
<tr>
<td>SS</td>
<td>TCGA-VCF</td>
<td>##FORMAT=<id=ss,number=1,type=integer,description="variant modification,5='unknown"' non-adjacent="" normal,0="wildtype,1=germline,2=somatic,3=LOH,4=post-transcriptional" relative="" status="" to=""></id=ss,number=1,type=integer,description="variant></td>
</tr>
<tr>
<td>TE</td>
<td>TCGA-VCF</td>
<td>##FORMAT=<id=te,number=.,type=string,description="translational a="" codon"="" effect="" in="" of="" the="" variant=""></id=te,number=.,type=string,description="translational></td>
</tr>
<tr>
<td>AD</td>
<td>TCGA-VCF</td>
<td>##FORMAT=<id=ad,number=.,type=integer,description="depth 0="" 1="" 2="" 3..."="" alleles="" of="" reads="" supporting=""></id=ad,number=.,type=integer,description="depth></td>
</tr>
<tr>
<td>DP4</td>
<td>TCGA-VCF</td>
<td>##FORMAT=<id=dp4,number=4,type=integer,description="number alt-forward="" alt-reverse="" and="" bases"="" high-quality="" of="" ref-forward,="" ref-reverse,=""></id=dp4,number=4,type=integer,description="number></td>
</tr>
<tr>
<td>BQ</td>
<td>TCGA-VCF</td>
<td>##FORMAT=<id=bq,number=.,type=integer,description="average alleles"="" base="" for="" quality="" reads="" supporting=""></id=bq,number=.,type=integer,description="average></td>
</tr>
<tr>
<td>VAQ</td>
<td>TCGA-VCF</td>
<td>##FORMAT=<id=vaq,number=1,type=integer,description="variant allele="" quality"=""></id=vaq,number=1,type=integer,description="variant></td>
</tr>
</tbody>
</table>
<h3>TCGA-specific meta-information</h3>
<h4>PEDIGREE lines</h4>
<p><strong>Format</strong>: <em>##PEDIGREE=<key1=value1,key2=value2,...></key1=value1,key2=value2,...></em><br/>
<strong>Required keys</strong>: Name_0,..,Name_N where N &gt;= 1;</p>
<p>PEDIGREE lines are used to specify derivation relationships between different
genomes. <em>Name_0</em> is associated with the derived genome and <em>Name_1</em> through
<em>Name_N</em> represent the genomes from which it is derived. In the case of tumor
clonal populations, one population is clonally derived from another. In the
example below, PRIMARY-TUMOR-GENOME is derived from GERMLINE-GENOME.</p>
<pre><code>##PEDIGREE=&lt;Name_0=&lt;G0-ID&gt;,Name_1=&lt;G1-ID&gt;,...,Name_N=&lt;GN-ID&gt;&gt;

where N is \&gt;= 1;

Example:

##PEDIGREE=&lt;Name_0=PRIMARY-TUMOR-GENOME,Name_1=GERMLINE-GENOME&gt;
</code></pre>
<h4>SAMPLE lines</h4>
<p><strong>Format</strong>: <em>##SAMPLE=<key1=value1,key2=value2,...></key1=value1,key2=value2,...></em><br/>
<strong>Required keys</strong>: ID, SampleName, Individual, File, Platform, Source, Accession</p>
<p>For UUID-compliant files, following rules should be followed:</p>
<p><strong>Required keys</strong>: ID, SampleName, Individual, SampleUUID, SampleTCGABarcode,
File, Platform, Source, Accession</p>
<ul>
<li>
<p>Value assigned to "SampleUUID" should be a valid <a href="https://docs.gdc.cancer.gov/Encyclopedia/pages/UUID/">aliquot
    UUID</a> in the database.</p>
</li>
<li>
<p>Value assigned to "SampleTCGABarcode" should represent the aliquot-level
    metadata associated with SampleUUID. This metadata mapping is originally
    received by the DCC from BCR.</p>
</li>
</ul>
<blockquote>
<p>Example:</p>
<p>##SAMPLE=&lt;ID=NORMAL,SampleUUID=151626d0-a650-447a-a2b0-1df53c31c682,SampleTCGABarcode=TCGA-A1-A0SD-01A-11R-A114-13,Individual=TCGA-A1-A0SD,Description="Normal",File=TCGA-A1-A0SD-01A-11R-A114-13.bam,SequenceSource=Exome,Platform=Illumina,Source=dbGAP,Accession=1234,Genomes=<germline,tumor>,Mixture=&lt;0.1,0.9&gt;,Genome_Description=&lt;"Germline
  contamination","Tumor genome"&gt;&gt;</germline,tumor></p>
</blockquote>
<p>SAMPLE lines are used to include additional metadata about each sample for which
data is represented in the VCF file. All samples are listed in the column header
line following the FORMAT column (Figure 1). Each of these samples should have
its own HEADER declaration where the sample identifier in the column header
should be the same as the value assigned to "ID" key in the corresponding
declaration. Value assigned to "SampleName" should be a valid <a href="https://docs.gdc.cancer.gov/Encyclopedia/pages/TCGA_Barcode/">aliquot
barcode</a> / <a href="https://docs.gdc.cancer.gov/Encyclopedia/pages/UUID/">UUID</a>
in the database. The declaration lists information about the sample (source,
platform, source file, etc.) and can also be used to indicate if the sample is a
mixture of different kind of genomes. In the example below, "Genomes", "Mixture"
and 'Genome_Description" tags represent comma-separated list of different
genomes that a sample contains, proportion of each genome in the sample, and a
brief description of each genome respectively.</p>
<h2 id='SAMPLE=&lt;ID=id,SampleName=sampleName,Individual=individual,Description="description",'>SAMPLE=&lt;ID=id,SampleName=sampleName,Individual=individual,Description="description",</h2>
<p>File=bamfile,Platform=platformName,Source=source,Accession=acc,Genomes=<g1-id,g2-id,..,gk-id>,Mixture=
<n1,n2,..,nk>,Genome_Description=&lt;"S1","S2",..,"SK"&gt;&gt;</n1,n2,..,nk></g1-id,g2-id,..,gk-id></p>
<p>Example:</p>
<h2 id="None">SAMPLE=&lt;ID=NORMAL,SampleName=TCGA-06-0881-10A-01W,Individual=TCGA-06-0881,Description="Normal",File=TCGA-06-0881-10A-01W-0421-09.bam,Platform=Illumina,Source=dbGAP,Accession=1234,Genomes=<germline,tumor>,Mixture=&lt;0.1,0.9&gt;,Genome_Description=&lt;"Germline</germline,tumor></h2>
<p>contamination","Tumor genome"&gt;&gt;</p>
<ul>
<li>
<p>"Description" field for genome mixture has been renamed to
    "Genome_Description" to distinguish it from sample description.</p>
</li>
<li>
<p>Values for tags related to genome mixture (Genomes, Mixture,
    Genome_Description) are within angle brackets.</p>
</li>
</ul>
<h3>Column header meta-information</h3>
<p><strong>Format</strong>: Tab-delimited line starting with "#" and containing headers for all
columns in the BODY as shown below.</p>
<p>This is a mandatory header line where the first 8 fields are fixed and have to
defined in the column header. "FORMAT" onwards are optional and are included to
encapsulate per-sample/genome genotype data.</p>
<h1><i class="fas fa-book"></i>   CHROM POS ID REF ALT QUAL FILTER INFO FORMAT <sample1 genome1="" or=""> &lt;SAMPLE2</sample1></h1><hr/>
<p>or GENOME2&gt; ...</p>
<p>BODY</p>
<h3>Variant records</h3>
<p>Data lines are tab-delimited and list information about individual variants and
associated genotypes across samples. The first 8 fields (Figure 1) are required
to be listed in the VCF column header line. Some of these fields require
non-null values (see Table 6) for each record. For the remaining fixed fields,
even if the field does not have an associated value, it still needs to be
specified with a missing value identifier ("." in VCF 4.1). Subsequent fields
are optional.</p>
<p><strong>Table 6: Description of fields in the BODY of a VCF file</strong></p>
<table class="table">
<thead>
<tr>
<th><strong>Index</strong></th>
<th><strong>Field</strong></th>
<th><strong>Case-sensitive</strong></th>
<th><strong>Description</strong></th>
<th><strong>Data type</strong></th>
<th><strong>Sample values</strong></th>
<th><strong>Required*</strong></th>
<th><strong>Additional notes</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>CHROM</td>
<td>Yes</td>
<td><em>Chromosome</em>: an identifier from the reference genome or the assembly file defined in the HEADER.</td>
<td>Alphanumeric string</td>
<td>20</td>
<td>Yes</td>
<td>Chromosome name should not contain "chr" prefix, e.g., "chr10" will be an invalid entry</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td><em>([1-22], X, Y, MT, <id>)</id></em></td>
<td><ctg1></ctg1></td>
<td></td>
<td></td>
</tr>
<tr>
<td>2</td>
<td>POS</td>
<td>Yes</td>
<td><em>Position</em>: The reference position, with the 1st base having position 1.</td>
<td>Non-negative integer</td>
<td>1110696</td>
<td>Yes</td>
<td>---</td>
</tr>
<tr>
<td>3</td>
<td>ID</td>
<td>Yes</td>
<td><em>Identifier</em>: Semi-colon separated list of unique identifiers if available.</td>
<td>String, no white-space or semi-colons</td>
<td>rs6054257_66370</td>
<td>No</td>
<td><strong>Important</strong>: When using an rsID as the variant identifier, please append chromosomal location of the variant to the ID. For example, if the variant is at chr7:6013153 and the corresponding rsID is rs10000, then the variant ID should be rs10000_6013153. This is to ensure that there is a consistent rule for satisfying the condition for unique IDs even if a file contains single rsID that maps to multiple variants.</td>
</tr>
<tr>
<td>4</td>
<td>REF</td>
<td>Yes</td>
<td><em>Reference allele(s)</em>: Reference allele at the position.</td>
<td>String</td>
<td>GTCT</td>
<td>Yes</td>
<td>Value in POS field refers to the position of the first base in the REF string.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td><em>([ACGTN]+</em> )</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>5</td>
<td>ALT</td>
<td>Yes</td>
<td><em>Alternate allele(s)</em>: Comma separated list of alternate non-reference alleles called on at least one of the samples. Angle-bracketed ID String (\&lt;ID&gt;) can also be used for symbolically representing alternate alleles.</td>
<td>String; no whitespace, commas, or angle-brackets in the ID string</td>
<td>G,GTCT</td>
<td>No</td>
<td>if ALT==<id>, ID needs to be defined in the header as</id></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td><em>([ACGTN]+, <id>, .)</id></em></td>
<td>.</td>
<td></td>
<td>##ALT=<id=*id*,description=*"description"*></id=*id*,description=*"description"*></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td><ins:me:alu></ins:me:alu></td>
<td></td>
<td></td>
</tr>
<tr>
<td>6</td>
<td>QUAL</td>
<td>Yes</td>
<td><em>Quality score</em>: Phred-scaled quality score for the assertion made in ALT.</td>
<td>Integer &gt;= 0</td>
<td>50</td>
<td>No</td>
<td>Scores should be non-negative integers or missing values</td>
</tr>
<tr>
<td>7</td>
<td>FILTER</td>
<td>Yes</td>
<td><em>Filtering results</em>: PASS if this position has passed all filters, Otherwise, if the site has not passed all filters, a semicolon-separated list of codes for filters that fail.</td>
<td>String, no whitespace or semi-colon</td>
<td>PASS</td>
<td>No</td>
<td>"0" is reserved and cannot be used as a filter String.</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>q10;s50</td>
<td></td>
<td></td>
</tr>
<tr>
<td>8</td>
<td>INFO</td>
<td>Yes</td>
<td><em>Additional information</em>: INFO fields are encoded as a semicolon-separated series of keys (same as ID in an INFO declaration) with optional values in the format <em><key=value></key=value></em>.</td>
<td>String, no whitespace, semi-colons, or equal-signs</td>
<td>NS=3;DP=14;</td>
<td>No</td>
<td>---</td>
</tr>
<tr>
<td>9</td>
<td>FORMAT</td>
<td>Yes</td>
<td><em>Genotype sub-fields</em>: If genotype data is present in the file, the fixed fields are followed by a FORMAT column. The field contains a colon-separated list of all pre-defined FORMAT sub-fields (same as ID in a FORMAT declaration) that are applicable to all samples that follow.</td>
<td>String, no whitespace, sub-fields cannot contain colon</td>
<td>GT:GQ:DP:HQ</td>
<td>No</td>
<td>"GT" must be the first sub-field if it is present in the FORMAT field.</td>
</tr>
<tr>
<td>10</td>
<td><sample></sample></td>
<td>Case should be same as in "ID" tag of ##SAMPLE declaration in the header</td>
<td><em>Per-sample genotype information</em>: An arbitrary number of sample IDs can be added to the column header line and a variant record in the BODY can contain genotype information corresponding to FORMAT column for each sample. Contains a colon-separated list of values assigned to each of the sub-fields in FORMAT column.</td>
<td>String, no whitespace, sub-fields cannot contain colon</td>
<td>0|0:48:1:51,51</td>
<td>No</td>
<td>Values are assigned to FORMAT sub-fields in the SAME order as specified in "FORMAT" column. All samples in any given row for a variant record MUST contain values for all sub-fields as defined in "FORMAT" column. If any of the fields does not have an associated value, then missing value identifier (".") should be used for that field. However, "." cannot be used as a value for any of the IDs in the FORMAT field (e.g., GT:.:DP would lead to an error).</td>
</tr>
</tbody>
</table>
<ul>
<li>A "Required" field cannot contain missing value identifier for any record
listed in data lines.</li>
</ul>
<h1><i class="fas fa-book"></i>   Extensions for TCGA data</h1><hr/>
<p>TCGA data includes but is not limited to SNP's and small indels. A variant
representation format for cancer data should be able to support more complex
variation types such as structural variants, complex rearrangements and RNA-Seq
variants. The following sub-sections present an overview of the extensions that
have been added to clearly describe such variations in a VCF file.</p>
<h2 id="Structural variants">Structural variants</h2>
<p>A <a href="http://www.ncbi.nlm.nih.gov/dbvar/content/overview/">structural variant</a> (SV)
can be defined as a region of DNA that includes a variation in the structure of
the chromosome. Such variations could be due to inversions and balanced
translocations or genomic imbalances (insertions and deletions), also referred
to as copy number variants (CNVs). Certain features have been added to the
format in order to clearly describe structural variants in a VCF file. A
detailed description of the extensions is available
<a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">here</a>.</p>
<h2 id="Complex rearrangements">Complex rearrangements</h2>
<p>Chromosomal rearrangements are caused by breakage of DNA double helices at two
different locations. The broken ends in turn rejoin to produce a new chromosomal
arrangement. Complex rearrangements involving more than two breaks are
frequently observed in cancer genomes. Certain modifications need to be made to
the VCF standard to adequately represent such variations in a VCF file. A
detailed specification of the proposed extensions to describe rearrangements in
a VCF file is available
<a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">here</a>.
Figure 2 illustrates some of the concepts relevant to VCF records for complex
rearrangements.</p>
<p><strong>Figure 2: Adjacencies and breakends in a chromosomal rearrangement</strong> (adapted
from VCF 4.1 specification)</p>
<table class="table">
<thead>
<tr>
<th><img alt="media" src="images/ccr_VCF.png"/></th>
</tr>
</thead>
<tbody>
<tr>
<td></td>
</tr>
</tbody>
</table>
<p>A VCF file has one line for each of the two breakends in an adjacency. Table 7
provides a list of sub-fields that have been added to describe breakends. An
INFO sub-field (<strong>SVTYPE=BND</strong>) is used to indicate a breakend record.
Sub-fields MATEID and PARID are used to represent variant record IDs of
corresponding mates and partners respectively.</p>
<p><strong>Table 7: Fields added for breakends</strong></p>
<table class="table">
<thead>
<tr>
<th><strong>Field:Sub-field</strong></th>
<th><strong>Description</strong></th>
<th><strong>Declaration in HEADER</strong></th>
<th><strong>Required</strong></th>
<th><strong>(Sample values in BODY)</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>INFO:<strong>SVTYPE</strong></td>
<td>Type of structural variant; SVTYPE is set to "BND" for breakend records</td>
<td>##INFO=<id=svtype,number=1,type=string,description="type of="" structural="" variant"=""></id=svtype,number=1,type=string,description="type></td>
<td>Yes</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td><em>SVTYPE=BND</em></td>
<td>(SVTYPE=BND for breakend records)</td>
<td></td>
</tr>
<tr>
<td>INFO:<strong>MATEID</strong></td>
<td>ID of corresponding mate of the breakend record</td>
<td>##INFO=<id=mateid,number=.,type=string,description="id breakend"="" mate="" of=""></id=mateid,number=.,type=string,description="id></td>
<td>No</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td><em>MATEID=bnd_U</em></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO:<strong>PARID</strong></td>
<td>ID of corresponding partner of the breakend record</td>
<td>##INFO=<id=parid,number=.,type=string,description="id breakend"="" of="" partner=""></id=parid,number=.,type=string,description="id></td>
<td>No</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td><em>PARID=bnd_V</em></td>
<td></td>
<td></td>
</tr>
<tr>
<td>INFO:<strong>EVENT</strong></td>
<td>ID of event associated to breakend</td>
<td>##INFO=<id=event,number=.,type=string,description="id breakend="" event"="" of=""></id=event,number=.,type=string,description="id></td>
<td>No</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td><em>EVENT=RR0</em></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<p>The specification for ALT field deviates from the standard format for breakend
records. ALT field for a breakend record can be represented in four possible
ways based on the type of replacement.</p>
<p>REF ALT Description</p>
<p>s t[p[ piece extending to the right of p is joined after t</p>
<p>s t]p] reverse comp piece extending left of p is joined after t</p>
<p>s ]p]t piece extending to the left of p is joined before t</p>
<p>s [p[t reverse comp piece extending right of p is joined before t</p>
<p>Legend:</p>
<p>s: sequence of REF bases beginning at position POS</p>
<p>t: sequence of bases that replaces "s"</p>
<p>p: position of the breakend mate indicating the first mapped base that joins at
the adjacency; represented as a string of the form "chr:pos"</p>
<p>[]: square brackets indicate direction that the joined sequence continues in,
starting from p</p>
<h2 id="RNA-Seq variants">RNA-Seq variants</h2>
<p>VCF specifications have been extended to address expressed variants obtained
from RNA-Seq. Features added for structural variants from genome/exome
sequencing are applicable to RNA-Seq structural variants. However, RNA-Seq
breakends are represented by setting <strong>SVTYPE=FND</strong> instead of BND (Table 8)
since they can be different from those observed in DNA-Seq.</p>
<p><strong>Table 8: Fields added for RNA-Seq variants</strong></p>
<table class="table">
<thead>
<tr>
<th><strong>Field:Sub-field</strong></th>
<th><strong>Description</strong></th>
<th><strong>Declaration in HEADER</strong></th>
<th><strong>Required</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>INFO:<strong>SVTYPE</strong></td>
<td>Type of structural variant; SVTYPE is set to "FND" for breakends associated with RNA-Seq</td>
<td>##INFO=<id=svtype,number=1,type=string,description="type of="" structural="" variant"=""></id=svtype,number=1,type=string,description="type></td>
<td>Yes</td>
</tr>
<tr>
<td></td>
<td></td>
<td><em>SVTYPE=FND</em></td>
<td>(required for RNA-Seq breakend records; SVTYPE=FND)</td>
</tr>
</tbody>
</table>
<p>VCF files for RNA-Seq variants may include gene-related annotations. However,
this is not a standard feature of VCF files as eventually all VCF variants will
be annotated using information in Generic Annotation File (GAF). Additional INFO
and FORMAT sub-fields have been included to describe the characteristics of
expressed nucleotide variants (Table 8a).</p>
<p><strong>Table 8a: Annotation fields added for RNA-Seq variants</strong></p>
<table class="table">
<thead>
<tr>
<th><strong>Field:Sub-field</strong></th>
<th><strong>Description</strong></th>
<th><strong>Declaration in HEADER</strong></th>
<th><strong>Required</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>INFO:<strong>SID</strong></td>
<td>Unique identifiers from the gene annotation source as specified in ##geneAnno; "unknown" should be used if identifier is not known; comma-separated list of IDs can be used if variant overlaps with multiple features</td>
<td>##INFO=<id=sid,number=.,type=string,description=unique annotation="" from="" gene="" identifier="" or="" source="" unknown=""></id=sid,number=.,type=string,description=unique></td>
<td>No</td>
</tr>
<tr>
<td></td>
<td></td>
<td><em>SID=13,198</em></td>
<td></td>
</tr>
<tr>
<td>INFO:<strong>GENE</strong></td>
<td>HUGO gene symbol; "unknown" should be used when gene symbol is unknown; comma-separated list of genes can be used if variant overlaps with multiple transcripts/genes</td>
<td>##INFO=<id=gene,number=.,type=string,description=hugo gene="" symbol=""></id=gene,number=.,type=string,description=hugo></td>
<td>No</td>
</tr>
<tr>
<td></td>
<td></td>
<td><em>GENE=ERBB2,ERBB2</em></td>
<td></td>
</tr>
<tr>
<td>INFO:<strong>RGN</strong></td>
<td>Region where a nucleotide variant occurs in relation to a gene</td>
<td>##INFO=<id=rgn,number=.,type=string,description=region a="" gene="" in="" nucleotide="" occurs="" relation="" to="" variant="" where=""></id=rgn,number=.,type=string,description=region></td>
<td>No</td>
</tr>
<tr>
<td></td>
<td></td>
<td><em>RGN=exon,3_utr</em></td>
<td></td>
</tr>
<tr>
<td>INFO:<strong>RE</strong></td>
<td>Flag to indicate if position is known to have RNA-edits occur</td>
<td>##INFO=<id=re,number=0,type=flag,description=position have="" known="" occur="" rna-edits="" to=""></id=re,number=0,type=flag,description=position></td>
<td>No</td>
</tr>
<tr>
<td></td>
<td></td>
<td><em>RE</em></td>
<td></td>
</tr>
<tr>
<td>FORMAT:<strong>TE</strong></td>
<td>Translational effect of a nucleotide variant in a codon</td>
<td>##FORMAT=&lt;ID=TE,Number=.,Type=String,Description="Translational</td>
<td>No</td>
</tr>
<tr>
<td></td>
<td></td>
<td>effect of the variant in a codon"&gt;</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td><em>MIS,NA</em></td>
<td></td>
</tr>
</tbody>
</table>
<h2 id="Including validation status in VCF file">Including validation status in VCF file</h2>
<p>Somatic variations are often validated using follow-up experiments to confirm
the variant is not due to sequencing errors. Following points need to be
considered while including validation status in VCF file:</p>
<ul>
<li>
<p>A single VCF file will contain sequence data for a single case. The file
    could be the result of merging calls from different centers so validation
    can be performed on a set of variants reported in a merged VCF file.</p>
</li>
<li>
<p>Validation with secondary technology is performed after obtaining results
    from primary sequencing method. Therefore, validation is a confirmation step
    and may or may not be performed before a first-pass VCF file with all
    candidate mutations is generated and submitted to the DCC.</p>
</li>
<li>
<p>A single mutation can be verified with multiple independent methods and the
    results may or may not be in agreement.</p>
</li>
<li>
<p>If results from different methods are in conflict, the final validation
    status of the variant call needs to be inferred based on available
    information. This could be done manually or programatically.</p>
</li>
</ul>
<p><strong>Format validation</strong></p>
<p>Since validation data is added as additional genotype/sample columns, the file
will pass validation as long as all existing format rules are followed and
header declarations are correct.</p>
<p><strong>Sample TCGA VCF file with validation status</strong></p>
<p>Line1 ##fileformat=VCFv4.1</p>
<p>Line2 ##tcgaversion=1.1</p>
<p>Line3 ##fileDate=20120205</p>
<p>Line4 ##reference=file:///seq/references/1000GenomesPilot-NCBI36.fasta</p>
<p>Line5 ##FORMAT=<id=gt,number=1,type=string,description="genotype"></id=gt,number=1,type=string,description="genotype"></p>
<p>Line6 ##FORMAT=<id=gq,number=1,type=integer,description="genotype quality"=""></id=gq,number=1,type=integer,description="genotype></p>
<p>Line7 ##FORMAT=<id=ss,number=1,type=integer,description="variant 0='wildtype,1=germline,2=somatic,3=LOH,4=unknown"' non-adjacent="" normal,="" relative="" status="" to=""></id=ss,number=1,type=integer,description="variant></p>
<p>Line8 ##INFO=<id=vls,number=1,type=integer,description="final 0='wildtype,1=germline,2=somatic,3=LOH,4=unknown"' non-adjacent="" normal,="" relative="" status="" to="" validation=""></id=vls,number=1,type=integer,description="final></p>
<p>Line9 ##FILTER=<id=q10,description="quality 10"="" below=""></id=q10,description="quality></p>
<p>Line10</p>
<h2 id="None">SAMPLE=<id=normal,samplename=tcga-06-0881-10a-01w,individual=tcga-06-0881,description="normal",file=tcga-06-0881-10a-01w-0421-09_illumina.bam,platform=illumina,source=dbgap,accession=1234></id=normal,samplename=tcga-06-0881-10a-01w,individual=tcga-06-0881,description="normal",file=tcga-06-0881-10a-01w-0421-09_illumina.bam,platform=illumina,source=dbgap,accession=1234></h2>
<p>Line11</p>
<h2 id='SAMPLE=&lt;ID=NORMAL_454,SampleName=TCGA-06-0881-10A-01W,Individual=TCGA-06-0881,Description="Validation'>SAMPLE=&lt;ID=NORMAL_454,SampleName=TCGA-06-0881-10A-01W,Individual=TCGA-06-0881,Description="Validation</h2>
<p>normal sample tested with
454",File=TCGA-06-0881-10A-01W-0421-09_454.bam,Platform=454,Source=dbGAP,Accession=245&gt;</p>
<p>Line12</p>
<h2 id="None">SAMPLE=<id=tumor,samplename=tcga-06-0881-01a-01w,individual=tcga-06-0881,description="tumor",file=tcga-06-0881-01a-01w-0421-09.bam,platform=illumina,source=dbgap,accession=1234></id=tumor,samplename=tcga-06-0881-01a-01w,individual=tcga-06-0881,description="tumor",file=tcga-06-0881-01a-01w-0421-09.bam,platform=illumina,source=dbgap,accession=1234></h2>
<p>Line13</p>
<h2 id='SAMPLE=&lt;ID=TUMOR_454,SampleName=TCGA-06-0881-01A-01W,Individual=TCGA-06-0881,Description="Validation'>SAMPLE=&lt;ID=TUMOR_454,SampleName=TCGA-06-0881-01A-01W,Individual=TCGA-06-0881,Description="Validation</h2>
<p>tumor sample tested with
454",File=TCGA-06-0881-01A-01W-0421-09_454.bam,Platform=454,Source=dbGAP,Accession=3456&gt;</p>
<p>Line14</p>
<h2 id='SAMPLE=&lt;ID=TUMOR_Sanger,SampleName=TCGA-06-0881-01A-01W,Individual=TCGA-06-0881,Description="Validation'>SAMPLE=&lt;ID=TUMOR_Sanger,SampleName=TCGA-06-0881-01A-01W,Individual=TCGA-06-0881,Description="Validation</h2>
<p>tumor sample tested with Sanger
seq",File=.,Platform=Sanger_PCR,Source=.,Accession=.&gt;</p>
<p>Line15 #CHROM POS ID REF ALT QUAL FILTER INFO FORMAT NORMAL TUMOR NORMAL_454
TUMOR_454 TUMOR_Sanger</p>
<p>Line16 20 14370 var1 G A 29 PASS VLS=2 GT:GQ:SS 0/0:48:. 0/1:50:2 0/0:20:.
0/1:20:2 0/1:.:2</p>
<p>Line17 5 15000 var2 T C 35 PASS VLS=1 GT:GQ:SS 0/1:48:. 1/1:51:3 0/1:60:.
0/1:50:1 0/1:13:1</p>
<p>Line18 3 170089 var2 G T 30 PASS . GT:GQ:SS 0/1:48:. 0/1:51:1 .:.:. .:.:. .:.:.</p>
<p>The format follows these guidelines:</p>
<ol>
<li>
<p><strong>Sample columns</strong></p>
<ul>
<li>
<p>An additional column is included for every line of evidence used for
    validation. In the example above, tumor calls are verified with 454 and
    Sanger sequencing and normal calls are validated with 454. Therefore, 3
    genotype columns exist in addition to the NORMAL and TUMOR sequencing
    calls obtained with the primary sequencing method.</p>
</li>
<li>
<p>The validation platform name is appended to the original sample to
    distinguish the validation results from primary sequencing.
    <sample>_<platform> is used in the example above.</platform></sample></p>
<ul>
<li><strong>Note</strong>: \&lt;Platform&gt; can be obtained from DCC <a href="http://tcga-data.nci.nih.gov/datareports/codeTablesReport.htm">Code Tables
    Report</a>.
    The ##SAMPLE meta-information line also includes a 'Platform' tag
    where platform name is defined.</li>
</ul>
</li>
<li>
<p>Each new genotype column header added to the file (e.g., TUMOR_454,
    TUMOR_Sanger) has to be defined in the header using the ##SAMPLE
    meta-information line (e.g., Lines 13 and 14).</p>
</li>
<li>
<p>As per VCF specification, the order of FORMAT sub-fields is defined by
    the FORMAT column and all calls from primary and validation sequencing
    should comply with this order.</p>
</li>
<li>
<p>If a sub-field does not apply to any given validation call, it should be
    assigned a missing value (".").</p>
</li>
</ul>
</li>
<li>
<p><strong>FORMAT sub-field "SS"</strong></p>
<ul>
<li>For any given tumor genotype call, sub-field SS indicates variant status
    with respect to non-adjacent normal counterpart (0, 1, 2, 3, 4 or 5
    based on whether the variant is wildtype, germline, somatic, LOH,
    post-transcriptional modification, or unknown respectively). Therefore,
    each tumor genotype call (primary and secondary sequencing) will have
    its own corresponding SS sub-field.</li>
</ul>
</li>
<li>
<p><strong>INFO sub-field "VLS"</strong></p>
<ul>
<li>Sub-field VLS represents an inferred decision for a tumor genotype call
    and is based on the calls obtained with validation. In the example
    above, var1 shows a somatic call (SS=2) for the tumor sample based on
    primary sequencing, and both validation methods confirm this call.
    Therefore, the final validation status of var1 is a somatic variation
    (VLS=2). However, var2 has a LOH variant in tumor sample (SS=3) based on
    primary sequencing whereas both validation methods indicate that it is a
    germline variant (SS=1). In such a case, "VLS" has to be inferred from
    available information and could differ from the SS value assigned to the
    tumor sample based on primary sequencing.</li>
</ul>
</li>
</ol>
<h1><i class="fas fa-book"></i>   Validation rules</h1><hr/>
<p>At the minimum, every file needs to go through the checks listed below.
Following is an example of a VCF file that shows certain violations cited in the
listed validation steps. Please note that line numbers in the file segment below
are added for illustration purposes alone and are not expected to be found in an
actual VCF file.</p>
<p>Line1 ##fileformat=VCFv4.1</p>
<p>Line2 ##fileDate=20090805</p>
<p>Line3 ##source=myImputationProgramV3.1</p>
<p>Line4 ##reference=file:///seq/references/1000GenomesPilot-NCBI36.fasta</p>
<p>Line5 ##INFO=<id=ns,number=1,type=integer,description="number data"="" of="" samples="" with=""></id=ns,number=1,type=integer,description="number></p>
<p>Line6 ##INFO=<id=db,number=0,type=flag,description="dbsnp membership"=""></id=db,number=0,type=flag,description="dbsnp></p>
<p>Line7 ##FORMAT=<id=gt,number=1,type=string,description="genotype"></id=gt,number=1,type=string,description="genotype"></p>
<p>Line8 ##FORMAT=<id=gq,number=1,type=integer,description="genotype quality"=""></id=gq,number=1,type=integer,description="genotype></p>
<p>Line9 ##FORMAT=<id=dp,number=1,type=integer,description="read depth"=""></id=dp,number=1,type=integer,description="read></p>
<p>Line10 ##FORMAT=<id=pl,number=3,type=integer,description=" "="" aa,="" ab,="" bb="" for="" genotypes="" likelihoods="" normalized="" phred-scaled=""></id=pl,number=3,type=integer,description="></p>
<p>Line11 ##FILTER=<id=q10,description="quality 10"="" below=""></id=q10,description="quality></p>
<p>Line12 ##FILTER=<id=s50,description="less 50%="" data"="" have="" of="" samples="" than=""></id=s50,description="less></p>
<p>Line13 FILTER=<id=c10,description="shallow 10x"="" below="" coverage=""></id=c10,description="shallow></p>
<p>Line14 ##ALT=<id=del:me:alu,description="deletion alu="" element"="" of=""></id=del:me:alu,description="deletion></p>
<p>Line15 #CHROM POS ID REF ALT QUAL FILTER INFO FORMAT TCGA-02-0001-01
TCGA-02-0001-02</p>
<p>Line16 20 14370 var1 G A 29 q10 NS=2;DP=14 GT:GQ:DP 0|0:48 0|1:48:3</p>
<p>Line17 19 15000 var2 G A 35 q10;s50 NS=2.5 GQ:GT 48:0|0 51:0|1</p>
<p>Line18 19 16000 var3 C T 30 q10;s10 NS=2 GT:GQ:DP 0/2:48:3 0/1:51:4</p>
<p>Line19 2 14477 rs123 C \&lt;DEL:ME:ALU&gt; 12 PASS NS=3;DB GT:GQ 0/1:50 1/1:40</p>
<p>Line20 9 13567 . A \&lt;DUP&gt; 20 PASS NS=3 GT:GQ:PL 0/1:49:42,3 1/1:38:96,47/70</p>
<p>Line21 3 18901 rs456 T C 15 PASS NS=3/DB GT 0/1 1/1</p>
<p><strong>Important</strong>: A file will be validated as a TCGA VCF file only if it contains</p>
<h2 id="tcgaversion HEADER line (e.g., ##tcgaversion=1.1). The current acceptable">tcgaversion HEADER line (e.g., ##tcgaversion=1.1). The current acceptable</h2>
<p>version is 1.1.</p>
<ol>
<li>
<p>Mandatory <a href="#TCGAVariantCallFormat(VCF)1.1Specificat">header lines</a> should be
    present.</p>
</li>
<li>
<p>All meta-information header lines should be prefixed with "##".</p>
</li>
<li>
<p><a href="#TCGAVariantCallFormat(VCF)1.1Specificat">Column header</a> line should be
    prefixed with "#". A VCF file can contain only a single column header line
    that must contain all required field names.</p>
</li>
<li>
<p>Any line lacking the "##" or "#" prefix will be assumed to be a BODY data
    line and will have to follow the specified format. For example, Line13 leads
    to a violation as it lacks "##" or "#" but is not a tab-delimited row
    containing variant information.</p>
</li>
<li>
<p>HEADER lines cannot be present within the BODY of a file and vice-versa.</p>
</li>
<li>
<p><a href="#TCGAVariantCallFormat(VCF)1.1Specificat">INFO</a>,
    <a href="#TCGAVariantCallFormat(VCF)1.1Specificat">FORMAT</a> and
    <a href="#TCGAVariantCallFormat(VCF)1.1Specificat">FILTER</a>declarations should follow
    the format below where all keys are required but the order of keys is
    irrelevant.</p>
</li>
<li>
<h2 id="None">INFO=<id=id,number=number,type=type,description="description"></id=id,number=number,type=type,description="description"></h2>
</li>
<li>
<h2 id="None">FORMAT=<id=id,number=number,type=type,description="description"></id=id,number=number,type=type,description="description"></h2>
</li>
<li>
<h2 id="None">FILTER=<id=id,description="description"></id=id,description="description"></h2>
</li>
<li>
<p>Values assigned to <em>ID, Number, Type</em> and <em>Description</em> in INFO, FORMAT or
    FILTER declarations should follow the rules listed below. A detailed
    description of the declaration format is provided
    <a href="#TCGAVariantCallFormat(VCF)1.1Specificat">here</a>.</p>
<ol>
<li>
<p>If an INFO or FORMAT sub-field exists in Table 4 or 5 respectively (i.e.
    ID of the sub-field matches value in "Sub-field" column of the table)
    then <em>ID</em>, <em>Number</em>, <em>Type</em> and <em>Description</em> values for that sub-field
    declaration must match the corresponding value in "Formatted
    declaration" column of the table for that sub-field. (TCGA VCF 1.1)</p>
</li>
<li>
<p>ID, Number, Type !\~ /(\s|,|=|;)/</p>
</li>
<li>
<p><em>Number</em> is in {Integer&gt;=0, "A", "G", "."}</p>
</li>
<li>
<p><em>Type</em> is in {Integer, String, Float, Flag, Character}</p>
</li>
<li>
<p><em>Description</em> should be within double quotes and cannot itself contain a
    double quote</p>
</li>
<li>
<p><em>Description</em> string cannot contain leading or trailing whitespace after
    opening or before closing quotation marks; Line10 shows a violation as
    <em>Description</em> string contains leading and trailing whitespace.</p>
</li>
<li>
<p>If ID == "FORMAT", then Type != "Flag"</p>
</li>
</ol>
</li>
<li>
<p>Any INFO, FORMAT or FILTER sub-fields used in the BODY are required to be
    defined in the HEADER. For example, var1 (Line16) shows an example of a
    violation as read depth "DP" is assigned a value (DP=14) without being
    defined as an INFO sub-field in the HEADER.</p>
</li>
<li>
<p>Validation of <strong>INFO</strong> sub-fields:</p>
<ol>
<li>
<p>An INFO sub-field should be included for a variant record in the BODY as
    <em>\&lt;key=value&gt;</em> (e.g., NS=2) where <em>key</em>is the "ID" value of the
    sub-field in the HEADER declaration.</p>
<ul>
<li><strong>Exception</strong>: An INFO field of "Flag" <em>Type</em> will not be assigned a
    value in the BODY. The presence of a flag in INFO column merely
    indicates that the variant record satisfies a condition associated
    with the flag. For example, Line19 has a "DB" flag without a value
    entry in the INFO column. "DB" in the INFO column indicates that the
    variant exists in dbSNP.</li>
</ul>
</li>
<li>
<p>Multiple INFO sub-fields can be associated with a single variant record
    using ";" as a separator (e.g., Line16). Line21 has a violation as "/"
    is used as a separator in INFO column.</p>
</li>
<li>
<p>If INFO field "VLS" is defined for a record, its value can only be 0, 1,
    2, 3, 4, or 5 based on whether the mutation is wildtype, germline,
    somatic, LOH, post-transcriptional modification, or unknown.</p>
</li>
</ol>
</li>
<li>
<p>Validation of <strong>FORMAT</strong> sub-fields:</p>
<ol>
<li>
<p>FORMAT column for a variant record contains a colon-separated list of
    all pre-defined FORMAT sub-fields (identified by "ID" value in the
    HEADER declaration) that are applicable to all samples that follow. A
    ":" is the only valid separator for sub-fields.</p>
</li>
<li>
<p>Number of colon-separated sub-fields in FORMAT column should equal to
    number of colon-separated values assigned to each sample. For example,
    var1 (Line16) violates this rule for the sample TCGA-02-0001-01 as there
    are 3 sub-fields in FORMAT column but only 2 values in the sample
    column.</p>
</li>
<li>
<p>Following FORMAT fields are required for all variant records in a VCF
    file. Missing value (".") is allowed for these fields.</p>
<ul>
<li>
<p>Genotype (<strong>GT</strong>)</p>
</li>
<li>
<p>Read depth (<strong>DP</strong>)</p>
</li>
<li>
<p>Reads supporting ALT (<strong>AD</strong> or <strong>DP4</strong>)</p>
</li>
<li>
<p>Average base quality for reads supporting alleles (<strong>BQ</strong>)</p>
</li>
<li>
<p>Somatic status of the variant (<strong>SS</strong>). SS can be 0, 1, 2, 3, 4, or
    5 depending on whether relative to normal the variant is wildtype,
    germline, somatic, LOH, post-transcriptional modification, or
    unknown respectively</p>
</li>
</ul>
</li>
<li>
<p><em>GT</em> must be the first sub-field in the string FORMAT. For example, var2
    (Line17) violates this rule as GT is not the first sub-field even though
    it is present in the FORMAT field.</p>
<ul>
<li>
<p>GT is a required sub field for all variants. Missing value (".") is
    allowed for GT. GT is not a required sub field and can be omitted
    for a variant row if none of the samples have genotype calls
    available (TCGA VCF 1.1)</p>
</li>
<li>
<p><em>GT</em> represents the genotype, encoded as allele values separated by
    either of / (genotype unphased) or | (genotype phased). The
    allele values are 0 for the reference allele (in REF field), 1 for
    the first allele listed in ALT, 2 for the second allele list in ALT
    and so on. Examples: 0/1, 1|0, or 1/2, etc.</p>
</li>
<li>
<p><em>GT</em>is assigned only one allele value for haploid calls (e.g. on Y
    chromosome). Therefore, if CHROM=="Y" then<em>GT</em>should have only one
    allele value assigned to it (e.g., "1", "0", ".", etc.) instead of
    two alleles (e.g., "1/1", "0|0"). If CHROM=="MT" then There is no
    constraint on the number of alleles as long as the number is bounded
    within the alleles listed in REF and/or ALT (e.g., 0/1, 0/1/2, 1 are
    all valid values for MT if REF and ALT have one and two allele
    values respectively).</p>
</li>
<li>
<p>All samples should have values assigned to <em>GT</em> for any given
    variant. If an allele cannot be called for a sample at a given
    locus, . will be specified for each missing allele in the <em>GT</em>
    field (for example "./." for a diploid genotype and "." for haploid
    genotype).</p>
</li>
<li>
<p>Validation should include ensuring that allele number in <em>GT</em> is
    within the range of alleles specified in ALT and REF. For example,
    var3 (Line18) violates this rule as it lists GT as "0/2" for sample
    TCGA-02-0001-01 but ALT contains only one allele so the only
    acceptable allele numbers are 0 (REF) and 1 (ALT).</p>
</li>
</ul>
</li>
</ol>
</li>
<li>
<p>If an INFO or FORMAT sub-field is declared in the header AND is assigned a
    value for a variant record in the body, the data type should be consistent
    with the expected type defined in the <em>Type</em>key of the corresponding
    declaration. For example, var2 (Line17) violates this rule as the definition
    for "NS" INFO sub-field states the data type is integer whereas the variant
    record contains a float value (2.5) assigned to the sub-field.</p>
<ol>
<li><strong>Exception</strong>: The rule does not apply if <em>Type</em> of a field is not
    defined or is incorrectly defined (e.g., field not declared in HEADER,
    <em>Type</em> not included in declaration, incorrect value for <em>Type)</em>. It also
    does not apply to any missing values (denoted with ".") in the record as
    they do not have an associated data type.</li>
</ol>
</li>
<li>
<p>Multiple comma-separated values (corresponding to value assigned to <em>Number</em>
    key in declaration) can be specified for an INFO or FORMAT sub-field for a
    variant record. No other character can be used as separator. Line20 shows a
    violation as a "/" is used as separator between 2nd and 3rd values for
    <em>"PL"</em> FORMAT sub-field in the second sample column.</p>
</li>
<li>
<p>If <em>Number</em> tag is assigned a known bounded value (an integer, "A", "G") for
    an INFO/FORMAT sub-field, it should be consistent with number of values
    specified for any variant record in BODY of file. For example, Line20 shows
    a violation as <em>"PL"</em> is associated with 3 integer values (Line10) but the
    variant record has only 2 comma-separated integer values (42,3) for
    TCGA-02-0001-01.</p>
</li>
<li>
<p>Validation of <strong>FILTER</strong> sub-fields:</p>
<ol>
<li>
<p>Valid values for FILTER column are "PASS" or a code for the filter that
    the variant call fails (e.g., "q10" in Line16). The code must correspond
    to the "ID" value of the corresponding FILTER declaration.</p>
</li>
<li>
<p>If a call fails multiple filters, FILTER column should contain
    semicolon-separated list of all failed filter codes (e.g., "q10;s50" in
    Line17). A ";" is the only valid separator.</p>
</li>
<li>
<p>All codes listed in the FILTER column must have a well-formed
    declaration in the HEADER. Line18 shows a violation as "q10" does not
    have an associated definition in the HEADER.</p>
</li>
</ol>
</li>
<li>
<p>\&lt;TCGA-VCF&gt; Validation of
    <a href="#TCGAVariantCallFormat(VCF)1.1Specificat">SAMPLE</a> meta-information lines:</p>
<ol>
<li>
<p>Each sample ID in the column header (immediately after FORMAT column)
    must have an associated HEADER declaration where value assigned to "ID"
    tag in the declaration is the same as sample ID used in the column name.</p>
</li>
<li>
<p>Declaration must contain all required fields.</p>
</li>
<li>
<p>Genome mixture tags (Genomes, Mixture, Genome_Description) are enclosed
    within angle brackets (&lt;&gt;) and can have multiple comma-separated
    values.</p>
</li>
<li>
<p>If more than one of the genome mixture tags (Genomes, Mixture,
    Genome_Description) are defined in a SAMPLE meta-information line, then
    number of comma-separated values should be the same for all defined
    tags. For example, "Genomes=<g1,g2>,Mixture=&lt;0.1,0.8,0.1&gt;" would
    lead to a violation as Mixture has 3 values while Genomes has only 2
    values.</g1,g2></p>
</li>
<li>
<p>Individual values in "Genomes" are strings without white-space, comma or
    angle brackets.</p>
</li>
<li>
<p>Individual values in "Mixture" represent proportion (floating point
    number &gt;= 0 and &lt;= 1) of each genome in the sample and all
    comma-separated values should add up to a sum of 1.</p>
</li>
<li>
<p>Individual values in "Genome_Description" are strings surrounded by
    double quotes where the string itself cannot contain a double quote.</p>
</li>
<li>
<p>The value assigned to "SampleName" must be a valid <a href="https://docs.gdc.cancer.gov/Encyclopedia/pages/TCGA_Barcode/">aliquot
    barcode</a> / <a href="https://docs.gdc.cancer.gov/Encyclopedia/pages/UUID/">UUID</a>
    in the database (TCGA VCF 1.1).</p>
</li>
</ol>
</li>
<li>
<p><tcga-vcf> Validation of
    <a href="#TCGAVariantCallFormat(VCF)1.1Specificat">PEDIGREE</a> meta-information lines:</tcga-vcf></p>
<ol>
<li>
<p>Declaration line should follow the format:</p>
</li>
<li>
<h2 id="None">PEDIGREE=<name_0=g0-id,name_1=g1-id,...,name_n=gn-id></name_0=g0-id,name_1=g1-id,...,name_n=gn-id></h2>
</li>
</ol>
</li>
</ol>
<blockquote>
<p>where:</p>
</blockquote>
<ul>
<li>
<p>N &gt;= 1</p>
</li>
<li>
<p>Name_0 through Name_N are arbitrary (not literal) strings that cannot
    contain white-space, comma, or angle brackets (TCGA VCF 1.1)</p>
</li>
<li>
<p>G0-ID through GN-ID are strings that cannot contain white-space, comma, or
    angle brackets. Each of these should be a header for the genotype columns
    immediately after FORMAT column and should be defined using "ID" tag in the
    corresponding ##SAMPLE meta-information line. (TCGA VCF 1.1)</p>
</li>
<li>
<p>The keys and values used in the <name_n=value_n> should be unique across
    assignments in any given PEDIGREE declaration.</name_n=value_n></p>
</li>
<li>
<p>Validation of custom meta-information fields:</p>
<ol>
<li>
<p>If a user-created custom meta-information declaration is encountered and
    the corresponding key/value structure and content have not been defined
    in this specification, the line should be validated to ensure it follows
    one of the following two formats:</p>
</li>
<li>
<h2 id="key=value">key=value</h2>
</li>
<li>
<p>Example:</p>
</li>
<li>
<h2 id="None"><individual=tcga-24-0980></individual=tcga-24-0980></h2>
</li>
<li>
<p>OR</p>
</li>
<li>
<h2 id="None">FIELDTYPE=<key1=value1,key2=value2,...></key1=value1,key2=value2,...></h2>
</li>
<li>
<p>Example:</p>
</li>
<li>
<h2 id='contig=&lt;ID=20,length=62435964,assembly=B36,md5=f126cdf8a6e0c7f379d618ff66beb2da,species="Homo'>contig=&lt;ID=20,length=62435964,assembly=B36,md5=f126cdf8a6e0c7f379d618ff66beb2da,species="Homo</h2>
<pre><code>sapiens",taxonomy=x&gt;
</code></pre>
</li>
</ol>
</li>
</ul>
<blockquote>
<p>where:</p>
</blockquote>
<ul>
<li>
<p>key !\~ /(\s|,|=|;)/</p>
</li>
<li>
<p>value !\~ /(\s|,|=|;)/ UNLESS <em>value</em> is within double quotes, in which
    case it cannot itself contain a double quote or leading/trailing whitespace
    OR if <em>value</em> is within angle brackets.</p>
</li>
<li>
<p><em>CHROM</em>, <em>POS</em>, and <em>REF</em> are required fields and cannot contain missing
    value identifiers. Please refer to <a href="#TCGAVariantCallFormat(VCF)1.1Specificat">Table
    6</a> for acceptable values.</p>
<ol>
<li>
<p><tcga-vcf> <em>CHROM</em> is in {[1-22], X, Y, MT,<chr_id>} where chr_ID
    cannot contain whitespace or &lt;&gt;</chr_id></tcga-vcf></p>
</li>
<li>
<p>If CHROM == <chr_id> then the VCF file MUST have a declaration for
    assembly file in the HEADER. Please note that values assigned to the
    field are currently not being validated.</chr_id></p>
</li>
<li>
<h2 id="assembly=url or filename">assembly=url or filename</h2>
</li>
<li>
<p>Example:</p>
</li>
<li>
<h2 id="assembly=ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/sv/breakpoint_assemblies.fasta">assembly=ftp://ftp-trace.ncbi.nih.gov/1000genomes/ftp/release/sv/breakpoint_assemblies.fasta</h2>
</li>
<li>
<h2 id="assembly=breakpoint_assemblies.fasta">assembly=breakpoint_assemblies.fasta</h2>
</li>
<li>
<p><em>POS</em> is a non-negative integer</p>
</li>
<li>
<p><em>REF</em> =\~ /[ACGTN]+/</p>
</li>
</ol>
</li>
<li>
<p><tcga-vcf> <em>ALT</em> is in {[ACGTN]+, ".", <id>, <strong>SV_ALT</strong>};</id></tcga-vcf></p>
<ol>
<li>
<p>String SV_ALT can be in one of the following four formats and can be
    used in the <em>ALT</em>field ONLY when the corresponding INFO field has the
    key-value pair "SVTYPE=BND" or "SVTYPE=FND".</p>
</li>
<li>
<p>Format Example</p>
</li>
<li>
<p>seq[chr:pos[ G[17:198982[</p>
</li>
<li>
<p>seq]chr:pos] GC]1:238909]</p>
</li>
<li>
<p>]chr:pos]seq ]\&lt;ctg1&gt;:235788]GCNA</p>
</li>
<li>
<p>[chr:pos[seq [1:2812734[ACT</p>
</li>
</ol>
</li>
</ul>
<blockquote>
<p>where:</p>
</blockquote>
<ul>
<li>
<p><em>seq</em> is in {[ACGTN]+, "."}</p>
</li>
<li>
<p><em>chr</em> is in {\&lt;chr_ID&gt;, [1-22], X, Y, MT} where <em>chr_ID</em> is a string</p>
</li>
<li>
<p><em>pos</em> is a non-negative integer</p>
</li>
<li>
<p>Similar to 18b, if chr == <chr_id> (where <em>chr_ID</em> is a string) then the
    VCF file must have an ##assembly declaration in the HEADER.</chr_id></p>
</li>
<li>
<p>If <em>ALT</em> is assigned a value in <id> format, (e.g., rs123 in Line19),
    <id> should be defined in the HEADER as
    ##ALT=<id=*id*,description="*description*"> (Line14) where ID cannot
    contain white-space or angle brackets. Line20 shows a violation of this rule
    as <em>ALT==</em><dup> but there is no corresponding <em>ALT</em> declaration in the
    HEADER with <id=dup>.</id=dup></dup></id=*id*,description="*description*"></id></id></p>
</li>
<li>
<p>ALT can contain multiple comma-separated values. No other character can be
    used as a separator.</p>
</li>
<li>
<p>No two records are allowed to have the the same <em>ID</em> value. Two records can,
    however, have the same <em>CHROM</em> and <em>POS</em>values.</p>
<ol>
<li><strong>Exception</strong>: Multiple records in a file are allowed to have the same
    missing value identifier (".") as <em>ID</em>.</li>
</ol>
</li>
<li>
<p><em>QUAL</em> field can only contain non-negative integers or "." (missing value).</p>
</li>
<li>
<p><tcga-vcf> If INFO sub-field "VT" is declared and used in the BODY, its
    value can only be in {SNP, INS, DEL}</tcga-vcf></p>
</li>
<li>
<p><tcga-vcf> If FORMAT sub-field "SS" is declared and used in the BODY, its
    value can be 0, 1, 2, 3, 4 or 5 depending on whether relative to normal the
    variant is wildtype, germline, somatic, LOH, post-transcriptional
    modification, or unknown respectively.</tcga-vcf></p>
</li>
<li>
<p><tcga-vcf> "DP" sub-field for read depth can be defined in INFO (combined
    depth across all samples) or FORMAT (depth in a specific sample) field. If
    both INFO and FORMAT have values for the sub-field, then sum of DP values
    across all FORMAT sample columns should be equal to DP value in the INFO
    field.</tcga-vcf></p>
</li>
<li>
<p><tcga-vcf> Validation of <strong>complex rearrangement</strong> records:</tcga-vcf></p>
<ol>
<li>If INFO field includes key-value pairs "SVTYPE=BND" or "SVTYPE=FND" and
    has values for "MATEID" and/or "PARID", then the value (or multiple
    comma-separated values) assigned to MATEID or PARID should exist in the
    file as "ID" field for another variant record.</li>
</ol>
</li>
<li>
<p><tcga-vcf> Validation of RNA-Seq <strong>annotation fields</strong>:</tcga-vcf></p>
<ol>
<li>
<p>If INFO field includes "SID", "GENE" or "RGN" keys with associated
    values, then file MUST contain a declaration for ##geneAnno in the
    HEADER.</p>
</li>
<li>
<p>Number of comma-separated values in the optional INFO sub-fields "SID",
    "GENE" and "RGN" and the FORMAT sub-field "TE" must be the same if more
    than one of these sub-fields are defined for a record.</p>
</li>
<li>
<p>INFO sub-field "RGN" is in {5_utr, 3_utr, exon, intron, ncds, sp}.</p>
</li>
<li>
<p>FORMAT sub-field "TE" is in {SIL, MIS, NSNS, NSTP, FSH, NA}</p>
</li>
<li>
<p>If "RGN" and "TE" have the same number of comma-separated values, then
    "RGN" must be "exon" for "TE" to have any value other than "NA". For
    example, if "RGN=exon,intron,intron" then having "MIS,SIL,NA" for TE
    would lead to a violation as the 2nd value for RGN is "intron" but the
    corresponding TE value is "SIL" instead of "NA".</p>
</li>
</ol>
</li>
<li>
<p><tcga-vcf> Validation of
    <a href="#TCGAVariantCallFormat(VCF)1.1Specificat">vcfProcessLog</a> tags:</tcga-vcf></p>
</li>
</ul>
<h2 id="None">vcfProcessLog=&lt;InputVCF=<file1.vcf>,InputVCFSource=<varcaller1>,InputVCFVer=&lt;1.0&gt;,InputVCFParam=<a1,c2>,InputVCFgeneAnno=<anno1.gaf>&gt;</anno1.gaf></a1,c2></varcaller1></file1.vcf></h2>
<p>OR</p>
<h2 id="None">vcfProcessLog=&lt;InputVCF=<file1.vcf,file2.vcf,file3.vcf>,InputVCFSource=<varcaller1,varcaller2,varcaller3>,InputVCFVer=&lt;1.0,2.1,2.0&gt;,</varcaller1,varcaller2,varcaller3></file1.vcf,file2.vcf,file3.vcf></h2>
<p>InputVCFParam=<a1,c2;a1,b1;a1,b1>,InputVCFgeneAnno=<anno1.gaf,anno2.gaf,anno3.gaf>,</anno1.gaf,anno2.gaf,anno3.gaf></a1,c2;a1,b1;a1,b1></p>
<p>MergeSoftware=<sw1,sw2>,MergeParam=<a1,a2;b1,b2>,MergeVer=&lt;2.1,3.0&gt;,MergeContact=<a href="mailto:johndoe@xyz.edu">johndoe@xyz.edu</a>&gt;</a1,a2;b1,b2></sw1,sw2></p>
<ol>
<li>
<p>Individual values for each tag are enclosed within angle brackets (&lt;&gt;)
    instead of double quotes.</p>
<ol>
<li>
<p>If a field contains multiple values, they are separated by comma.
    <strong>Exception</strong>: Separator for multiple values in <em>InputVCFParam</em> and
    <em>MergeParam</em> is a ";" instead of ",". Individual values within these
    tags can contain comma-separated parameters (e.g., &lt;a1,c2;a1,b1;a1,b1
    in the example given above).</p>
</li>
<li>
<p>If <em>InputVCF</em> tag has multiple comma-separated values assigned to it
    (please refer to the second example above), then <em>InputVCFSource,
    InputVCFVer, InputVCFParam,</em> and <em>InputVCFgeneAnno</em> must contain the
    same number of values. If a value is not known, it should be substituted
    with the missing value identifier (".").</p>
</li>
<li>
<p>If <em>InputVCF</em> contains only a single value, then all tags =\~ /Merge.*/
    are optional and can either be omitted or can contain missing value
    identifier ("."). The reason is that attribute related to merging VCF
    files are applicable only if multiple input VCF files are being merged.</p>
</li>
<li>
<p>If <em>MergeSoftware</em> contains multiple comma-separated values,
    <em>MergeParam</em> and <em>MergeVer</em> should contain the same number of values.
    There is no such constraint for <em>MergeContact.</em></p>
</li>
</ol>
</li>
</ol>
<p>UUID-compliant files should satisfy the following criteria:</p>
<ol>
<li>
<p>The value assigned to "SampleUUID" must be a valid aliquot
<a href="https://docs.gdc.cancer.gov/Encyclopedia/pages/UUID/">UUID</a>.</p>
</li>
<li>
<p>The <a href="https://docs.gdc.cancer.gov/Encyclopedia/pages/TCGA_Barcode/">metadata</a> represented by
    "SampleTCGABarcode" must correspond to SampleUUID at the DCC.</p>
</li>
<li>
<p>If ##INDIVIDUAL is declared in the header, then values assigned to
SampleUUID in all ##SAMPLE declarations should correspond to the same
participant ID, and the TCGA barcode for the participant should be assigned to</p>
</li>
</ol>
<h2 id="INDIVIDUAL.">INDIVIDUAL.</h2>
<h2 id="Handling failed checks">Handling failed checks</h2>
<ul>
<li>
<p>A VCF file would be required to pass ALL the checks listed above and any
    violation will lead to a "Failed" validation.</p>
</li>
<li>
<p>Even if a failure is encountered, the file would still need to go through
    all other checks for validation to be complete. Exception to this
    requirement would include cases where execution of one validation check is
    dependent on the success of another prerequisite step. For example, number
    of values associated with a FORMAT field for a variant record cannot be
    validated if the field itself is not declared in the HEADER or has a missing
    <em>Number</em> tag.</p>
</li>
<li>
<p>A summary of all failed checks should be provided as an output.</p>
</li>
</ul>
<h2 id="Test files">Test files</h2>
<p>The following table lists sample files that can be used to test various
validation steps. Parent directory is
<a href="https://tcga-data.nci.nih.gov/tcgafiles/tcgajamboree/Benchmark_VCF">https://tcga-data.nci.nih.gov/tcgafiles/tcgajamboree/Benchmark_VCF</a>.</p>
<p><strong>Table 9: Test files known to pass/fail validation steps</strong></p>
<table class="table">
<thead>
<tr>
<th><strong>Expected result</strong></th>
<th><strong>Validation</strong></th>
<th><strong>Test file</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>Success</td>
<td>file with no failures</td>
<td>/vcfFormat/TCGA-24-0980_IlluminaGA-DNASeq_exome.format2.vcf</td>
</tr>
<tr>
<td>Failure</td>
<td>"chr" prefix for chromosome names</td>
<td>/BI/20110324/BCM-GBM_solid.TCGA-06-0208.mut.vcf</td>
</tr>
<tr>
<td>Failure</td>
<td>column header line has no "#" as beginning</td>
<td>/genome.wustl.edu/20110420/TCGA-06-0145-01A-01W-0224-08.vcf.gz</td>
</tr>
<tr>
<td>Failure</td>
<td>double quotes missing from SAMPLE metadata lines Description</td>
<td>/BCM/TCGA-06-0145_IlluminaGA-DNASeq_exome.vcf</td>
</tr>
<tr>
<td>Failure</td>
<td>scores in QUAL column are negative integers</td>
<td>/UCSC/20110420/TCGA-13-0723_W_capture.vcf</td>
</tr>
</tbody>
</table>
<ul>
<li>
<p>missing values for FORMAT fields</p>
</li>
<li>
<p>trailing whitespace at the end of description strings in metadata</p>
</li>
<li>
<p>filter not listed in FILTER metadata line</p>
</li>
<li>
<p>in multi-allelic sites, different alternative alleles in the ALT field are
    separated by "/" instead of ","</p>
</li>
</ul></div>
      </div>
    </div>

    <footer id="docs-footer">
      <div class="container-fluid footer-container">
        <div>
          <a href="https://portal.gdc.cancer.gov" target="_blank">Site Home</a>
          |
          <a href="http://www.cancer.gov/global/web/policies" target="_blank"
            >Policies</a
          >
          |
          <a
            href="http://www.cancer.gov/global/web/policies/accessibility"
            target="_blank"
            >Accessibility</a
          >
          |
          <a
            href="http://www.cancer.gov/global/web/policies/foia"
            target="_blank"
            >FOIA</a
          >
        </div>
        <div>
          <a href="http://www.hhs.gov" target="_blank"
            >U.S. Department of Health and Human Services</a
          >
          |
          <a href="http://www.nih.gov" target="_blank"
            >National Institutes of Health</a
          >
          |
          <a href="http://www.cancer.gov" target="_blank"
            >National Cancer Institute</a
          >
          | <a href="http://www.usa.gov" target="_blank">USA.gov</a>
        </div>
        <div>NIH... Turning Discovery Into Health </div>

        <div class="version-caption">GDC Docs Version 2.0</div>
      </div>
    </footer>

    <script>
      /* When the user clicks on the button, 
toggle between hiding and showing the dropdown content */

      // Close the dropdown if the user clicks outside of it

      var toggler = document.getElementById("myToggle");
      toggler.onclick = function () {
        var myDropdown = document.getElementById("myDropdown");
        myDropdown.classList.toggle("show");
      };
      window.onclick = function (e) {
        if (!e.target.matches(".dropbtn")) {
          var myDropdown = document.getElementById("myDropdown");
          if (myDropdown.classList.contains("show")) {
            myDropdown.classList.remove("show");
          }
        }
      };
    </script>

    <!-- JavaScript Bundle with Popper -->
    <script
      src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
      integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
      crossorigin="anonymous"
    ></script>
  </body>
</html>