<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link href="vendor/fontawesome/css/all.min.css" rel="stylesheet">
    <!-- CSS only -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=EB+Garamond:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&family=Source+Sans+Pro:ital,wght@0,300;0,400;0,600;0,700;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet">
    
      <link rel="stylesheet" href=/home/jacob/Work/Docsite-Revamp/css/styles.css>
    
      <link rel="stylesheet" href=/home/jacob/Work/Docsite-Revamp/css/docs.css>
    
    <!--<link rel="stylesheet" href="../../css/styles.css">
    <link rel="stylesheet" href="../../css/encyclopedia.css">-->
</head>
<body>
    <!---->
    <nav class="navbar navbar-expand-md navbar-light bg-light">
        <div class="container-fluid">
                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
                    <span class="navbar-toggler-icon"></span>
                </button>
                <!-- Collapsed navigation -->
          <div class="my-container">
            <!-- Main title -->
            <a class="navbar-brand float-start" href="/.">
                <img class="brand-image" src="../../img/nih-header-logo-color.png" alt="NIH National Cancer Institute GDC Documentation">
                <span class="hidden">NIH National Cancer Institute GDC Documentation</span>
            </a>
            <!-- Expander button -->
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>

            <!-- GDC Apps -->
            <div class="pull-right">
              <div class="gdc-app-list dropbtn">
                <div class="dropbtn">
                  <a id='myToggle' class="dropbtn dropdown-toggle">
                    <span class="dropbtn"><i class="fas fa-th"></i> GDC Apps</span>
                  </a>
                  <div id="myDropdown" class="dropdown-content container">
                    <div class="row gdc-app-row">
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                    </div>
                    <div class="row gdc-app-row">
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                    </div>
                    <div class="row gdc-app-row">
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                    </div>
                  </div>
                </div>
                <div data-dropdown-menu="" class="dropdown-menu gdc-apps-menu-container" role="menu">
                  <div class="gdc-apps-menu-grid">
                    <div class="menu-column">
                      <a href="https://portal.gdc.cancer.gov/" title="Data Portal" target="_blank" class="menu-item">
                        <img width="21px" src="/img/GDC-App-data-portal-blue.svg" class="icon icon-gdc-portal home" alt="GDC Data Portal">
                        <div>Data Portal</div>
                      </a>
                      <a href="https://gdc.cancer.gov/developers/gdc-application-programming-interface-api" title="GDC API" target="_blank" class="menu-item">
                        <span class="icon icon-gdc-portal-api">
                          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span>
                          </span>
                        <div>API</div>
                      </a>
                      <a href="https://docs.gdc.cancer.gov/" title="GDC Docs" target="_blank" class="active menu-item">
                        <span class="icon icon-gdc-docs">
                          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span><span class="path11"></span><span class="path12"></span><span class="path13"></span><span class="path14"></span>
                        </span>
                        <div>Documentation</div>
                      </a>
                      <a href="https://portal.gdc.cancer.gov/legacy-archive" title="GDC Legacy Archive" target="_blank" class="menu-item">
                        <span class="icon icon-gdc-legacy-archive">
                          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span>
                        </span>
                        <div>Legacy Archive</div>
                      </a>
                    </div>
                    <div class="menu-column">
                      <a href="https://gdc.cancer.gov/" title="GDC Website" target="_blank" class="menu-item">
                        <img width="21px" src="/img/GDC-App-website-blue.svg" class="icon icon-gdc-portal home" alt="GDC Website">
                        <div>Website</div>
                      </a>
                      <a href="https://gdc.cancer.gov/access-data/gdc-data-transfer-tool" class="menu-item" title="GDC Data Transfer Tool" target="_blank">
                        <span class="icon icon-gdc-data-transer-tool">
                          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span>
                        </span>
                        <div>Data Transfer Tool</div>
                      </a>
                      <a href="https://portal.gdc.cancer.gov/submission/" title="GDC Submission Portal" target="_blank" class="menu-item">
                        <span class="icon icon-gdc-submission-portal">
                          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span>
                        </span>
                        <div>Data Submission Portal</div>
                      </a>
                    </div>
                  </div>
                </div>
              </div>
            </div>
              <div class="container-fluid">
              <div class="collapse navbar-collapse" id="navbarNavDropdown">
                <ul class="navbar-nav navbar-center me-auto">
                <li class="nav-item">
                    <a class="nav-link" href="#">Home</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link active" aria-current="page" href="#">API</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#">Data Submission</a>
                </li>
                <li class="nav-item dropdown">
                    <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-bs-toggle="dropdown" aria-expanded="false">
                    Data Transfer Tool
                    </a>
                    <ul class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                    <li><a class="dropdown-item" href="#">Action</a></li>
                    <li><a class="dropdown-item" href="#">Another action</a></li>
                    <li><a class="dropdown-item" href="#">Something else here</a></li>
                    </ul>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="#">Data Dictionary</a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="#">Data</a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="#">Encyclopedia</a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="#"><i class="fas fa-search"></i></a>
                </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </nav>

      <div class="row">
      <div class="col-md-3">
        <div class="container-fluid sidebar-container">
          <div class="sidebar">
            <ul class="main">
              <li class="main"><a href="#">Getting Started</a></li>
              <ul>
                <li><a href='#Endpoints'>Endpoints</a></li><li><a href='#Genes Endpoint Examples'>Genes Endpoint Examples</a></li><li><a href='#Simple Somatic Mutation Endpoint Examples'>Simple Somatic Mutation Endpoint Examples</a></li><li><a href='#Simple Somatic Mutation Occurrence Endpoint Examples'>Simple Somatic Mutation Occurrence Endpoint Examples</a></li><li><a href='#Copy Number Variation Endpoint Examples'>Copy Number Variation Endpoint Examples</a></li><li><a href='#Copy Number Variation Occurrence Enpoint Examples'>Copy Number Variation Occurrence Enpoint Examples</a></li><li><a href='#Analysis Endpoints'>Analysis Endpoints</a></li>
              </ul>
              <li class="inactive"><a href="#">Search and Retrieval</a></li>
              <li class="inactive"><a href="#">Data Analysis</a></li>
              <li class="inactive"><a href="#">BAM Slicing</a></li>
              <li class="inactive"><a href="#">Submission</a></li>
              <li class="inactive"><a href="#">Python Examples</a></li>
              <li class="inactive"><a href="#">GraphQL Examples</a></li>
              <li class="inactive"><a href="#">System Information</a></li>
              <li class="inactive"><a href="#">Additional Examples</a></li>
              <li class="inactive"><a href="#">Appendix A: Available Fields</a></li>
              <li class="inactive"><a href="#">Appendix B: Key Terms</a></li>
              <li class="inactive"><a href="#">Appendix C: Format of Submission Queries and Responses</a></li>
              <li class="inactive"><a href="#">Release Notes</a></li>
            </ul>
          </div>
        </div>
      </div>
      <div class="col-md-9">
        <div class="container-fluid doc-container">
          <h1><i class="fas fa-book"></i>   Data Analysis</h1><hr>
<p>The GDC DAVE tools use the same API as the rest of the Data Portal and takes advantage of several new endpoints. Similar to the <a href="http://docs.gdc.cancer.gov/Data_Portal/Users_Guide/Exploration/">GDC Data Portal Exploration</a> feature, the GDC data analysis endpoints allow API users to programmatically explore data in the GDC using advanced filters at a gene and mutation level. Survival analysis data is also available.  </p>
<h2>Endpoints</h2>
<p>The following data analysis endpoints are available from the GDC API:</p>
<table class='table'>
<thead>
<tr>
<th><strong>Node</strong></th>
<th><strong>Endpoint</strong></th>
<th><strong>Description</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Genes</strong></td>
<td><strong>/genes</strong></td>
<td>Allows users to access summary information about each gene using its Ensembl ID.</td>
</tr>
<tr>
<td><strong>SSMS</strong></td>
<td><strong>/ssms</strong></td>
<td>Allows users to access information about each somatic mutation. For example, a <code>ssm</code> would represent the transition of C to T at position 52000 of chromosome 1.</td>
</tr>
<tr>
<td></td>
<td><strong>/ssms/<code>&lt;ssm_id&gt;</code></strong></td>
<td>Get information about a specific ssm using a <code>&lt;ssm_id&gt;</code>, often supplemented with the <code>expand</code> option to show fields of interest.</td>
</tr>
<tr>
<td></td>
<td><strong>/ssm_occurrences</strong></td>
<td>A <code>ssm</code> entity as applied to a single instance (case). An example of a <code>ssm occurrence</code> would be that the transition of C to T at position 52000 of chromosome 1 occurred in patient TCGA-XX-XXXX.</td>
</tr>
<tr>
<td></td>
<td><strong>/ssm_occurrences/<code>&lt;ssm_occurrences_id&gt;</code></strong></td>
<td>Get information about a specific ssm occurrence using a <code>&lt;ssm_occurrences_id&gt;</code>, often supplemented with the <code>expand</code> option to show fields of interest.</td>
</tr>
<tr>
<td><strong>CNVS</strong></td>
<td><strong>/cnvs</strong></td>
<td>Allows users to access data about copy number variations (cnvs). This data will be specifc to cnvs and not a specific case.</td>
</tr>
<tr>
<td></td>
<td><strong>/cnvs/<code>&lt;cnv_id&gt;</code></strong></td>
<td>Get information about a specific copy number variation using a <code>&lt;cnv_id&gt;</code>, often supplemented with the <code>expand</code> option to show fields of interest.</td>
</tr>
<tr>
<td></td>
<td><strong>/cnvs/ids</strong></td>
<td>This endpoint will retrieve nodes that contain the queried cnv_id. This is accomplished by adding the query parameter: /cnvs/ids?query=<code>&lt;cnv_id&gt;</code>.</td>
</tr>
<tr>
<td></td>
<td><strong>/cnv_occurrences</strong></td>
<td>A <code>cnv</code> entity as applied to a single case.</td>
</tr>
<tr>
<td></td>
<td><strong>/cnv_occurrences/<code>&lt;cnv_occurrence_id&gt;</code></strong></td>
<td>Get information about a specific copy number variation occurrence using a <code>&lt;cnv_occurrence_id&gt;</code>, often supplemented with the <code>expand</code> option to show fields of interest.</td>
</tr>
<tr>
<td></td>
<td><strong>/cnv_occurrences/ids</strong></td>
<td>This endpoint will retrieve nodes that contain the queried cnv_occurrence_id. This is accomplished by adding the query parameter: /cnv_occurrences/ids?query=<code>&lt;cnv_occurrences_id&gt;</code></td>
</tr>
<tr>
<td><strong>Analysis</strong></td>
<td><strong>/analysis/top_cases_counts_by_genes</strong></td>
<td>Returns the number of cases with a mutation in each gene listed in the gene_ids parameter for each project. Note that this endpoint cannot be used with the <code>format</code> or <code>fields</code> parameters.</td>
</tr>
<tr>
<td></td>
<td><strong>/analysis/top_mutated_genes_by_project</strong></td>
<td>Returns a list of genes that have the most mutations within a given project.</td>
</tr>
<tr>
<td></td>
<td><strong>/analysis/top_mutated_cases_by_gene</strong></td>
<td>Generates information about the cases that are most affected by mutations in a given number of genes</td>
</tr>
<tr>
<td></td>
<td><strong>/analysis/mutated_cases_count_by_project</strong></td>
<td>Returns counts for the number of cases that have associated <code>ssm</code> data in each project. The number of affected cases can be found under "case_with_ssm": {"doc_count": $case_count}.</td>
</tr>
<tr>
<td></td>
<td><strong>/analysis/survival</strong></td>
<td>Survival plots can be generated in the Data Portal for different subsets of data, based upon many query factors such as variants, disease type and projects. This endpoint can be used to programmatically retrieve the raw data to generate these plots and apply different filters to the data. (see Survival Example)</td>
</tr>
</tbody>
</table>
<p>The methods for retrieving information from these endpoints are very similar to those used for the <code>cases</code> and <code>files</code> endpoints. These methods are explored in depth in the <a href="https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/">API Search and Retrieval</a> documentation. The <code>_mapping</code> parameter can also be used with each of these endpoints to generate a list of potential fields.  For example:</p>
<p><code>https://api.gdc.cancer.gov/ssms/_mapping</code></p>
<p>While it is not an endpoint, the <code>observation</code> entity is featured in the visualization section of the API. The <code>observation</code> entity provides information from the MAF file, such as read depth and normal genotype, that supports the validity of the associated <code>ssm</code>. An example is demonstrated below:</p>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/ssms/57bb3f2e-ec05-52c2-ab02-7065b7d24849?expand=occurrence.case.observation.read_depth&amp;pretty=true&quot;
</code></pre>
<pre><code class="language-Response">{
  &quot;data&quot;: {
    &quot;ncbi_build&quot;: &quot;GRCh38&quot;,
    &quot;occurrence&quot;: [
      {
        &quot;case&quot;: {
          &quot;observation&quot;: [
            {
              &quot;read_depth&quot;: {
                &quot;t_ref_count&quot;: 321,
                &quot;t_alt_count&quot;: 14,
                &quot;t_depth&quot;: 335,
                &quot;n_depth&quot;: 115
              }
            }
          ]
        }
      }
    ],
    &quot;tumor_allele&quot;: &quot;G&quot;,
    &quot;mutation_type&quot;: &quot;Simple Somatic Mutation&quot;,
    &quot;end_position&quot;: 14304578,
    &quot;reference_allele&quot;: &quot;C&quot;,
    &quot;ssm_id&quot;: &quot;57bb3f2e-ec05-52c2-ab02-7065b7d24849&quot;,
    &quot;start_position&quot;: 14304578,
    &quot;mutation_subtype&quot;: &quot;Single base substitution&quot;,
    &quot;cosmic_id&quot;: null,
    &quot;genomic_dna_change&quot;: &quot;chr5:g.14304578C&gt;G&quot;,
    &quot;gene_aa_change&quot;: [
      &quot;TRIO L229V&quot;,
      &quot;TRIO L437V&quot;,
      &quot;TRIO L447V&quot;,
      &quot;TRIO L496V&quot;
    ],
    &quot;chromosome&quot;: &quot;chr5&quot;
  },
  &quot;warnings&quot;: {}
}
</code></pre>
<h2>Genes Endpoint Examples</h2>
<p><strong>Example 1:</strong> A user would like to access information about the gene <code>ZMPSTE24</code>, which has an Ensembl gene ID of <code>ENSG00000084073</code>.  This would be accomplished by appending <code>ENSG00000084073</code> (<code>gene_id</code>) to the <code>genes</code> endpoint.</p>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/genes/ENSG00000084073?pretty=true&quot;
</code></pre>
<pre><code class="language-Response">{
  &quot;data&quot;: {
    &quot;canonical_transcript_length&quot;: 3108,
    &quot;description&quot;: &quot;This gene encodes a member of the peptidase M48A family. The encoded protein is a zinc metalloproteinase involved in the two step post-translational proteolytic cleavage of carboxy terminal residues of farnesylated prelamin A to form mature lamin A. Mutations in this gene have been associated with mandibuloacral dysplasia and restrictive dermopathy. [provided by RefSeq, Jul 2008]&quot;,
    &quot;cytoband&quot;: [
      &quot;1p34.2&quot;
    ],
    &quot;gene_start&quot;: 40258107,
    &quot;canonical_transcript_length_genomic&quot;: 36078,
    &quot;gene_id&quot;: &quot;ENSG00000084073&quot;,
    &quot;gene_strand&quot;: 1,
    &quot;canonical_transcript_length_cds&quot;: 1425,
    &quot;gene_chromosome&quot;: &quot;1&quot;,
    &quot;synonyms&quot;: [
      &quot;FACE-1&quot;,
      &quot;HGPS&quot;,
      &quot;PRO1&quot;,
      &quot;STE24&quot;,
      &quot;Ste24p&quot;
    ],
    &quot;is_cancer_gene_census&quot;: null,
    &quot;biotype&quot;: &quot;protein_coding&quot;,
    &quot;gene_end&quot;: 40294184,
    &quot;canonical_transcript_id&quot;: &quot;ENST00000372759&quot;,
    &quot;symbol&quot;: &quot;ZMPSTE24&quot;,
    &quot;name&quot;: &quot;zinc metallopeptidase STE24&quot;
  },
  &quot;warnings&quot;: {}
}
</code></pre>
<p><strong>Example 2:</strong> A user wants a subset of elements such as a list of coordinates for all genes on chromosome 7. The query can be filtered for only results from chromosome 7 using a JSON-formatted query that is URL-encoded.</p>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/genes?pretty=true&amp;fields=gene_id,symbol,gene_start,gene_end&amp;format=tsv&amp;size=2000&amp;filters=%7B%0D%0A%22op%22%3A%22in%22%2C%0D%0A%22content%22%3A%7B%0D%0A%22field%22%3A%22gene_chromosome%22%2C%0D%0A%22value%22%3A%5B%0D%0A%227%22%0D%0A%5D%0D%0A%7D%0D%0A%7D&quot;
</code></pre>
<pre><code class="language-Response">gene_start      gene_end        symbol  id
28995231        29195451        CPVL    ENSG00000106066
33014114        33062797        NT5C3A  ENSG00000122643
143052320       143053347       OR6V1   ENSG00000225781
100400826       100428992       ZCWPW1  ENSG00000078487
73861159        73865893        WBSCR28 ENSG00000175877
64862999        64864370        EEF1DP4 ENSG00000213640
159231435       159233377       PIP5K1P2        ENSG00000229435
141972631       141973773       TAS2R38 ENSG00000257138
16646131        16706523        BZW2    ENSG00000136261
149239651       149255609       ZNF212  ENSG00000170260
57405025        57405090        MIR3147 ENSG00000266168
130393771       130442433       CEP41   ENSG00000106477
150800403       150805120       TMEM176A        ENSG00000002933
93591573        93911265        GNGT1   ENSG00000127928
117465784       117715971       CFTR    ENSG00000001626
5879827 5886362 OCM     ENSG00000122543
144118461       144119360       OR2A15P ENSG00000239981
30424527        30478784        NOD1    ENSG00000106100
137227341       137343865       PTN     ENSG00000105894
84876554        84876956        HMGN2P11        ENSG00000232605
107470018       107475659       GPR22   ENSG00000172209
31330711        31330896        RP11-463M14.1   ENSG00000271027
78017057        79453574        MAGI2   ENSG00000187391
55736779        55739605        CICP11  ENSG00000237799
142111749       142222324       RP11-1220K2.2   ENSG00000257743
(truncated)
</code></pre>
<h2>Simple Somatic Mutation Endpoint Examples</h2>
<p><strong>Example 1</strong>: Similar to the <code>/genes</code> endpoint, a user would like to retrieve information about the mutation based on its COSMIC ID. This would be accomplished by creating a JSON filter, which will then be encoded to URL for the <code>curl</code> command.</p>
<pre><code class="language-Filter"> {
   &quot;op&quot;:&quot;in&quot;,
   &quot;content&quot;:{
      &quot;field&quot;:&quot;cosmic_id&quot;,
      &quot;value&quot;:[
         &quot;COSM4860838&quot;
      ]
   }
}
</code></pre>
<pre><code class="language-Shell">curl 'https://api.gdc.cancer.gov/ssms?pretty=true&amp;filters=%7B%0A%22op%22%3A%22in%22%2C%0A%22content%22%3A%7B%0A%22field%22%3A%22cosmic_id%22%2C%0A%22value%22%3A%5B%0A%22COSM4860838%22%0A%5D%0A%7D%0A%7D%0A'
</code></pre>
<pre><code class="language-Response">{
  &quot;data&quot;: {
    &quot;hits&quot;: [
      {
        &quot;ncbi_build&quot;: &quot;GRCh38&quot;,
        &quot;mutation_type&quot;: &quot;Simple Somatic Mutation&quot;,
        &quot;mutation_subtype&quot;: &quot;Single base substitution&quot;,
        &quot;end_position&quot;: 62438203,
        &quot;reference_allele&quot;: &quot;C&quot;,
        &quot;ssm_id&quot;: &quot;8b3c1a7a-e4e0-5200-9d46-5767c2982145&quot;,
        &quot;start_position&quot;: 62438203,
        &quot;cosmic_id&quot;: [
          &quot;COSM4860838&quot;,
          &quot;COSM731764&quot;,
          &quot;COSM731765&quot;
        ],
        &quot;id&quot;: &quot;8b3c1a7a-e4e0-5200-9d46-5767c2982145&quot;,
        &quot;tumor_allele&quot;: &quot;T&quot;,
        &quot;gene_aa_change&quot;: [
          &quot;CADPS G1147G&quot;,
          &quot;CADPS G1187G&quot;,
          &quot;CADPS G1217G&quot;,
          &quot;CADPS G1226G&quot;,
          &quot;CADPS G127G&quot;,
          &quot;CADPS G218G&quot;,
          &quot;CADPS G95G&quot;
        ],
        &quot;chromosome&quot;: &quot;chr3&quot;,
        &quot;genomic_dna_change&quot;: &quot;chr3:g.62438203C&gt;T&quot;
      }
    ],
    &quot;pagination&quot;: {
      &quot;count&quot;: 1,
      &quot;sort&quot;: &quot;&quot;,
      &quot;from&quot;: 0,
      &quot;page&quot;: 1,
      &quot;total&quot;: 1,
      &quot;pages&quot;: 1,
      &quot;size&quot;: 10
    }
  },
  &quot;warnings&quot;: {}
}
</code></pre>
<p><strong>Example 2:</strong> Based on the previous example's <code>ssm_id</code> (<code>8b3c1a7a-e4e0-5200-9d46-5767c2982145</code>), a user would like to look at the consequences and the VEP impact due to this ssm.</p>
<pre><code class="language-Shell">curl 'https://api.gdc.cancer.gov/ssms/8b3c1a7a-e4e0-5200-9d46-5767c2982145?pretty=true&amp;expand=consequence.transcript&amp;fields=consequence.transcript.annotation.vep_impact'
</code></pre>
<pre><code class="language-JSON">{
  &quot;data&quot;: {
    &quot;consequence&quot;: [
      {
        &quot;transcript&quot;: {
          &quot;aa_start&quot;: 127, 
          &quot;consequence_type&quot;: &quot;synonymous_variant&quot;, 
          &quot;aa_end&quot;: 127, 
          &quot;transcript_id&quot;: &quot;ENST00000466621&quot;, 
          &quot;is_canonical&quot;: false, 
          &quot;aa_change&quot;: &quot;G127G&quot;, 
          &quot;annotation&quot;: {
            &quot;vep_impact&quot;: &quot;LOW&quot;
          }, 
          &quot;ref_seq_accession&quot;: &quot;&quot;
        }
      }, 
      {
        &quot;transcript&quot;: {
          &quot;aa_start&quot;: 95, 
          &quot;consequence_type&quot;: &quot;synonymous_variant&quot;, 
          &quot;aa_end&quot;: 95, 
          &quot;transcript_id&quot;: &quot;ENST00000613879&quot;, 
          &quot;is_canonical&quot;: false, 
          &quot;aa_change&quot;: &quot;G95G&quot;, 
          &quot;annotation&quot;: {
            &quot;vep_impact&quot;: &quot;LOW&quot;
          }, 
          &quot;ref_seq_accession&quot;: &quot;&quot;
        }
      }, 
      {
        &quot;transcript&quot;: {
          &quot;aa_start&quot;: 218, 
          &quot;consequence_type&quot;: &quot;synonymous_variant&quot;, 
          &quot;aa_end&quot;: 218, 
          &quot;transcript_id&quot;: &quot;ENST00000473635&quot;, 
          &quot;is_canonical&quot;: false, 
          &quot;aa_change&quot;: &quot;G218G&quot;, 
          &quot;annotation&quot;: {
            &quot;vep_impact&quot;: &quot;LOW&quot;
          }, 
          &quot;ref_seq_accession&quot;: &quot;&quot;
        }
      }, 
      {
        &quot;transcript&quot;: {
          &quot;aa_start&quot;: null, 
          &quot;consequence_type&quot;: &quot;non_coding_transcript_exon_variant&quot;, 
          &quot;aa_end&quot;: null, 
          &quot;transcript_id&quot;: &quot;ENST00000474560&quot;, 
          &quot;is_canonical&quot;: false, 
          &quot;aa_change&quot;: null, 
          &quot;annotation&quot;: {
            &quot;vep_impact&quot;: &quot;MODIFIER&quot;
          }, 
          &quot;ref_seq_accession&quot;: &quot;&quot;
        }
      }, 
      {
        &quot;transcript&quot;: {
          &quot;aa_start&quot;: 1226, 
          &quot;consequence_type&quot;: &quot;synonymous_variant&quot;, 
          &quot;aa_end&quot;: 1226, 
          &quot;transcript_id&quot;: &quot;ENST00000383710&quot;, 
          &quot;is_canonical&quot;: true, 
          &quot;aa_change&quot;: &quot;G1226G&quot;, 
          &quot;annotation&quot;: {
            &quot;vep_impact&quot;: &quot;LOW&quot;
          }, 
          &quot;ref_seq_accession&quot;: &quot;NM_003716.3&quot;
        }
      }, 
      {
        &quot;transcript&quot;: {
          &quot;aa_start&quot;: 1187, 
          &quot;consequence_type&quot;: &quot;synonymous_variant&quot;, 
          &quot;aa_end&quot;: 1187, 
          &quot;transcript_id&quot;: &quot;ENST00000283269&quot;, 
          &quot;is_canonical&quot;: false, 
          &quot;aa_change&quot;: &quot;G1187G&quot;, 
          &quot;annotation&quot;: {
            &quot;vep_impact&quot;: &quot;LOW&quot;
          }, 
          &quot;ref_seq_accession&quot;: &quot;NM_183394.2&quot;
        }
      }, 
      {
        &quot;transcript&quot;: {
          &quot;aa_start&quot;: 1147, 
          &quot;consequence_type&quot;: &quot;synonymous_variant&quot;, 
          &quot;aa_end&quot;: 1147, 
          &quot;transcript_id&quot;: &quot;ENST00000357948&quot;, 
          &quot;is_canonical&quot;: false, 
          &quot;aa_change&quot;: &quot;G1147G&quot;, 
          &quot;annotation&quot;: {
            &quot;vep_impact&quot;: &quot;LOW&quot;
          }, 
          &quot;ref_seq_accession&quot;: &quot;NM_183393.2&quot;
        }
      }, 
      {
        &quot;transcript&quot;: {
          &quot;aa_start&quot;: 1217, 
          &quot;consequence_type&quot;: &quot;synonymous_variant&quot;, 
          &quot;aa_end&quot;: 1217, 
          &quot;transcript_id&quot;: &quot;ENST00000612439&quot;, 
          &quot;is_canonical&quot;: false, 
          &quot;aa_change&quot;: &quot;G1217G&quot;, 
          &quot;annotation&quot;: {
            &quot;vep_impact&quot;: &quot;LOW&quot;
          }, 
          &quot;ref_seq_accession&quot;: &quot;&quot;
        }
      }
    ]
  }
</code></pre>
<h2>Simple Somatic Mutation Occurrence Endpoint Examples</h2>
<p><strong>Example 1:</strong> A user wants to determine the chromosome in case <code>TCGA-DU-6407</code> that contains the greatest number of <code>ssms</code>. As this relates to mutations that are observed in a case, the <code>ssm_occurrences</code> endpoint is used.</p>
<pre><code class="language-Filter">{  
   &quot;op&quot;:&quot;in&quot;,
   &quot;content&quot;:{  
      &quot;field&quot;:&quot;case.submitter_id&quot;,
      &quot;value&quot;:[&quot;TCGA-DU-6407&quot;]
   }
}
</code></pre>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/ssm_occurrences?format=tsv&amp;fields=ssm.chromosome&amp;size=5000&amp;filters=%7B%0D%0A%22op%22%3A%22in%22%2C%0D%0A%22content%22%3A%7B%0D%0A%22field%22%3A%22case.submitter_id%22%2C%0D%0A%22value%22%3A%5B%0D%0A%22TCGA-DU-6407%22%0D%0A%5D%0D%0A%7D%0D%0A%7D&quot;
</code></pre>
<pre><code class="language-tsv">ssm.chromosome  id
chr3    552c09d1-69b1-5c04-b543-524a6feae3eb
chr10   391011ff-c1fd-5e2a-a128-652bc660f64c
chr10   1378cbc4-af88-55bb-b2e5-185bb4246d7a
chr10   3a2b3870-a395-5bc3-8c8f-0d40b0f2202c
chr1    4a93d7a5-988d-5055-80da-999dc3b45d80
chrX    22a07c7c-16ba-51df-a9a9-1e41e2a45225
chr12   dbc5eafa-ea26-5f1c-946c-b6974a345b69
chr11   02ae553d-1f27-565d-96c5-2c3cfca7264a
chr2    faee73a9-4804-58ea-a91f-18c3d901774f
chr6    97c5b38b-fc96-57f5-8517-cc702b3aa70a
chr17   0010a89d-9434-5d97-8672-36ee394767d0
chr19   f08dcc53-eadc-5ceb-bf31-f6b38629e4cb
chrX    19ca262d-b354-54a0-b582-c4719e37e91d
chr19   c44a93a1-5c73-5cff-b40e-98ce7e5fe57b
chr3    b67f31b5-0341-518e-8fcc-811cd2e36af1
chr1    94abd5fd-d539-5a4a-8719-9615cf7cec5d
chr17   1476a543-2951-5ec4-b165-67551b47d810
chr2    b4822fc9-f0cc-56fd-9d97-f916234e309d
chr2    3548ecfe-5186-51e7-8f40-37f4654cd260
chr16   105e7811-4601-5ccb-ae93-e7107923599e
chr2    99b3aad4-d368-506d-99d6-047cbe5dff0f
chr13   9dc3f7cd-9efa-530a-8524-30d067e49d54
chr21   1267330b-ae6d-5e25-b19e-34e98523679e
chr16   c77f7ce5-fbe6-5da4-9a7b-b528f8e530cb
chr10   2cb06277-993e-5502-b2c5-263037c45d18
chr17   d25129ad-3ad7-584f-bdeb-fba5c3881d32
chr17   a76469cb-973c-5d4d-bf82-7cf4e8f6c129
chr10   727c9d57-7b74-556f-aa5b-e1ca1f76d119
chr15   b4a86ffd-e60c-5c9c-aaa1-9e9f02d86116
chr5    3a023e72-da92-54f7-aa18-502c1076b2b0
</code></pre>
<p><strong>Example 2:</strong> A user has retrieved a <code>ssm_occurrence</code>, and would like to determine if that case also has tissue slides and transcriptome profiling data.</p>
<pre><code class="language-Shell">curl 'https://api.gdc.cancer.gov/ssm_occurrences/6fd8527d-5c40-5604-8fa9-0ce798eec231?pretty=true&amp;expand=case,case.summary.experimental_strategies'
</code></pre>
<pre><code class="language-Json">{
  &quot;data&quot;: {
    &quot;case&quot;: {
      &quot;disease_type&quot;: &quot;Nevi and Melanomas&quot;, 
      &quot;updated_datetime&quot;: &quot;2018-09-06T18:42:50.098635-05:00&quot;, 
      &quot;created_datetime&quot;: null, 
      &quot;summary&quot;: {
        &quot;experimental_strategies&quot;: [
          {
            &quot;file_count&quot;: 3, 
            &quot;experimental_strategy&quot;: &quot;miRNA-Seq&quot;
          }, 
          {
            &quot;file_count&quot;: 1, 
            &quot;experimental_strategy&quot;: &quot;Tissue Slide&quot;
          }, 
          {
            &quot;file_count&quot;: 18, 
            &quot;experimental_strategy&quot;: &quot;WXS&quot;
          }, 
          {
            &quot;file_count&quot;: 1, 
            &quot;experimental_strategy&quot;: &quot;Diagnostic Slide&quot;
          }, 
          {
            &quot;file_count&quot;: 4, 
            &quot;experimental_strategy&quot;: &quot;RNA-Seq&quot;
          }, 
          {
            &quot;file_count&quot;: 4, 
            &quot;experimental_strategy&quot;: &quot;Genotyping Array&quot;
          }, 
          {
            &quot;file_count&quot;: 1, 
            &quot;experimental_strategy&quot;: &quot;Methylation Array&quot;
          }
        ]
      }, 
      &quot;state&quot;: &quot;released&quot;, 
      &quot;case_id&quot;: &quot;590b5e18-d837-4c0e-becf-80520db57c0f&quot;, 
      &quot;primary_site&quot;: &quot;Skin&quot;, 
      &quot;submitter_id&quot;: &quot;TCGA-Z2-A8RT&quot;, 
      &quot;available_variation_data&quot;: [
        &quot;cnv&quot;, 
        &quot;ssm&quot;
      ]
    }, 
    &quot;ssm_occurrence_id&quot;: &quot;6fd8527d-5c40-5604-8fa9-0ce798eec231&quot;
  }
</code></pre>
<h2>Copy Number Variation Endpoint Examples</h2>
<p><strong>Example 1:</strong> A user is interested in finding the first 30 cnvs found on chromosome 4 that have a cnv loss. </p>
<pre><code class="language-Filter">{
    &quot;op&quot;: &quot;and&quot;,
    &quot;content&quot;: [
        {
            &quot;op&quot;: &quot;in&quot;,
            &quot;content&quot;: {
                &quot;field&quot;: &quot;chromosome&quot;,
                &quot;value&quot;: [
                    &quot;4&quot;
                ]
            }
        },
        {
            &quot;op&quot;: &quot;in&quot;,
            &quot;content&quot;: {
                &quot;field&quot;: &quot;cnv_change&quot;,
                &quot;value&quot;: [
                    &quot;Loss&quot;
                ]
            }
        }
    ]
}
</code></pre>
<pre><code class="language-Shell">curl 'https://api.gdc.cancer.gov/cnvs?filters=%7B%0D%0A+++%22op%22%3A+%22and%22%2C%0D%0A++++%22content%22%3A+%5B%0D%0A++++++++%7B%0D%0A++++++++++++%22op%22%3A+%22in%22%2C%0D%0A++++++++++++%22content%22%3A+%7B%0D%0A++++++++++++++++%22field%22%3A+%22chromosome%22%2C%0D%0A++++++++++++++++%22value%22%3A+%5B%0D%0A++++++++++++++++++++%224%22%0D%0A++++++++++++++++%5D%0D%0A++++++++++++%7D%0D%0A++++++++%7D%2C%0D%0A++++++++%7B%0D%0A++++++++++++%22op%22%3A+%22in%22%2C%0D%0A++++++++++++%22content%22%3A+%7B%0D%0A++++++++++++++++%22field%22%3A+%22cnv_change%22%2C%0D%0A++++++++++++++++%22value%22%3A+%5B%0D%0A++++++++++++++++++++%22Loss%22%0D%0A++++++++++++++++%5D%0D%0A++++++++++++%7D%0D%0A++++++++%7D%0D%0A++++%5D%0D%0A%7D&amp;size=30&amp;sort=start_position&amp;format=tsv'
</code></pre>
<pre><code class="language-tsv">ncbi_build  cnv_id  gene_level_cn   cnv_change  end_position    start_position  id  chromosome
GRCh38  d18e0dc8-7d56-5d9e-84fd-4f2cf3353c66    True    Loss    88211 53285 d18e0dc8-7d56-5d9e-84fd-4f2cf3353c66    4
GRCh38  357a6606-8a64-5827-b776-e71f44b7e05f    True    Loss    163989  124480  357a6606-8a64-5827-b776-e71f44b7e05f    4
GRCh38  eda45f5f-6a57-5fae-b8ad-5d67a14423f1    True    Loss    305321  270675  eda45f5f-6a57-5fae-b8ad-5d67a14423f1    4
GRCh38  64d82c29-0f20-5a8f-8599-7afb550ab403    True    Loss    384864  337814  64d82c29-0f20-5a8f-8599-7afb550ab403    4
GRCh38  f9d24781-34cb-51ff-99c2-84c83a8348ac    True    Loss    499156  425815  f9d24781-34cb-51ff-99c2-84c83a8348ac    4
GRCh38  56209b45-3b2c-5862-85bb-362722bae857    True    Loss    540196  499210  56209b45-3b2c-5862-85bb-362722bae857    4
GRCh38  04b976d8-90ad-501d-b672-e14816582339    True    Loss    670782  625584  04b976d8-90ad-501d-b672-e14816582339    4
GRCh38  574939d6-bf4f-57e9-9c86-629b3d8de664    True    Loss    674338  672436  574939d6-bf4f-57e9-9c86-629b3d8de664    4
GRCh38  b2ebf724-0a08-542e-ad1e-392a30208140    True    Loss    682033  673580  b2ebf724-0a08-542e-ad1e-392a30208140    4
GRCh38  4e37e683-6f9f-5e80-8e3b-78d0cdf3c28e    True    Loss    689441  681829  4e37e683-6f9f-5e80-8e3b-78d0cdf3c28e    4
GRCh38  06837ab7-8242-518f-a24c-dce8a0140b01    True    Loss    770640  705748  06837ab7-8242-518f-a24c-dce8a0140b01    4
GRCh38  9f877f14-55ea-5e19-afa0-d294d1700b4b    True    Loss    826198  784957  9f877f14-55ea-5e19-afa0-d294d1700b4b    4
GRCh38  bde18311-8a8a-52ef-bcc0-3b6660509df0    True    Loss    932373  849276  bde18311-8a8a-52ef-bcc0-3b6660509df0    4
GRCh38  31c65477-0e54-5be3-b1f6-3f249850ef79    True    Loss    958656  932387  31c65477-0e54-5be3-b1f6-3f249850ef79    4
GRCh38  c26f1b4d-d4c3-5685-8789-fb0051f8a188    True    Loss    986895  958887  c26f1b4d-d4c3-5685-8789-fb0051f8a188    4
GRCh38  0aa931e9-7ec1-57e7-9cb9-ec66a8da5689    True    Loss    993440  979073  0aa931e9-7ec1-57e7-9cb9-ec66a8da5689    4
GRCh38  162a9e1d-e1ee-5478-9291-6ba8082d5776    True    Loss    1004506 986997  162a9e1d-e1ee-5478-9291-6ba8082d5776    4
GRCh38  6a4d4aef-2289-54f5-b78b-797db8c3a9f2    True    Loss    1026897 1009936 6a4d4aef-2289-54f5-b78b-797db8c3a9f2    4
GRCh38  3c26920b-fb93-5595-81a0-770df0c88246    True    Loss    1113562 1056250 3c26920b-fb93-5595-81a0-770df0c88246    4
GRCh38  7036724d-1a73-5b2b-ae02-c2dc5b3333d7    True    Loss    1208962 1166932 7036724d-1a73-5b2b-ae02-c2dc5b3333d7    4
GRCh38  30b408be-db7b-579b-bbde-4a265c6291ce    True    Loss    1249953 1211448 30b408be-db7b-579b-bbde-4a265c6291ce    4
GRCh38  a7c6f097-bba8-5859-838d-8b3b4610c9e6    True    Loss    1340147 1289851 a7c6f097-bba8-5859-838d-8b3b4610c9e6    4
GRCh38  8fd4f4e8-ddf3-574b-ac19-3112a2778b22    True    Loss    1388049 1347266 8fd4f4e8-ddf3-574b-ac19-3112a2778b22    4
GRCh38  2315f6cc-9d91-58b8-9f3e-f0d36cd6846c    True    Loss    1395989 1391552 2315f6cc-9d91-58b8-9f3e-f0d36cd6846c    4
GRCh38  1480d682-fe0e-5ba1-bf4e-ac84945f194a    True    Loss    1406331 1402932 1480d682-fe0e-5ba1-bf4e-ac84945f194a    4
GRCh38  280e825e-1c51-506b-a4b5-3dc85fd79cbe    True    Loss    1684302 1617915 280e825e-1c51-506b-a4b5-3dc85fd79cbe    4
GRCh38  607e36e3-6b1d-5564-9670-759668053ceb    True    Loss    1712555 1692800 607e36e3-6b1d-5564-9670-759668053ceb    4
GRCh38  93b6ccc4-d88d-5040-936f-a23c9006a965    True    Loss    1721358 1715952 93b6ccc4-d88d-5040-936f-a23c9006a965    4
GRCh38  f6f660d2-5a68-5e49-92b1-a816be39e0fe    True    Loss    1745176 1721490 f6f660d2-5a68-5e49-92b1-a816be39e0fe    4
GRCh38  a0c069d1-dcb0-5833-8fff-211cd6e3719a    True    Loss    1808872 1793307 a0c069d1-dcb0-5833-8fff-211cd6e3719a    4
</code></pre>
<p><strong>Example 2:</strong> A user wants to determine the location and identity of the gene affected by the cnv <code>5052be09-2bbe-5175-a0ae-fc568ea75339</code>, and determine whether the gene is found within the Cancer Gene Census.</p>
<pre><code class="language-Shell">curl 'https://api.gdc.cancer.gov/cnvs/5052be09-2bbe-5175-a0ae-fc568ea75339?pretty=true&amp;expand=consequence.gene'
</code></pre>
<pre><code class="language-Json">{
  &quot;data&quot;: {
    &quot;ncbi_build&quot;: &quot;GRCh38&quot;, 
    &quot;cnv_id&quot;: &quot;5052be09-2bbe-5175-a0ae-fc568ea75339&quot;, 
    &quot;gene_level_cn&quot;: true, 
    &quot;cnv_change&quot;: &quot;Gain&quot;, 
    &quot;end_position&quot;: 110346681, 
    &quot;start_position&quot;: 110338506, 
    &quot;consequence&quot;: [
      {
        &quot;gene&quot;: {
          &quot;symbol&quot;: &quot;RBM15&quot;, 
          &quot;is_cancer_gene_census&quot;: &quot;True&quot;, 
          &quot;biotype&quot;: &quot;protein_coding&quot;, 
          &quot;gene_id&quot;: &quot;ENSG00000162775&quot;
        }
      }
    ], 
    &quot;chromosome&quot;: &quot;1&quot;
  }
</code></pre>
<h2>Copy Number Variation Occurrence Enpoint Examples</h2>
<p><strong>Example 1:</strong> A user is interested in finding cases that have both cnv and ssm data for females diagnosed with Squamous Cell Neoplasms and have a cnv gain change on chromosome 9. It is important to note that for a case like this, where multiple arguments are need for one filtered field, it is easier for the API to have multiple filters for the same field, <code>case.available_variation_data</code> in this example, than having one filter with multiple arguments.</p>
<pre><code class="language-Filter">{
    &quot;op&quot;: &quot;and&quot;,
    &quot;content&quot;: [
        {
            &quot;op&quot;: &quot;in&quot;,
            &quot;content&quot;: {
                &quot;field&quot;: &quot;cnv.cnv_change&quot;,
                &quot;value&quot;: [
                    &quot;Gain&quot;
                ]
            }
        },
        {
            &quot;op&quot;: &quot;in&quot;,
            &quot;content&quot;: {
                &quot;field&quot;: &quot;case.demographic.gender&quot;,
                &quot;value&quot;: [
                    &quot;female&quot;
                ]
            }
        },
        {
            &quot;op&quot;: &quot;in&quot;,
            &quot;content&quot;: {
                &quot;field&quot;: &quot;case.available_variation_data&quot;,
                &quot;value&quot;: [
                    &quot;cnv&quot;
                ]
            }
        },
        {
            &quot;op&quot;: &quot;in&quot;,
            &quot;content&quot;: {
                &quot;field&quot;: &quot;case.available_variation_data&quot;,
                &quot;value&quot;: [
                    &quot;ssm&quot;
                ]
            }
        },
        {
            &quot;op&quot;: &quot;in&quot;,
            &quot;content&quot;: {
                &quot;field&quot;: &quot;cnv.chromosome&quot;,
                &quot;value&quot;: [
                    &quot;9&quot;
                ]
            }
        },
        {
            &quot;op&quot;: &quot;in&quot;,
            &quot;content&quot;: {
                &quot;field&quot;: &quot;case.disease_type&quot;,
                &quot;value&quot;: [
                    &quot;Squamous Cell Neoplasms&quot;
                ]
            }
        }
    ]
}

</code></pre>
<pre><code class="language-Shell">curl 'https://api.gdc.cancer.gov/cnv_occurrences?filters=%7B%0D%0A++++%22op%22%3A+%22and%22%2C%0D%0A++++%22content%22%3A+%5B%0D%0A++++++++%7B%0D%0A++++++++++++%22op%22%3A+%22in%22%2C%0D%0A++++++++++++%22content%22%3A+%7B%0D%0A++++++++++++++++%22field%22%3A+%22cnv.cnv_change%22%2C%0D%0A++++++++++++++++%22value%22%3A+%5B%0D%0A++++++++++++++++++++%22Gain%22%0D%0A++++++++++++++++%5D%0D%0A++++++++++++%7D%0D%0A++++++++%7D%2C%0D%0A++++++++%7B%0D%0A++++++++++++%22op%22%3A+%22in%22%2C%0D%0A++++++++++++%22content%22%3A+%7B%0D%0A++++++++++++++++%22field%22%3A+%22case.demographic.gender%22%2C%0D%0A++++++++++++++++%22value%22%3A+%5B%0D%0A++++++++++++++++++++%22female%22%0D%0A++++++++++++++++%5D%0D%0A++++++++++++%7D%0D%0A++++++++%7D%2C%0D%0A++++++++%7B%0D%0A++++++++++++%22op%22%3A+%22in%22%2C%0D%0A++++++++++++%22content%22%3A+%7B%0D%0A++++++++++++++++%22field%22%3A+%22case.available_variation_data%22%2C%0D%0A++++++++++++++++%22value%22%3A+%5B%0D%0A++++++++++++++++++++%22cnv%22%0D%0A++++++++++++++++%5D%0D%0A++++++++++++%7D%0D%0A++++++++%7D%2C%0D%0A++++++++%7B%0D%0A++++++++++++%22op%22%3A+%22in%22%2C%0D%0A++++++++++++%22content%22%3A+%7B%0D%0A++++++++++++++++%22field%22%3A+%22case.available_variation_data%22%2C%0D%0A++++++++++++++++%22value%22%3A+%5B%0D%0A++++++++++++++++++++%22ssm%22%0D%0A++++++++++++++++%5D%0D%0A++++++++++++%7D%0D%0A++++++++%7D%2C%0D%0A++++++++%7B%0D%0A++++++++++++%22op%22%3A+%22in%22%2C%0D%0A++++++++++++%22content%22%3A+%7B%0D%0A++++++++++++++++%22field%22%3A+%22cnv.chromosome%22%2C%0D%0A++++++++++++++++%22value%22%3A+%5B%0D%0A++++++++++++++++++++%229%22%0D%0A++++++++++++++++%5D%0D%0A++++++++++++%7D%0D%0A++++++++%7D%2C%0D%0A++++++++%7B%0D%0A++++++++++++%22op%22%3A+%22in%22%2C%0D%0A++++++++++++%22content%22%3A+%7B%0D%0A++++++++++++++++%22field%22%3A+%22case.disease_type%22%2C%0D%0A++++++++++++++++%22value%22%3A+%5B%0D%0A++++++++++++++++++++%22Squamous+Cell+Neoplasms%22%0D%0A++++++++++++++++%5D%0D%0A++++++++++++%7D%0D%0A++++++++%7D%0D%0A++++%5D%0D%0A%7D&amp;fields=case.available_variation_data,case.case_id&amp;format=tsv'
</code></pre>
<pre><code class="language-tsv">case.case_id    case.available_variation_data.1 case.available_variation_data.0 id
638035f6-2909-4a44-980f-468ac5d74e18    ssm cnv e76d2aaf-f951-5a51-a949-a241dba61f73
ad98977b-e159-410a-b8c2-f4e8a07f9784    ssm cnv ff3506b8-ee80-570f-ad2d-4ab4a7363b82
c83c52f4-3815-4f49-8218-cf80aaa62e2f    ssm cnv e73696c5-386f-5cae-aa10-f8628f32ee0e
dac27c24-cdbf-4527-9214-178fde3d098a    ssm cnv 77885824-fae1-5116-9851-694255249cc8
0e91d7b5-ce35-4671-ab9f-cfd5369b557c    ssm cnv 526529ae-8e59-597e-aea1-cc0b06a82e76
ea34663c-f40e-4a3e-9ac0-65d5e9eef12b    ssm cnv e4a0c034-44d4-5dea-912a-ce331d9a9512
05026179-b1da-411e-a286-89727b1ae380    ssm cnv 30bdc04c-54a5-53ca-bdd0-b808f23da266
f1a1bbf9-4751-4fb4-8a2b-19f8d4ba57bd    ssm cnv 02e3fbb3-da8f-5983-8d10-189e641ddf11
a6ec75d4-1c90-4527-bfae-aa91d2dae082    ssm cnv 94b0e8be-1130-5b88-9103-6756bdabf67b
107f6b9a-2883-4499-a40a-ec25bc834a06    ssm cnv ad831f27-e6f5-5b78-8a15-0b652621ea4c
</code></pre>
<p><strong>Example 2:</strong> A user is interested in the first cnv occurrence (<code>e76d2aaf-f951-5a51-a949-a241dba61f73</code>) from the previous example, and would like to know more about the case exposures and demographics.</p>
<pre><code class="language-Shell">curl 'https://api.gdc.cancer.gov/cnv_occurrences/e76d2aaf-f951-5a51-a949-a241dba61f73?pretty=true&amp;expand=cnv,case,case.exposures,case.demographic'
</code></pre>
<pre><code class="language-Json">{
  &quot;data&quot;: {
    &quot;cnv&quot;: {
      &quot;ncbi_build&quot;: &quot;GRCh38&quot;, 
      &quot;cnv_id&quot;: &quot;0d475712-c11e-51fb-b6e6-407d12978057&quot;, 
      &quot;gene_level_cn&quot;: true, 
      &quot;cnv_change&quot;: &quot;Gain&quot;, 
      &quot;end_position&quot;: 133348131, 
      &quot;variant_status&quot;: &quot;Tumor only&quot;, 
      &quot;start_position&quot;: 133338323, 
      &quot;chromosome&quot;: &quot;9&quot;
    }, 
    &quot;case&quot;: {
      &quot;disease_type&quot;: &quot;Squamous Cell Neoplasms&quot;, 
      &quot;updated_datetime&quot;: &quot;2018-09-06T11:07:45.510627-05:00&quot;, 
      &quot;created_datetime&quot;: null, 
      &quot;demographic&quot;: {
        &quot;updated_datetime&quot;: &quot;2018-09-06T11:07:45.510627-05:00&quot;, 
        &quot;created_datetime&quot;: null, 
        &quot;gender&quot;: &quot;female&quot;, 
        &quot;year_of_birth&quot;: 1954, 
        &quot;submitter_id&quot;: &quot;TCGA-EA-A3HR_demographic&quot;, 
        &quot;state&quot;: &quot;released&quot;, 
        &quot;race&quot;: &quot;white&quot;, 
        &quot;demographic_id&quot;: &quot;dd8576a8-bd62-55e7-b0df-7233ceded2fb&quot;, 
        &quot;ethnicity&quot;: &quot;not hispanic or latino&quot;, 
        &quot;year_of_death&quot;: null
      }, 
      &quot;submitter_id&quot;: &quot;TCGA-EA-A3HR&quot;, 
      &quot;state&quot;: &quot;released&quot;, 
      &quot;case_id&quot;: &quot;638035f6-2909-4a44-980f-468ac5d74e18&quot;, 
      &quot;primary_site&quot;: &quot;Cervix uteri&quot;, 
      &quot;available_variation_data&quot;: [
        &quot;cnv&quot;, 
        &quot;ssm&quot;
      ], 
      &quot;exposures&quot;: [
        {
          &quot;cigarettes_per_day&quot;: null, 
          &quot;weight&quot;: 86, 
          &quot;updated_datetime&quot;: &quot;2018-09-06T11:07:45.510627-05:00&quot;, 
          &quot;created_datetime&quot;: null, 
          &quot;alcohol_intensity&quot;: null, 
          &quot;bmi&quot;: 40, 
          &quot;years_smoked&quot;: null, 
          &quot;submitter_id&quot;: &quot;TCGA-EA-A3HR_exposure&quot;, 
          &quot;alcohol_history&quot;: null, 
          &quot;state&quot;: &quot;released&quot;, 
          &quot;tobacco_smoking_status&quot;: null, 
          &quot;tobacco_smoking_onset_year&quot;: null, 
          &quot;tobacco_smoking_quit_year&quot;: null, 
          &quot;exposure_id&quot;: &quot;0e7265ab-bf65-50c7-bf33-96a7ac452d7c&quot;, 
          &quot;height&quot;: 146, 
          &quot;pack_years_smoked&quot;: null
        }
      ]
    }, 
    &quot;cnv_occurrence_id&quot;: &quot;e76d2aaf-f951-5a51-a949-a241dba61f73&quot;
  }
</code></pre>
<h2>Analysis Endpoints</h2>
<p>In addition to the <code>ssms</code>, <code>ssm_occurrences</code>, and <code>genes</code> endpoints mentioned previously, several <code>/analysis</code> endpoints were designed to quickly retrieve specific datasets used for visualization display.  </p>
<p><strong>Example 1:</strong> The <code>/analysis/top_cases_counts_by_genes</code> endpoint gives the number of cases with a mutation in each gene listed in the <code>gene_ids</code> parameter for each project. Note that this endpoint cannot be used with the <code>format</code> or <code>fields</code> parameters. In this instance, the query will produce the number of cases in each projects with mutations in the gene <code>ENSG00000155657</code>.</p>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/analysis/top_cases_counts_by_genes?gene_ids=ENSG00000155657&amp;pretty=true&quot;
</code></pre>
<p>This JSON-formatted output is broken up by project. For an example, see the following text:</p>
<pre><code class="language-json">          &quot;genes&quot;: {
            &quot;my_genes&quot;: {
              &quot;gene_id&quot;: {
                &quot;buckets&quot;: [
                  {
                    &quot;key&quot;: &quot;ENSG00000155657&quot;,
                    &quot;doc_count&quot;: 45
                  }
                ],
                &quot;sum_other_doc_count&quot;: 0,
                &quot;doc_count_error_upper_bound&quot;: 0
              },
              &quot;doc_count&quot;: 45
            },
            &quot;doc_count&quot;: 12305
          },
          &quot;key&quot;: &quot;TCGA-GBM&quot;,
          &quot;doc_count&quot;: 45
        }
</code></pre>
<p>This portion of the output shows TCGA-GBM including 45 cases that have <code>ssms</code> in the gene <code>ENSG00000155657</code>.</p>
<p><strong>Example 2:</strong> The following demonstrates a use of the <code>/analysis/top_mutated_genes_by_project</code> endpoint.  This will output the genes that are mutated in the most cases in "TCGA-DLBC" and will count the mutations that have a <code>HIGH</code> or <code>MODERATE</code> impact on gene function. Note that the <code>score</code> field does not represent the number of mutations in a given gene, but a calculation that is used to determine which genes have the greatest number of unique mutations.  </p>
<pre><code class="language-json">{  
   &quot;op&quot;:&quot;AND&quot;,
   &quot;content&quot;:[  
      {  
         &quot;op&quot;:&quot;in&quot;,
         &quot;content&quot;:{  
            &quot;field&quot;:&quot;case.project.project_id&quot;,
            &quot;value&quot;:[  
               &quot;TCGA-DLBC&quot;
            ]
         }
      },
      {  
         &quot;op&quot;:&quot;in&quot;,
         &quot;content&quot;:{  
            &quot;field&quot;:&quot;case.ssm.consequence.transcript.annotation.impact&quot;,
            &quot;value&quot;:[  
               &quot;HIGH&quot;,
               &quot;MODERATE&quot;
            ]
         }
      }
   ]
}
</code></pre>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/analysis/top_mutated_genes_by_project?fields=gene_id,symbol&amp;filters=%7B%22op%22%3A%22AND%22%2C%22content%22%3A%5B%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22case.project.project_id%22%2C%22value%22%3A%5B%22TCGA-DLBC%22%5D%7D%7D%2C%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22case.ssm.consequence.transcript.annotation.impact%22%2C%22value%22%3A%5B%22HIGH%22%2C%22MODERATE%22%5D%7D%7D%5D%7D&amp;pretty=true&quot;
</code></pre>
<pre><code class="language-Response">{
  &quot;data&quot;: {
    &quot;hits&quot;: [
      {
        &quot;_score&quot;: 14.0,
        &quot;symbol&quot;: &quot;IGHV2-70&quot;,
        &quot;gene_id&quot;: &quot;ENSG00000274576&quot;
      },
      {
        &quot;_score&quot;: 14.0,
        &quot;symbol&quot;: &quot;IGLV3-1&quot;,
        &quot;gene_id&quot;: &quot;ENSG00000211673&quot;
      },
      {
        &quot;_score&quot;: 14.0,
        &quot;symbol&quot;: &quot;IGHM&quot;,
        &quot;gene_id&quot;: &quot;ENSG00000211899&quot;
      },
      {
        &quot;_score&quot;: 11.0,
        &quot;symbol&quot;: &quot;KMT2D&quot;,
        &quot;gene_id&quot;: &quot;ENSG00000167548&quot;
      },
      {
        &quot;_score&quot;: 11.0,
        &quot;symbol&quot;: &quot;IGLL5&quot;,
        &quot;gene_id&quot;: &quot;ENSG00000254709&quot;
      },
      {
        &quot;_score&quot;: 11.0,
        &quot;symbol&quot;: &quot;BTG2&quot;,
        &quot;gene_id&quot;: &quot;ENSG00000159388&quot;
      },
      {
        &quot;_score&quot;: 9.0,
        &quot;symbol&quot;: &quot;CARD11&quot;,
        &quot;gene_id&quot;: &quot;ENSG00000198286&quot;
      },
      {
        &quot;_score&quot;: 9.0,
        &quot;symbol&quot;: &quot;IGHG1&quot;,
        &quot;gene_id&quot;: &quot;ENSG00000211896&quot;
      },
      {
        &quot;_score&quot;: 9.0,
        &quot;symbol&quot;: &quot;IGLC2&quot;,
        &quot;gene_id&quot;: &quot;ENSG00000211677&quot;
      },
      {
        &quot;_score&quot;: 9.0,
        &quot;symbol&quot;: &quot;LRP1B&quot;,
        &quot;gene_id&quot;: &quot;ENSG00000168702&quot;
      }
    ],
    &quot;pagination&quot;: {
      &quot;count&quot;: 10,
      &quot;sort&quot;: &quot;None&quot;,
      &quot;from&quot;: 0,
      &quot;page&quot;: 1,
      &quot;total&quot;: 3214,
      &quot;pages&quot;: 322,
      &quot;size&quot;: 10
    }
  },
  &quot;warnings&quot;: {}
}
</code></pre>
<p><strong>Example 3:</strong> The <code>/analysis/top_mutated_cases_by_gene</code> endpoint will generate information about the cases that are most affected by mutations in a given number of genes. Below, the file count for each category is given for the cases most affected by mutations in these 50 genes.  The size of the output is limited to two cases with the <code>size=2</code> parameter, but a higher value can be set by the user.</p>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/analysis/top_mutated_cases_by_gene?fields=diagnoses.days_to_death,diagnoses.age_at_diagnosis,diagnoses.vital_status,diagnoses.primary_diagnosis,demographic.gender,demographic.race,demographic.ethnicity,case_id,summary.data_categories.file_count,summary.data_categories.data_category&amp;filters=%7B%22op%22%3A%22and%22%2C%22content%22%3A%5B%7B%22op%22%3A%22%3D%22%2C%22content%22%3A%7B%22field%22%3A%22cases.project.project_id%22%2C%22value%22%3A%22TCGA-DLBC%22%7D%7D%2C%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22genes.gene_id%22%2C%22value%22%3A%5B%22ENSG00000166710%22%2C%22ENSG00000005339%22%2C%22ENSG00000083857%22%2C%22ENSG00000168769%22%2C%22ENSG00000100906%22%2C%22ENSG00000184677%22%2C%22ENSG00000101680%22%2C%22ENSG00000101266%22%2C%22ENSG00000028277%22%2C%22ENSG00000140968%22%2C%22ENSG00000181827%22%2C%22ENSG00000116815%22%2C%22ENSG00000275221%22%2C%22ENSG00000139083%22%2C%22ENSG00000112851%22%2C%22ENSG00000112697%22%2C%22ENSG00000164134%22%2C%22ENSG00000009413%22%2C%22ENSG00000071626%22%2C%22ENSG00000135407%22%2C%22ENSG00000101825%22%2C%22ENSG00000104814%22%2C%22ENSG00000166415%22%2C%22ENSG00000142867%22%2C%22ENSG00000254585%22%2C%22ENSG00000139718%22%2C%22ENSG00000077721%22%2C%22ENSG00000130294%22%2C%22ENSG00000117245%22%2C%22ENSG00000117318%22%2C%22ENSG00000270550%22%2C%22ENSG00000163637%22%2C%22ENSG00000166575%22%2C%22ENSG00000065526%22%2C%22ENSG00000156453%22%2C%22ENSG00000128191%22%2C%22ENSG00000055609%22%2C%22ENSG00000204469%22%2C%22ENSG00000187605%22%2C%22ENSG00000185875%22%2C%22ENSG00000110888%22%2C%22ENSG00000007341%22%2C%22ENSG00000173198%22%2C%22ENSG00000115568%22%2C%22ENSG00000163714%22%2C%22ENSG00000125772%22%2C%22ENSG00000080815%22%2C%22ENSG00000189079%22%2C%22ENSG00000120837%22%2C%22ENSG00000143951%22%5D%7D%7D%2C%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22ssms.consequence.transcript.annotation.impact%22%2C%22value%22%3A%5B%22HIGH%22%5D%7D%7D%5D%7D&amp;pretty=true&amp;size=2&quot;
</code></pre>
<pre><code class="language-Response">{
  &quot;data&quot;: {
    &quot;hits&quot;: [
      {
        &quot;_score&quot;: 7.0,
        &quot;diagnoses&quot;: [
          {
            &quot;days_to_death&quot;: null,
            &quot;vital_status&quot;: &quot;alive&quot;,
            &quot;age_at_diagnosis&quot;: 18691,
            &quot;primary_diagnosis&quot;: &quot;c83.3&quot;
          }
        ],
        &quot;case_id&quot;: &quot;eda9496e-be80-4a13-bf06-89f0cc9e937f&quot;,
        &quot;demographic&quot;: {
          &quot;gender&quot;: &quot;male&quot;,
          &quot;race&quot;: &quot;white&quot;,
          &quot;ethnicity&quot;: &quot;hispanic or latino&quot;
        },
        &quot;summary&quot;: {
          &quot;data_categories&quot;: [
            {
              &quot;file_count&quot;: 1,
              &quot;data_category&quot;: &quot;DNA Methylation&quot;
            },
            {
              &quot;file_count&quot;: 5,
              &quot;data_category&quot;: &quot;Transcriptome Profiling&quot;
            },
            {
              &quot;file_count&quot;: 1,
              &quot;data_category&quot;: &quot;Biospecimen&quot;
            },
            {
              &quot;file_count&quot;: 16,
              &quot;data_category&quot;: &quot;Simple Nucleotide Variation&quot;
            },
            {
              &quot;file_count&quot;: 1,
              &quot;data_category&quot;: &quot;Clinical&quot;
            },
            {
              &quot;file_count&quot;: 4,
              &quot;data_category&quot;: &quot;Copy Number Variation&quot;
            },
            {
              &quot;file_count&quot;: 4,
              &quot;data_category&quot;: &quot;Raw Sequencing Data&quot;
            }
          ]
        }
      },
      {
        &quot;_score&quot;: 4.0,
        &quot;diagnoses&quot;: [
          {
            &quot;days_to_death&quot;: null,
            &quot;vital_status&quot;: &quot;alive&quot;,
            &quot;age_at_diagnosis&quot;: 27468,
            &quot;primary_diagnosis&quot;: &quot;c83.3&quot;
          }
        ],
        &quot;case_id&quot;: &quot;a43e5f0e-a21f-48d8-97e0-084d413680b7&quot;,
        &quot;demographic&quot;: {
          &quot;gender&quot;: &quot;male&quot;,
          &quot;race&quot;: &quot;white&quot;,
          &quot;ethnicity&quot;: &quot;not hispanic or latino&quot;
        },
        &quot;summary&quot;: {
          &quot;data_categories&quot;: [
            {
              &quot;file_count&quot;: 1,
              &quot;data_category&quot;: &quot;DNA Methylation&quot;
            },
            {
              &quot;file_count&quot;: 5,
              &quot;data_category&quot;: &quot;Transcriptome Profiling&quot;
            },
            {
              &quot;file_count&quot;: 1,
              &quot;data_category&quot;: &quot;Biospecimen&quot;
            },
            {
              &quot;file_count&quot;: 16,
              &quot;data_category&quot;: &quot;Simple Nucleotide Variation&quot;
            },
            {
              &quot;file_count&quot;: 1,
              &quot;data_category&quot;: &quot;Clinical&quot;
            },
            {
              &quot;file_count&quot;: 4,
              &quot;data_category&quot;: &quot;Copy Number Variation&quot;
            },
            {
              &quot;file_count&quot;: 4,
              &quot;data_category&quot;: &quot;Raw Sequencing Data&quot;
            }
          ]
        }
      }
    ],
    &quot;pagination&quot;: {
      &quot;count&quot;: 2,
      &quot;sort&quot;: &quot;None&quot;,
      &quot;from&quot;: 0,
      &quot;page&quot;: 1,
      &quot;total&quot;: 27,
      &quot;pages&quot;: 14,
      &quot;size&quot;: 2
    }
  },
  &quot;warnings&quot;: {}
}
</code></pre>
<p><strong>Example 4:</strong>  The <code>/analysis/mutated_cases_count_by_project</code> endpoint produces counts for the number of cases that have associated <code>ssm</code> data in each project. The number of affected cases can be found under <code>"case_with_ssm": {"doc_count": $case_count}</code>.  </p>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/analysis/mutated_cases_count_by_project?size=0&amp;pretty=true&quot;
</code></pre>
<pre><code class="language-Response">{
  &quot;hits&quot;: {
    &quot;hits&quot;: [],
    &quot;total&quot;: 14551,
    &quot;max_score&quot;: 0.0
  },
  &quot;_shards&quot;: {
    &quot;successful&quot;: 9,
    &quot;failed&quot;: 0,
    &quot;total&quot;: 9
  },
  &quot;took&quot;: 4,
  &quot;aggregations&quot;: {
    &quot;projects&quot;: {
      &quot;buckets&quot;: [
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 216
            },
            &quot;doc_count&quot;: 637
          },
          &quot;key&quot;: &quot;TARGET-NBL&quot;,
          &quot;doc_count&quot;: 1127
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 1044
            },
            &quot;doc_count&quot;: 7625
          },
          &quot;key&quot;: &quot;TCGA-BRCA&quot;,
          &quot;doc_count&quot;: 1098
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 8
            },
            &quot;doc_count&quot;: 579
          },
          &quot;key&quot;: &quot;TARGET-AML&quot;,
          &quot;doc_count&quot;: 988
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 34
            },
            &quot;doc_count&quot;: 290
          },
          &quot;key&quot;: &quot;TARGET-WT&quot;,
          &quot;doc_count&quot;: 652
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 396
            },
            &quot;doc_count&quot;: 3197
          },
          &quot;key&quot;: &quot;TCGA-GBM&quot;,
          &quot;doc_count&quot;: 617
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 443
            },
            &quot;doc_count&quot;: 3880
          },
          &quot;key&quot;: &quot;TCGA-OV&quot;,
          &quot;doc_count&quot;: 608
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 569
            },
            &quot;doc_count&quot;: 3874
          },
          &quot;key&quot;: &quot;TCGA-LUAD&quot;,
          &quot;doc_count&quot;: 585
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 542
            },
            &quot;doc_count&quot;: 3874
          },
          &quot;key&quot;: &quot;TCGA-UCEC&quot;,
          &quot;doc_count&quot;: 560
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 339
            },
            &quot;doc_count&quot;: 3547
          },
          &quot;key&quot;: &quot;TCGA-KIRC&quot;,
          &quot;doc_count&quot;: 537
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 510
            },
            &quot;doc_count&quot;: 3671
          },
          &quot;key&quot;: &quot;TCGA-HNSC&quot;,
          &quot;doc_count&quot;: 528
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 513
            },
            &quot;doc_count&quot;: 3606
          },
          &quot;key&quot;: &quot;TCGA-LGG&quot;,
          &quot;doc_count&quot;: 516
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 496
            },
            &quot;doc_count&quot;: 3536
          },
          &quot;key&quot;: &quot;TCGA-THCA&quot;,
          &quot;doc_count&quot;: 507
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 497
            },
            &quot;doc_count&quot;: 3520
          },
          &quot;key&quot;: &quot;TCGA-LUSC&quot;,
          &quot;doc_count&quot;: 504
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 498
            },
            &quot;doc_count&quot;: 3490
          },
          &quot;key&quot;: &quot;TCGA-PRAD&quot;,
          &quot;doc_count&quot;: 500
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 470
            },
            &quot;doc_count&quot;: 3289
          },
          &quot;key&quot;: &quot;TCGA-SKCM&quot;,
          &quot;doc_count&quot;: 470
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 433
            },
            &quot;doc_count&quot;: 3188
          },
          &quot;key&quot;: &quot;TCGA-COAD&quot;,
          &quot;doc_count&quot;: 461
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 441
            },
            &quot;doc_count&quot;: 3095
          },
          &quot;key&quot;: &quot;TCGA-STAD&quot;,
          &quot;doc_count&quot;: 443
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 412
            },
            &quot;doc_count&quot;: 2884
          },
          &quot;key&quot;: &quot;TCGA-BLCA&quot;,
          &quot;doc_count&quot;: 412
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 0
            },
            &quot;doc_count&quot;: 0
          },
          &quot;key&quot;: &quot;TARGET-OS&quot;,
          &quot;doc_count&quot;: 381
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 375
            },
            &quot;doc_count&quot;: 2635
          },
          &quot;key&quot;: &quot;TCGA-LIHC&quot;,
          &quot;doc_count&quot;: 377
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 305
            },
            &quot;doc_count&quot;: 2142
          },
          &quot;key&quot;: &quot;TCGA-CESC&quot;,
          &quot;doc_count&quot;: 307
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 288
            },
            &quot;doc_count&quot;: 2033
          },
          &quot;key&quot;: &quot;TCGA-KIRP&quot;,
          &quot;doc_count&quot;: 291
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 255
            },
            &quot;doc_count&quot;: 1821
          },
          &quot;key&quot;: &quot;TCGA-SARC&quot;,
          &quot;doc_count&quot;: 261
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 149
            },
            &quot;doc_count&quot;: 1192
          },
          &quot;key&quot;: &quot;TCGA-LAML&quot;,
          &quot;doc_count&quot;: 200
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 184
            },
            &quot;doc_count&quot;: 1293
          },
          &quot;key&quot;: &quot;TCGA-ESCA&quot;,
          &quot;doc_count&quot;: 185
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 183
            },
            &quot;doc_count&quot;: 1285
          },
          &quot;key&quot;: &quot;TCGA-PAAD&quot;,
          &quot;doc_count&quot;: 185
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 179
            },
            &quot;doc_count&quot;: 1253
          },
          &quot;key&quot;: &quot;TCGA-PCPG&quot;,
          &quot;doc_count&quot;: 179
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 158
            },
            &quot;doc_count&quot;: 1169
          },
          &quot;key&quot;: &quot;TCGA-READ&quot;,
          &quot;doc_count&quot;: 172
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 150
            },
            &quot;doc_count&quot;: 1018
          },
          &quot;key&quot;: &quot;TCGA-TGCT&quot;,
          &quot;doc_count&quot;: 150
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 123
            },
            &quot;doc_count&quot;: 867
          },
          &quot;key&quot;: &quot;TCGA-THYM&quot;,
          &quot;doc_count&quot;: 124
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 66
            },
            &quot;doc_count&quot;: 556
          },
          &quot;key&quot;: &quot;TCGA-KICH&quot;,
          &quot;doc_count&quot;: 113
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 92
            },
            &quot;doc_count&quot;: 620
          },
          &quot;key&quot;: &quot;TCGA-ACC&quot;,
          &quot;doc_count&quot;: 92
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 83
            },
            &quot;doc_count&quot;: 605
          },
          &quot;key&quot;: &quot;TCGA-MESO&quot;,
          &quot;doc_count&quot;: 87
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 80
            },
            &quot;doc_count&quot;: 560
          },
          &quot;key&quot;: &quot;TCGA-UVM&quot;,
          &quot;doc_count&quot;: 80
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 0
            },
            &quot;doc_count&quot;: 163
          },
          &quot;key&quot;: &quot;TARGET-RT&quot;,
          &quot;doc_count&quot;: 75
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 48
            },
            &quot;doc_count&quot;: 346
          },
          &quot;key&quot;: &quot;TCGA-DLBC&quot;,
          &quot;doc_count&quot;: 58
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 57
            },
            &quot;doc_count&quot;: 399
          },
          &quot;key&quot;: &quot;TCGA-UCS&quot;,
          &quot;doc_count&quot;: 57
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 51
            },
            &quot;doc_count&quot;: 306
          },
          &quot;key&quot;: &quot;TCGA-CHOL&quot;,
          &quot;doc_count&quot;: 51
        },
        {
          &quot;case_summary&quot;: {
            &quot;case_with_ssm&quot;: {
              &quot;doc_count&quot;: 0
            },
            &quot;doc_count&quot;: 13
          },
          &quot;key&quot;: &quot;TARGET-CCSK&quot;,
          &quot;doc_count&quot;: 13
        }
      ],
      &quot;sum_other_doc_count&quot;: 0,
      &quot;doc_count_error_upper_bound&quot;: 0
    }
  },
  &quot;timed_out&quot;: false
}
</code></pre>
<h3>Survival Analysis Endpoint</h3>
<p><a href="/Data_Portal/Users_Guide/Exploration/#survival-analysis">Survival plots</a> are generated for different subsets of data, based on variants or projects, in the GDC Data Portal. The <code>/analysis/survival</code> endpoint can be used to programmatically retrieve the raw data used to generate these plots and apply different filters. Note that the <code>fields</code> and <code>format</code> parameters cannot be modified.</p>
<p><strong>Example 1:</strong> A user wants to download data to generate a survival plot for cases from the project TCGA-DLBC.</p>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/analysis/survival?filters=%5B%7B%22op%22%3A%22%3D%22%2C%22content%22%3A%7B%22field%22%3A%22cases.project.project_id%22%2C%22value%22%3A%22TCGA-DLBC%22%7D%7D%5D&amp;pretty=true&quot;
</code></pre>
<pre><code class="language-Response">{
  &quot;overallStats&quot;: {},
  &quot;results&quot;: [
    {
      &quot;donors&quot;: [
        {
          &quot;survivalEstimate&quot;: 1,
          &quot;id&quot;: &quot;dc87a809-95de-4eb7-a1c2-2650475f2d7e&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 1
        },
        {
          &quot;survivalEstimate&quot;: 1,
          &quot;id&quot;: &quot;4dd86ebd-ef16-4b2b-9ea0-5d1d7afef257&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 17
        },
        {
          &quot;survivalEstimate&quot;: 1,
          &quot;id&quot;: &quot;0bf573ac-cd1e-42d8-90cf-b30d7b08679c&quot;,
          &quot;censored&quot;: false,
          &quot;time&quot;: 58
        },
        {
          &quot;survivalEstimate&quot;: 0.9777777777777777,
          &quot;id&quot;: &quot;f978cb0f-d319-4c01-b4c5-23ae1403a106&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 126
        },
        {
          &quot;survivalEstimate&quot;: 0.9777777777777777,
          &quot;id&quot;: &quot;a43e5f0e-a21f-48d8-97e0-084d413680b7&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 132
        },
        {
          &quot;survivalEstimate&quot;: 0.9777777777777777,
          &quot;id&quot;: &quot;1843c82e-7a35-474f-9f79-c0a9af9aa09c&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 132
        },
        {
          &quot;survivalEstimate&quot;: 0.9777777777777777,
          &quot;id&quot;: &quot;0030a28c-81aa-44b0-8be0-b35e1dcbf98c&quot;,
          &quot;censored&quot;: false,
          &quot;time&quot;: 248
        },
        {
          &quot;survivalEstimate&quot;: 0.9539295392953929,
          &quot;id&quot;: &quot;f553f1a9-ecf2-4783-a609-6adca7c4c597&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 298
        },
        {
          &quot;survivalEstimate&quot;: 0.9539295392953929,
          &quot;id&quot;: &quot;f784bc3a-751b-4025-aab2-0af2f6f24266&quot;,
          &quot;censored&quot;: false,
          &quot;time&quot;: 313
        },
        {
          &quot;survivalEstimate&quot;: 0.929469807518588,
          &quot;id&quot;: &quot;29e3d122-15a1-4235-a356-b1a9f94ceb39&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 385
        },
        {
          &quot;survivalEstimate&quot;: 0.929469807518588,
          &quot;id&quot;: &quot;0e251c03-bf86-4ed8-b45d-3cbc97160502&quot;,
          &quot;censored&quot;: false,
          &quot;time&quot;: 391
        },
        {
          &quot;survivalEstimate&quot;: 0.9043490019099776,
          &quot;id&quot;: &quot;e6365b38-bc44-400c-b4aa-18ce8ff5bfce&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 427
        },
        {
          &quot;survivalEstimate&quot;: 0.9043490019099776,
          &quot;id&quot;: &quot;b56bdbdb-43af-4a03-a072-54dd22d7550c&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 553
        },
        {
          &quot;survivalEstimate&quot;: 0.9043490019099776,
          &quot;id&quot;: &quot;31bbad4e-3789-42ec-9faa-1cb86970f723&quot;,
          &quot;censored&quot;: false,
          &quot;time&quot;: 595
        },
        {
          &quot;survivalEstimate&quot;: 0.8777505018538018,
          &quot;id&quot;: &quot;0e9fcccc-0630-408d-a121-2c6413824cb7&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 679
        },
        {
          &quot;survivalEstimate&quot;: 0.8777505018538018,
          &quot;id&quot;: &quot;a5b188f0-a6d3-4d4a-b04f-36d47ec05338&quot;,
          &quot;censored&quot;: false,
          &quot;time&quot;: 708
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;ed746cb9-0f2f-48ce-923a-3a9f9f00b331&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 719
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;c85f340e-584b-4f3b-b6a5-540491fc8ad2&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 730
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;69f23725-adca-48ac-9b33-80a7aae24cfe&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 749
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;67325322-483f-443f-9ffa-2a20d108a2fb&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 751
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;eda9496e-be80-4a13-bf06-89f0cc9e937f&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 765
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;25ff86af-beb4-480c-b706-f3fe0306f7cf&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 788
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;1d0db5d7-39ca-466d-96b3-0d278c5ea768&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 791
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;c8cde9ea-89e9-4ee8-8a46-417a48f6d3ab&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 832
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;f0a326d2-1f3e-4a5d-bca8-32aaccc52338&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 946
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;a8e2df1e-4042-42af-9231-3a00e83489f0&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 965
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;e56e4d9c-052e-4ec6-a81b-dbd53e9c8ffe&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 972
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;45b0cf9f-a879-417f-8f39-7770552252c0&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 982
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;1f971af1-6772-4fe6-8d35-bbe527a037fe&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 1081
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;33365d22-cb83-4d8e-a2d1-06b675f75f6e&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 1163
        },
        {
          &quot;survivalEstimate&quot;: 0.8503207986708705,
          &quot;id&quot;: &quot;6a21c948-cd85-4150-8c01-83017d7dc1ed&quot;,
          &quot;censored&quot;: false,
          &quot;time&quot;: 1252
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;f855dad1-6ffc-493e-ba6c-970874bc9210&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 1299
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;c1c06604-5ae2-4a53-b9c0-eb210d38e3f0&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 1334
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;58e66976-4507-4552-ac53-83a49a142dde&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 1373
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;ea54dbad-1b23-41cc-9378-d4002a8fca51&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 1581
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;d7df78b5-24f1-4ff4-bd9b-f0e6bec8289a&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 1581
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;29aff186-c321-4ff9-b81b-105e27e620ff&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 1617
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;5eff68ff-f6c3-40c9-9fc8-00e684a7b712&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 1739
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;f8cf647b-1447-4ac3-8c43-bef07765cabf&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 2131
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;c3d662ee-48d0-454a-bb0c-77d3338d3747&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 2983
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;6e9437f0-a4ed-475c-ab0e-bf1431c70a90&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 3333
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;fdecb74f-ac4e-46b1-b23a-5f7fde96ef9f&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 3394
        },
        {
          &quot;survivalEstimate&quot;: 0.8003019281608192,
          &quot;id&quot;: &quot;a468e725-ad4b-411d-ac5c-2eacc68ec580&quot;,
          &quot;censored&quot;: false,
          &quot;time&quot;: 3553
        },
        {
          &quot;survivalEstimate&quot;: 0.6402415425286554,
          &quot;id&quot;: &quot;1ea575f1-f731-408b-a629-f5f4abab569e&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 3897
        },
        {
          &quot;survivalEstimate&quot;: 0.6402415425286554,
          &quot;id&quot;: &quot;7a589441-11ef-4158-87e7-3951d86bc2aa&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 4578
        },
        {
          &quot;survivalEstimate&quot;: 0.6402415425286554,
          &quot;id&quot;: &quot;3622cf29-600f-4410-84d4-a9afeb41c475&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 5980
        },
        {
          &quot;survivalEstimate&quot;: 0.6402415425286554,
          &quot;id&quot;: &quot;3f5a897d-1eaa-4d4c-8324-27ac07c90927&quot;,
          &quot;censored&quot;: false,
          &quot;time&quot;: 6425
        }
      ],
      &quot;meta&quot;: {
        &quot;id&quot;: 140429063094496
      }
    }
  ]
}
</code></pre>
<p><strong>Example 2:</strong> Here the survival endpoint is used to compare two survival plots for TCGA-BRCA cases.  One plot will display survival information about cases with a particular mutation (in this instance: <code>chr3:g.179234297A&gt;G</code>) and the other plot will display information about cases without that mutation. This type of query will also print the results of a chi-squared analysis between the two subsets of cases.  </p>
<pre><code class="language-json">[  
  {  
    &quot;op&quot;:&quot;and&quot;,
    &quot;content&quot;:[  
      {  
        &quot;op&quot;:&quot;=&quot;,
        &quot;content&quot;:{  
          &quot;field&quot;:&quot;cases.project.project_id&quot;,
          &quot;value&quot;:&quot;TCGA-BRCA&quot;
        }
      },
      {  
        &quot;op&quot;:&quot;=&quot;,
        &quot;content&quot;:{  
          &quot;field&quot;:&quot;gene.ssm.ssm_id&quot;,
          &quot;value&quot;:&quot;edd1ae2c-3ca9-52bd-a124-b09ed304fcc2&quot;
        }
      }
    ]
  },
  {  
    &quot;op&quot;:&quot;and&quot;,
    &quot;content&quot;:[  
      {  
        &quot;op&quot;:&quot;=&quot;,
        &quot;content&quot;:{  
          &quot;field&quot;:&quot;cases.project.project_id&quot;,
          &quot;value&quot;:&quot;TCGA-BRCA&quot;
        }
      },
      {  
        &quot;op&quot;:&quot;excludeifany&quot;,
        &quot;content&quot;:{  
          &quot;field&quot;:&quot;gene.ssm.ssm_id&quot;,
          &quot;value&quot;:&quot;edd1ae2c-3ca9-52bd-a124-b09ed304fcc2&quot;
        }
      }
    ]
  }
]
</code></pre>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/analysis/survival?filters=%5B%7B%22op%22%3A%22and%22%2C%22content%22%3A%5B%7B%22op%22%3A%22%3D%22%2C%22content%22%3A%7B%22field%22%3A%22cases.project.project_id%22%2C%22value%22%3A%22TCGA-BRCA%22%7D%7D%2C%7B%22op%22%3A%22%3D%22%2C%22content%22%3A%7B%22field%22%3A%22gene.ssm.ssm_id%22%2C%22value%22%3A%22edd1ae2c-3ca9-52bd-a124-b09ed304fcc2%22%7D%7D%5D%7D%2C%7B%22op%22%3A%22and%22%2C%22content%22%3A%5B%7B%22op%22%3A%22%3D%22%2C%22content%22%3A%7B%22field%22%3A%22cases.project.project_id%22%2C%22value%22%3A%22TCGA-BRCA%22%7D%7D%2C%7B%22op%22%3A%22excludeifany%22%2C%22content%22%3A%7B%22field%22%3A%22gene.ssm.ssm_id%22%2C%22value%22%3A%22edd1ae2c-3ca9-52bd-a124-b09ed304fcc2%22%7D%7D%5D%7D%5D&amp;pretty=true&quot;
</code></pre>
<pre><code class="language-json2">{
  &quot;overallStats&quot;: {
    &quot;degreesFreedom&quot;: 1,
    &quot;chiSquared&quot;: 0.8577589072612264,
    &quot;pValue&quot;: 0.35436660628146011
  },
  &quot;results&quot;: [
    {
      &quot;donors&quot;: [
        {
          &quot;survivalEstimate&quot;: 1,
          &quot;id&quot;: &quot;a991644b-3ee6-4cda-acf0-e37de48a49fc&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 10
        },
        {
          &quot;survivalEstimate&quot;: 1,
          &quot;id&quot;: &quot;2e1e3bf0-1708-4b65-936c-48b89eb8966a&quot;,
          &quot;censored&quot;: true,
          &quot;time&quot;: 19
        },
(truncated)
],
&quot;meta&quot;: {
  &quot;id&quot;: 140055251282040
}
},
{
&quot;donors&quot;: [
  {
    &quot;survivalEstimate&quot;: 1,
    &quot;id&quot;: &quot;5e4187c9-98f8-4bdb-a8da-6a914e96f47a&quot;,
    &quot;censored&quot;: true,
    &quot;time&quot;: -31
  },
(truncated)
</code></pre>
<p>The output represents two sets of coordinates delimited as objects with the <code>donors</code> tag. One set of coordinates will generate a survival plot representing TCGA-BRCA cases that have the mutation of interest and the other will generate a survival plot for the remaining cases in TCGA-BRCA.</p>
<p><strong>Example 3:</strong> Custom survival plots can be generated using the GDC API.  For example, a user could generate survival plot data comparing patients with a mutation in genes associated with a biological pathway with patients without mutations in that pathway. The following example compares a patient with at least one mutation in either gene <code>ENSG00000141510</code> or <code>ENSG00000155657</code> with patients that do not have mutations in these genes.</p>
<pre><code class="language-Query">[  
   {  
      &quot;op&quot;:&quot;and&quot;,
      &quot;content&quot;:[  
         {  
            &quot;op&quot;:&quot;=&quot;,
            &quot;content&quot;:{  
               &quot;field&quot;:&quot;cases.project.project_id&quot;,
               &quot;value&quot;:&quot;TCGA-BRCA&quot;
            }
         },
         {  
            &quot;op&quot;:&quot;=&quot;,
            &quot;content&quot;:{  
               &quot;field&quot;:&quot;gene.gene_id&quot;,
               &quot;value&quot;:[&quot;ENSG00000141510&quot;,&quot;ENSG00000155657&quot;]
            }
         }
      ]
   },
   {  
      &quot;op&quot;:&quot;and&quot;,
      &quot;content&quot;:[  
         {  
            &quot;op&quot;:&quot;=&quot;,
            &quot;content&quot;:{  
               &quot;field&quot;:&quot;cases.project.project_id&quot;,
               &quot;value&quot;:&quot;TCGA-BRCA&quot;
            }
         },
         {  
            &quot;op&quot;:&quot;excludeifany&quot;,
            &quot;content&quot;:{  
               &quot;field&quot;:&quot;gene.gene_id&quot;,
               &quot;value&quot;:[&quot;ENSG00000141510&quot;,&quot;ENSG00000155657&quot;]
            }
         }
      ]
   }
]
</code></pre>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/analysis/survival?filters=%5B%0D%0A%7B%0D%0A%22op%22%3A%22and%22%2C%0D%0A%22content%22%3A%5B%0D%0A%7B%0D%0A%22op%22%3A%22%3D%22%2C%0D%0A%22content%22%3A%7B%0D%0A%22field%22%3A%22cases.project.project_id%22%2C%0D%0A%22value%22%3A%22TCGA-BRCA%22%0D%0A%7D%0D%0A%7D%2C%0D%0A%7B%0D%0A%22op%22%3A%22%3D%22%2C%0D%0A%22content%22%3A%7B%0D%0A%22field%22%3A%22gene.gene_id%22%2C%0D%0A%22value%22%3A%5B%22ENSG00000141510%22%2C%22ENSG00000155657%22%5D%0D%0A%7D%0D%0A%7D%0D%0A%5D%0D%0A%7D%2C%0D%0A%7B%0D%0A%22op%22%3A%22and%22%2C%0D%0A%22content%22%3A%5B%0D%0A%7B%0D%0A%22op%22%3A%22%3D%22%2C%0D%0A%22content%22%3A%7B%0D%0A%22field%22%3A%22cases.project.project_id%22%2C%0D%0A%22value%22%3A%22TCGA-BRCA%22%0D%0A%7D%0D%0A%7D%2C%0D%0A%7B%0D%0A%22op%22%3A%22excludeifany%22%2C%0D%0A%22content%22%3A%7B%0D%0A%22field%22%3A%22gene.gene_id%22%2C%0D%0A%22value%22%3A%5B%22ENSG00000141510%22%2C%22ENSG00000155657%22%5D%0D%0A%7D%0D%0A%7D%0D%0A%5D%0D%0A%7D%0D%0A%5D&amp;pretty=true&quot;
</code></pre>
<p><strong>Example 4:</strong> Survival plots can be even more customizable when sets of case IDs are used. Two sets of case IDs (or barcodes) can be retrieved in a separate step based on custom criteria and compared in a survival plot. See below for an example query.</p>
<pre><code class="language-Json">[{  
   &quot;op&quot;:&quot;=&quot;,
   &quot;content&quot;:{  
      &quot;field&quot;:&quot;cases.submitter_id&quot;,
      &quot;value&quot;:[&quot;TCGA-HT-A74J&quot;,&quot;TCGA-43-A56U&quot;,&quot;TCGA-GM-A3XL&quot;,&quot;TCGA-A1-A0SQ&quot;,&quot;TCGA-K1-A6RV&quot;,&quot;TCGA-J2-A4AD&quot;,&quot;TCGA-XR-A8TE&quot;]
   }
},
{  
   &quot;op&quot;:&quot;=&quot;,
   &quot;content&quot;:{  
      &quot;field&quot;:&quot;cases.submitter_id&quot;,
      &quot;value&quot;:[&quot;TCGA-55-5899&quot;,&quot;TCGA-55-6642&quot;,&quot;TCGA-55-7907&quot;,&quot;TCGA-67-6216&quot;,&quot;TCGA-75-5146&quot;,&quot;TCGA-49-4510&quot;,&quot;TCGA-78-7159&quot;]
   }
}]
</code></pre>
<pre><code class="language-Shell">curl &quot;https://api.gdc.cancer.gov/analysis/survival?filters=%5B%7B%22op%22%3A%22%3D%22%2C%22content%22%3A%7B%22field%22%3A%22cases.submitter_id%22%2C%22value%22%3A%5B%22TCGA-HT-A74J%22%2C%22TCGA-43-A56U%22%2C%22TCGA-GM-A3XL%22%2C%22TCGA-A1-A0SQ%22%2C%22TCGA-K1-A6RV%22%2C%22TCGA-J2-A4AD%22%2C%22TCGA-XR-A8TE%22%5D%7D%7D%2C%7B%22op%22%3A%22%3D%22%2C%22content%22%3A%7B%22field%22%3A%22cases.submitter_id%22%2C%22value%22%3A%5B%22TCGA-55-5899%22%2C%22TCGA-55-6642%22%2C%22TCGA-55-7907%22%2C%22TCGA-67-6216%22%2C%22TCGA-75-5146%22%2C%22TCGA-49-4510%22%2C%22TCGA-78-7159%22%5D%7D%7D%5D&quot;
</code></pre>
        </div>
      </div>
    </div>

      <footer id="docs-footer">
        <div class="container-fluid footer-container">

      <div>
        <a href="https://portal.gdc.cancer.gov" target="_blank">Site Home</a>
        | <a href="http://www.cancer.gov/global/web/policies" target="_blank">Policies</a>
        | <a href="http://www.cancer.gov/global/web/policies/accessibility" target="_blank">Accessibility</a>
        | <a href="http://www.cancer.gov/global/web/policies/foia" target="_blank">FOIA</a>
      </div>
      <div>
        <a href="http://www.hhs.gov" target="_blank">U.S. Department of Health and Human Services</a>
        | <a href="http://www.nih.gov" target="_blank">National Institutes of Health</a>
        | <a href="http://www.cancer.gov" target="_blank">National Cancer Institute</a>
        | <a href="http://www.usa.gov" target="_blank">USA.gov</a>
      </div>
      <div>
        NIH... Turning Discovery Into Health 
      </div>

      <div class="version-caption">
        GDC Docs Version 2.0
      </div>
    </div>
    </footer>
    



    
<script>
/* When the user clicks on the button, 
toggle between hiding and showing the dropdown content */

// Close the dropdown if the user clicks outside of it

var toggler = document.getElementById("myToggle");
toggler.onclick= function(){
    var myDropdown = document.getElementById("myDropdown");
    myDropdown.classList.toggle('show');
}
window.onclick = function(e) {
    if (!e.target.matches('.dropbtn')) {
        var myDropdown = document.getElementById("myDropdown");
    if (myDropdown.classList.contains('show')) {
        myDropdown.classList.remove('show');
    }
  }
}

</script>

    <!-- JavaScript Bundle with Popper -->
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>      
</body>
</html>