<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link href="vendor/fontawesome/css/all.min.css" rel="stylesheet">
    <!-- CSS only -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=EB+Garamond:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&family=Source+Sans+Pro:ital,wght@0,300;0,400;0,600;0,700;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet">
    
      <link rel="stylesheet" href=/home/jacob/Work/Docsite-Revamp/css/styles.css>
    
      <link rel="stylesheet" href=/home/jacob/Work/Docsite-Revamp/css/docs.css>
    
    <!--<link rel="stylesheet" href="../../css/styles.css">
    <link rel="stylesheet" href="../../css/encyclopedia.css">-->
</head>
<body>
    <!---->
    <nav class="navbar navbar-expand-md navbar-light bg-light">
        <div class="container-fluid">
                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
                    <span class="navbar-toggler-icon"></span>
                </button>
                <!-- Collapsed navigation -->
          <div class="my-container">
            <!-- Main title -->
            <a class="navbar-brand float-start" href="/.">
                <img class="brand-image" src="../../img/nih-header-logo-color.png" alt="NIH National Cancer Institute GDC Documentation">
                <span class="hidden">NIH National Cancer Institute GDC Documentation</span>
            </a>
            <!-- Expander button -->
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>

            <!-- GDC Apps -->
            <div class="pull-right">
              <div class="gdc-app-list dropbtn">
                <div class="dropbtn">
                  <a id='myToggle' class="dropbtn dropdown-toggle">
                    <span class="dropbtn"><i class="fas fa-th"></i> GDC Apps</span>
                  </a>
                  <div id="myDropdown" class="dropdown-content container">
                    <div class="row gdc-app-row">
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                    </div>
                    <div class="row gdc-app-row">
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                    </div>
                    <div class="row gdc-app-row">
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                      <div class="col-sm gdc-app-list">
                        <i class="fas fa-lg fa-heartbeat"></i>
                      </div>
                    </div>
                  </div>
                </div>
                <div data-dropdown-menu="" class="dropdown-menu gdc-apps-menu-container" role="menu">
                  <div class="gdc-apps-menu-grid">
                    <div class="menu-column">
                      <a href="https://portal.gdc.cancer.gov/" title="Data Portal" target="_blank" class="menu-item">
                        <img width="21px" src="/img/GDC-App-data-portal-blue.svg" class="icon icon-gdc-portal home" alt="GDC Data Portal">
                        <div>Data Portal</div>
                      </a>
                      <a href="https://gdc.cancer.gov/developers/gdc-application-programming-interface-api" title="GDC API" target="_blank" class="menu-item">
                        <span class="icon icon-gdc-portal-api">
                          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span>
                          </span>
                        <div>API</div>
                      </a>
                      <a href="https://docs.gdc.cancer.gov/" title="GDC Docs" target="_blank" class="active menu-item">
                        <span class="icon icon-gdc-docs">
                          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span><span class="path11"></span><span class="path12"></span><span class="path13"></span><span class="path14"></span>
                        </span>
                        <div>Documentation</div>
                      </a>
                      <a href="https://portal.gdc.cancer.gov/legacy-archive" title="GDC Legacy Archive" target="_blank" class="menu-item">
                        <span class="icon icon-gdc-legacy-archive">
                          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span>
                        </span>
                        <div>Legacy Archive</div>
                      </a>
                    </div>
                    <div class="menu-column">
                      <a href="https://gdc.cancer.gov/" title="GDC Website" target="_blank" class="menu-item">
                        <img width="21px" src="/img/GDC-App-website-blue.svg" class="icon icon-gdc-portal home" alt="GDC Website">
                        <div>Website</div>
                      </a>
                      <a href="https://gdc.cancer.gov/access-data/gdc-data-transfer-tool" class="menu-item" title="GDC Data Transfer Tool" target="_blank">
                        <span class="icon icon-gdc-data-transer-tool">
                          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span>
                        </span>
                        <div>Data Transfer Tool</div>
                      </a>
                      <a href="https://portal.gdc.cancer.gov/submission/" title="GDC Submission Portal" target="_blank" class="menu-item">
                        <span class="icon icon-gdc-submission-portal">
                          <span class="path1"></span><span class="path2"></span><span class="path3"></span><span class="path4"></span><span class="path5"></span><span class="path6"></span><span class="path7"></span><span class="path8"></span><span class="path9"></span><span class="path10"></span>
                        </span>
                        <div>Data Submission Portal</div>
                      </a>
                    </div>
                  </div>
                </div>
              </div>
            </div>
              <div class="container-fluid">
              <div class="collapse navbar-collapse" id="navbarNavDropdown">
                <ul class="navbar-nav navbar-center me-auto">
                <li class="nav-item">
                    <a class="nav-link" href="#">Home</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link active" aria-current="page" href="#">API</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link" href="#">Data Submission</a>
                </li>
                <li class="nav-item dropdown">
                    <a class="nav-link dropdown-toggle" href="#" id="navbarDropdownMenuLink" role="button" data-bs-toggle="dropdown" aria-expanded="false">
                    Data Transfer Tool
                    </a>
                    <ul class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                    <li><a class="dropdown-item" href="#">Action</a></li>
                    <li><a class="dropdown-item" href="#">Another action</a></li>
                    <li><a class="dropdown-item" href="#">Something else here</a></li>
                    </ul>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="#">Data Dictionary</a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="#">Data</a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="#">Encyclopedia</a>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="#"><i class="fas fa-search"></i></a>
                </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </nav>

      <div class="row">
      <div class="col-md-3">
        <div class="container-fluid sidebar-container">
          <div class="sidebar">
            <ul class="main">
              <li class="main"><a href="#">Getting Started</a></li>
              <ul>
                <li><a href='#Table 1 - File column headers'>Table 1 - File column headers</a></li>
              </ul>
              <li class="inactive"><a href="#">Search and Retrieval</a></li>
              <li class="inactive"><a href="#">Data Analysis</a></li>
              <li class="inactive"><a href="#">BAM Slicing</a></li>
              <li class="inactive"><a href="#">Submission</a></li>
              <li class="inactive"><a href="#">Python Examples</a></li>
              <li class="inactive"><a href="#">GraphQL Examples</a></li>
              <li class="inactive"><a href="#">System Information</a></li>
              <li class="inactive"><a href="#">Additional Examples</a></li>
              <li class="inactive"><a href="#">Appendix A: Available Fields</a></li>
              <li class="inactive"><a href="#">Appendix B: Key Terms</a></li>
              <li class="inactive"><a href="#">Appendix C: Format of Submission Queries and Responses</a></li>
              <li class="inactive"><a href="#">Release Notes</a></li>
            </ul>
          </div>
        </div>
      </div>
      <div class="col-md-9">
        <div class="container-fluid doc-container">
          <h1><i class="fas fa-book"></i>   Mutation Annotation Format (MAF) - Legacy TCGA Specification</h1><hr>
<p><em>This definition was taken from the previously public wiki hosted by TCGA and reflects the MAF format
that was available during the active period of the TCGA project.</em>  </p>
<p><strong>Document Information</strong></p>
<p>The spec has been reverted to the June 26th version (version 20). Additional
changes are the removal of the "under construction" banner, changing all text to
black, and fixing a typo in the link to the MAF 2.2 specification.</p>
<p><strong>Specification for Mutation Annotation Format</strong><br />
Version 2.4.1<br />
June 20, 2014</p>
<p><strong>Contents</strong></p>
<ul>
<li>
<p>1 Current version changes</p>
</li>
<li>
<p>2 About MAF specifications</p>
<ul>
<li>
<p>2.1 Definition of open access MAF
    data</p>
</li>
<li>
<p>2.2 Somatic MAF vs. Protected
    MAF</p>
</li>
</ul>
</li>
<li>
<p>3 MAF file fields</p>
<ul>
<li>3.1 Table 1 - File column
    headers</li>
</ul>
</li>
<li>
<p>4 MAF file checks</p>
</li>
<li>
<p>5 MAF naming convention</p>
</li>
<li>
<p>6 Previous specification
    versions</p>
</li>
</ul>
<h1><i class="fas fa-book"></i>   Current version changes</h1><hr>
<p>This current revision is <strong>version 2.4.1</strong> of the Mutation Annotation Format
(MAF) specification.</p>
<p>The following items in the specification were added or modified in version 2.4.1
from version 2.4:</p>
<ul>
<li>
<p>Header for MAF file is "#version 2.4.1"</p>
</li>
<li>
<p>"Somatic" and "None" are the only acceptable values for "Mutation_Status"
    for a somatic.MAF (named .somatic.maf). When Mutation_Status is None,
    Validation_Status must be Invalid.</p>
</li>
<li>
<p>Centers need to make sure that Mutations_Status "None" doesn't include
    germline mutation.</p>
</li>
<li>
<p>For a somatic MAF, following rules should be satisfied:<br />
    SOMATIC = (A AND (B OR C OR D)) OR (E AND F)<br />
    A: <em>Mutation_Status</em> == "Somatic"<br />
    B: <em>Validation_Status</em> == "Valid"<br />
    C. <em>Verification_Status</em> == "Verified"<br />
    D. <em>Variant_Classification</em> is not {Intron, 5'UTR, 3'UTR, 5'Flank, 3'Flank,
    IGR}, which implies that <em>Variant_Classification</em> can only be
    \{Frame_Shift_Del, Frame_Shift_Ins, In_Frame_Del, In_Frame_Ins,
    Missense_Mutation, Nonsense_Mutation, Silent, Splice_Site,
    Translation_Start_Site, Nonstop_Mutation, RNA, Targeted_Region}.<br />
    E: <em>Mutations_status == "None"</em><br />
    F: <em>Validation_status == "Invalid"</em></p>
</li>
<li>
<p>Extra validation rules: If Validation_Status == Valid or Invalid, then
    Validation_Method != none (case insensitive).</p>
</li>
</ul>
<h1><i class="fas fa-book"></i>   About MAF specifications</h1><hr>
<p>Mutation annotation files should be transferred to the DCC. Those files should
be formatted using the mutation annotation format (MAF) that is described below.
File naming convention is also
<a href="#MutationAnnotationFormat(MAF)Specificat">below</a>.</p>
<p>Following categories of somatic mutations are reported in MAF files:</p>
<ul>
<li>
<p>Missense and nonsense</p>
</li>
<li>
<p>Splice site, defined as SNP within 2 bp of the splice junction</p>
</li>
<li>
<p>Silent mutations</p>
</li>
<li>
<p>Indels that overlap the coding region or splice site of a gene or the
    targeted region of a genetic element of interest.</p>
</li>
<li>
<p>Frameshift mutations</p>
</li>
<li>
<p>Mutations in regulatory regions</p>
</li>
</ul>
<h3>Definition of open access MAF data</h3>
<p>A large proportion of MAFs are submitted as discovery data and sites labeled as
somatic in these files overlap with known germline variants. In order to
minimize germline contamination in putative (unvalidated) somatic calls, certain
filtering criteria have been imposed. Based on current policy, open access MAF
data should:</p>
<ul>
<li>
<p><strong>include</strong> all validated somatic mutation calls</p>
</li>
<li>
<p><strong>include</strong> all unvalidated somatic mutation calls that overlap with a
    coding region or splice site</p>
</li>
<li>
<p><strong>exclude</strong> all other types of mutation calls (i.e., non-somatic calls
    (validated or not), unvalidated somatic calls that are not in coding region
    or splice sites, and dbSNP sites that are not annotated as somatic in dbSNP,
    COSMIC or OMIM)</p>
</li>
</ul>
<h3>Somatic MAF vs. Protected MAF</h3>
<p>Centers will submit to the DCC MAF archives that contain Somatic MAF
(named<strong>.somatic.maf</strong>) for open access data and an all-inclusive Protected MAF
(named<strong>.protected.maf</strong>) that does not filter any data out and represents the
original super-set of mutation calls. The files will be formatted using the
Mutation Annotation Format (MAF).</p>
<p>The following table lists some of the critical attributes of somatic and
protected MAF files and provides a comparison.</p>
<table class='table'>
<thead>
<tr>
<th>Attribute</th>
<th>Somatic MAF</th>
<th>Protected MAF</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>File naming</strong></td>
<td>Somatic MAFs should be named as<strong>*.somatic.maf</strong>and cannot contain 'germ' or 'protected' in file name.</td>
<td>Protected MAFs should be named as<strong>*.protected.maf</strong>and should not contain 'somatic' in the file name.</td>
</tr>
<tr>
<td><strong>Mutation category</strong></td>
<td>Somatic MAFs can only contain entries where<em>Mutation_Status</em>is "Somatic". If any other value is assigned to the field, the archive will fail. Experimentally validated or unvalidated (see next row) somatic mutations can be included in the file.</td>
<td>There is no such restriction for protected MAF. The file should contain all mutation calls including those from which .somatic.maf is derived.</td>
</tr>
<tr>
<td><strong>Filtering criteria</strong></td>
<td>In order to minimize germline contamination, somatic MAFs can contain unvalidated somatic mutations only from coding regions and splice sites, which implies:</td>
<td>There are no such constraints for mutations in protected MAF.</td>
</tr>
<tr>
<td></td>
<td>If <em>Validation_Status</em> <strong>is</strong>"Unknown",<em>V a riant_Classification</em> <strong>cannot</strong> be 3'UTR, 3'Flank, 5'UTR, 5'Flank, IGR, or Intron.<em>Variant_Classification</em>can only be \{Frame_Shift_Del, Frame_Shift_Ins, In_Frame_Del, In_Frame_Ins, Missense_Mutation, Nonsense_Mutation, Silent, Splice_Site, Translation_Start_Site, Nonstop_Mutation, RNA, Targeted_Region, De_novo_Start_InFrame, De_novo_Start_OutOfFrame\}.</td>
<td></td>
</tr>
<tr>
<td></td>
<td>There is no such constraint for experimentally validated (<em>Validation_Status</em>is "Valid") somatic mutations.</td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td>dbSNP sites that are not annotated as somatic in dbSNP, COSMIC or OMIM must be removed from somatic MAFs.</td>
<td></td>
</tr>
<tr>
<td><strong>Access level</strong></td>
<td>These files are deployed as open access data.</td>
<td>These files are deployed as protected data.</td>
</tr>
</tbody>
</table>
<h1><i class="fas fa-book"></i>   MAF file fields</h1><hr>
<p>The format of a MAF file is tab-delimited columns. Those columns are described
in Table 1 and are required in every MAF file. The order of the columns will be
validated by the DCC. Column headers and values <strong>are</strong> case sensitive where
specified. Columns may allow null values (i.e._ blank cells) and/or have
enumerated values. <strong>The validator looks for a header stating the version of the
specification to validate against (e.g. #version 2.4). If not, validation
fails.</strong> Any columns that come after the columns described in Table 1 are
optional. Optional columns are not validated by the DCC and can be in any order.</p>
<h2>Table 1 - File column headers</h2>
<table class='table'>
<thead>
<tr>
<th><strong>Index</strong></th>
<th><strong>MAF Column Header</strong></th>
<th><strong>Description of Values</strong></th>
<th><strong>Example</strong></th>
<th><strong>Case Sensitive</strong></th>
<th><strong>Null</strong></th>
<th><strong>Enumerated</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>Hugo_Symbol</td>
<td>HUGO symbol for the gene (HUGO symbols are <em>always</em> in all caps). If no gene exists within 3kb enter "Unknown".</td>
<td>EGFR</td>
<td>Yes</td>
<td>No</td>
<td>Set or Unknown</td>
</tr>
<tr>
<td></td>
<td></td>
<td>Source: <a href="http://genenames.org">http://genenames.org</a></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>2</td>
<td>Entrez_Gene_Id</td>
<td>Entrez gene ID (an integer). If no gene exists within 3kb enter "0".</td>
<td>1956</td>
<td>No</td>
<td>No</td>
<td>Set</td>
</tr>
<tr>
<td></td>
<td></td>
<td>Source: <a href="http://ncbi.nlm.nih.gov/sites/entrez?db=gene">http://ncbi.nlm.nih.gov/sites/entrez?db=gene</a></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>3</td>
<td>Center</td>
<td>Genome sequencing center reporting the variant. If multiple institutions report the same mutation separate list using semicolons. Non-GSC centers will be also supported if center name is an accepted center name.</td>
<td>hgsc.bcm.edu;genome.wustl.edu</td>
<td>Yes</td>
<td>No</td>
<td>Set</td>
</tr>
<tr>
<td>4</td>
<td>NCBI_Build</td>
<td>Any TGCA accepted genome identifier. Can be string, integer or a float.</td>
<td>hg18, hg19, GRCh37, GRCh37-lite, 36, 36.1, 37,</td>
<td>No</td>
<td>No</td>
<td>Set and Enumerated.</td>
</tr>
<tr>
<td>5</td>
<td>Chromosome</td>
<td>Chromosome number without "chr" prefix that contains the gene.</td>
<td>X, Y, M, 1, 2, etc.</td>
<td>Yes</td>
<td>No</td>
<td>Set</td>
</tr>
<tr>
<td>6</td>
<td>Start_Position</td>
<td>Lowest numeric position of the reported variant on the genomic reference sequence. Mutation start coordinate (1-based coordinate system).</td>
<td>999</td>
<td>No</td>
<td>No</td>
<td>Set</td>
</tr>
<tr>
<td>7</td>
<td>End_Position</td>
<td>Highest numeric genomic position of the reported variant on the genomic reference sequence. Mutation end coordinate (inclusive, 1-based coordinate system).</td>
<td>1000</td>
<td>No</td>
<td>No</td>
<td>Set</td>
</tr>
<tr>
<td>8</td>
<td>Strand</td>
<td>Genomic strand of the reported allele. Variants should always be reported on the positive genomic strand. (Currently, only the positive strand is an accepted value).</td>
<td>+</td>
<td>No</td>
<td>No</td>
<td>+</td>
</tr>
<tr>
<td>9</td>
<td>Variant_Classification</td>
<td>Translational effect of variant allele.</td>
<td>Missense_Mutation</td>
<td>Yes</td>
<td>No</td>
<td>Frame_Shift_Del, Frame_Shift_Ins, In_Frame_Del, In_Frame_Ins, Missense_Mutation, Nonsense_Mutation, Silent, Splice_Site, Translation_Start_Site, Nonstop_Mutation, 3'UTR, 3'Flank, 5'UTR, 5'Flank, IGR <em>(See Notes Section #1)</em> , Intron, RNA, Targeted_Region</td>
</tr>
<tr>
<td>10</td>
<td>Variant_Type</td>
<td>Type of mutation. TNP (tri-nucleotide polymorphism) is analogous to DNP but for 3 consecutive nucleotides. ONP (oligo-nucleotide polymorphism) is analogous to TNP but for consecutive runs of 4 or more.</td>
<td>INS</td>
<td>Yes</td>
<td>No</td>
<td>SNP, DNP, TNP, ONP, INS, DEL, or Consolidated <em>(See Notes Section #2)</em> )</td>
</tr>
<tr>
<td>11</td>
<td>Reference_Allele</td>
<td>The plus strand reference allele at this position. Include the sequence deleted for a deletion, or "-" for an insertion.</td>
<td>A</td>
<td>Yes</td>
<td>No</td>
<td>A,C,G,T and/or -</td>
</tr>
<tr>
<td>12</td>
<td>Tumor_Seq_Allele1</td>
<td>Primary data genotype. Tumor sequencing (discovery) allele 1. " -" for a deletion represent a variant. "-" for an insertion represents wild-type allele. Novel inserted sequence for insertion should not include flanking reference bases.</td>
<td>C</td>
<td>Yes</td>
<td>No</td>
<td>A,C,G,T and/or -</td>
</tr>
<tr>
<td>13</td>
<td>Tumor_Seq_Allele2</td>
<td>Primary data genotype. Tumor sequencing (discovery) allele 2. " -" for a deletion represents a variant. "-" for an insertion represents wild-type allele. Novel inserted sequence for insertion should not include flanking reference bases.</td>
<td>G</td>
<td>Yes</td>
<td>No</td>
<td>A,C,G,T and/or -</td>
</tr>
<tr>
<td>14</td>
<td>dbSNP_RS</td>
<td>Latest dbSNP rs ID (dbSNP_ID) or "novel" if there is no dbSNP record. source: <a href="http://ncbi.nlm.nih.gov/projects/SNP/">http://ncbi.nlm.nih.gov/projects/SNP/</a></td>
<td>rs12345</td>
<td>Yes</td>
<td>Yes</td>
<td>Set or "novel"</td>
</tr>
<tr>
<td>15</td>
<td>dbSNP_Val_Status</td>
<td>dbSNP validation status. Semicolon- separated list of validation statuses.</td>
<td>by2Hit2Allele;byCluster</td>
<td>No</td>
<td>Yes</td>
<td>by1000genomes;by2Hit2Allele; byCluster; byFrequency; byHapMap; byOtherPop; bySubmitter; alternate_allele <em>(See Notes Section #3)</em> <strong>Note that "none" will no longer be an acceptable value.</strong></td>
</tr>
<tr>
<td>16</td>
<td>Tumor_Sample_Barcode</td>
<td>BCR aliquot barcode for the tumor sample including the two additional fields indicating plate and well position. i.e. TCGA-SiteID-PatientID-SampleID-PortionID-PlateID-CenterID. The full TCGA Aliquot ID.</td>
<td>TCGA-02-0021-01A-01D-0002-04</td>
<td>Yes</td>
<td>No</td>
<td>Set</td>
</tr>
<tr>
<td>17</td>
<td>Matched_Norm_Sample_Barcode</td>
<td>BCR aliquot barcode for the matched normal sample including the two additional fields indicating plate and well position. i.e. TCGA-SiteID-PatientID-SampleID-PortionID-PlateID-CenterID. The full TCGA Aliquot ID; e.g. TCGA-02-0021-10A-01D-0002-04 (compare portion ID '10A' normal sample, to '01A' tumor sample).</td>
<td>TCGA-02-0021-10A-01D-0002-04</td>
<td>Yes</td>
<td>No</td>
<td>Set</td>
</tr>
<tr>
<td>18</td>
<td>Match_Norm_Seq_Allele1</td>
<td>Primary data. Matched normal sequencing allele 1. "-" for deletions; novel inserted sequence for INS not including flanking reference bases.</td>
<td>T</td>
<td>Yes</td>
<td>Yes</td>
<td>A,C,G,T and/or -</td>
</tr>
<tr>
<td>19</td>
<td>Match_Norm_Seq_Allele2</td>
<td>Primary data. Matched normal sequencing allele 2. "-" for deletions; novel inserted sequence for INS not including flanking reference bases.</td>
<td>ACGT</td>
<td>Yes</td>
<td>Yes</td>
<td>A,C,G,T and/or -</td>
</tr>
<tr>
<td>20</td>
<td>Tumor_Validation_Allele1</td>
<td>Secondary data from orthogonal technology. Tumor genotyping (validation) for allele 1. "-" for deletions; novel inserted sequence for INS not including flanking reference bases.</td>
<td>-</td>
<td>Yes</td>
<td>Yes</td>
<td>A,C,G,T and/or -</td>
</tr>
<tr>
<td>21</td>
<td>Tumor_Validation_Allele2</td>
<td>Secondary data from orthogonal technology. Tumor genotyping (validation) for allele 2. "-" for deletions; novel inserted sequence for INS not including flanking reference bases.</td>
<td>A</td>
<td>Yes</td>
<td>Yes</td>
<td>A,C,G,T and/or -</td>
</tr>
<tr>
<td>22</td>
<td>Match_Norm_Validation_Allele1</td>
<td>Secondary data from orthogonal technology. Matched normal genotyping (validation) for allele 1. "-" for deletions; novel inserted sequence for INS not including flanking reference bases.</td>
<td>C</td>
<td>Yes</td>
<td>Yes</td>
<td>A,C,G,T and/or -</td>
</tr>
<tr>
<td>23</td>
<td>Match_Norm_Validation_Allele2</td>
<td>Secondary data from orthogonal technology. Matched normal genotyping (validation) for allele 2. "-" for deletions; novel inserted sequence for INS not including flanking reference bases.</td>
<td>G</td>
<td>Yes</td>
<td>Yes</td>
<td>A,C,G,T and/or -</td>
</tr>
<tr>
<td>24</td>
<td>Verification_Status <em>(See Notes Section #4)</em></td>
<td>Second pass results from independent attempt using same methods as primary data source. Generally reserved for 3730 Sanger Sequencing.</td>
<td>Verified</td>
<td>Yes</td>
<td>Yes</td>
<td>Verified, Unknown</td>
</tr>
<tr>
<td>25</td>
<td>Validation_Status <em>(See Notes Section #5)</em></td>
<td>Second pass results from orthogonal technology.</td>
<td>Valid</td>
<td>Yes</td>
<td>No</td>
<td>Untested, Inconclusive, Valid, Invaild</td>
</tr>
<tr>
<td>26</td>
<td>Mutation_Status</td>
<td>Updated to reflect validation or verification status and to be in agreement with the <a href="https://wiki.nci.nih.gov/x/2gcYAw">VCF VLS</a> field. The values allowed in this field are constrained by the value in the Validation_Status field.</td>
<td>Somatic</td>
<td>Yes</td>
<td>No</td>
<td><strong>Validation_Status values:</strong> Untested, Inconslusive, Valid, Invalid - <strong>Allowed Mutations_Status Values for Untested and Inconclusive:</strong> <em>(See Notes Seciton #6)</em> None, Germline, Somatic, LOH, Post-transcriptional modification <strong>Unknown Allowed Mutation_status Values for Valid:</strong> <em>(See Notes Seciton #6)</em> Germline, Somatic, LOH, Post-transcriptional modification, Unknown - <strong>Allowed Mutations_Status Values for Invalid:</strong> <em>(See Notes Seciton #6)</em> none</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>27</td>
<td>Sequencing_Phase</td>
<td>TCGA sequencing phase. Phase should change under any circumstance that the targets under consideration change.</td>
<td>Phase_I</td>
<td>No</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr>
<td>28</td>
<td>Sequence_Source</td>
<td>Molecular assay type used to produce the analytes used for sequencing. Allowed values are a subset of the <a href="http://www.ncbi.nlm.nih.gov/viewvc/v1/trunk/sra/doc/SRA_1-5/">SRA 1.5</a> library_strategy field values. This subset matches those used at CGHub.</td>
<td>WGS;WXS</td>
<td>Yes</td>
<td>No</td>
<td><strong>Common TCGA values:</strong> WGS, WGA, WXS, RNA-Seq, miRNA-Seq, Bisulfite-Seq, VALIDATION, Other <strong>Other allowed values (per SRA 1.5)</strong>  ncRNA-Seq, WCS, CLONE, POOLCLONE, AMPLICON, CLONEEND, FINISHING, ChIP-Seq, MNase-Seq, DNase-Hypersensitivity, EST, FL-cDNA, CTS, MRE-Seq, MeDIP-Seq, MBD-Seq, Tn-Seq, FAIRE-seq, SELEX, RIP-Seq, ChIA-PET</td>
</tr>
<tr>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr>
<td>29</td>
<td>Validation_Method</td>
<td>The assay platforms used for the validation call. Examples: Sanger_PCR_WGA, Sanger_PCR_gDNA, 454_PCR_WGA, 454_PCR_gDNA; separate multiple entries using semicolons.</td>
<td>Sanger_PCR_WGA;Sanger_PCR_gDNA</td>
<td>No</td>
<td><strong>NO</strong>. I<strong>f Validation_Status = Untested then "none"</strong> If Validation_Status = Valid or Invalid, then not "none" (case insensitive)</td>
<td>No</td>
</tr>
<tr>
<td>30</td>
<td>Score</td>
<td>Not in use.</td>
<td>NA</td>
<td>No</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr>
<td>31</td>
<td>BAM_File</td>
<td>Not in use.</td>
<td>NA</td>
<td>No</td>
<td>Yes</td>
<td>No</td>
</tr>
<tr>
<td>32</td>
<td>Sequencer</td>
<td>Instrument used to produce primary data. Separate multiple entries using semicolons.</td>
<td>Illumina GAIIx;SOLID</td>
<td>Yes</td>
<td>No</td>
<td>Illumina GAIIx, Illumina HiSeq, SOLID, 454, ABI 3730xl, Ion Torrent PGM, Ion Torrent Proton, PacBio RS, Illumina MiSeq, Illumina HiSeq 2500, 454 GS FLX Titanium, AB SOLiD 4 System</td>
</tr>
<tr>
<td>33</td>
<td>Tumor_Sample_UUID</td>
<td>BCR aliquot UUID for tumor sample</td>
<td>550e8400-e29b-41d4-a716-446655440000</td>
<td>Yes</td>
<td>No</td>
<td></td>
</tr>
<tr>
<td>34</td>
<td>Matched_Norm_Sample_UUID</td>
<td>BCR aliquot UUID for matched normal</td>
<td>567e8487-e29b-32d4-a716-446655443246</td>
<td>Yes</td>
<td>No</td>
<td></td>
</tr>
</tbody>
</table>
<p><strong>Notes</strong><br/>
<em>1 Intergenic Region.</em> <br/>
<em>2 Consolidationd is used to indicate a site that was initially reported as a variant but subsequently removed from further analysis because it was consolidated into a new variant. For example, a SNP variant incorporated into a TNP variant.</em> <br/>
<em>3 Used when the discovered varieant differs from that of dbSNP.</em><br/>
<em>4 These MAF headers describe the technology that was used to confirm a mutation, whether the same technology ("verification") or a different technology ("validation") is used to prove that a variant is germline or a somatic mutation.</em><br/>
<em>5 These MAF headers describe the technology that was used toconfirm a mutation, whether the same technology (verification) or a different technology (validation) is used to prove that a variant is germline or a somatic mutation.</em><br/>
<em>6 Explanation of some Validation Status-Mutation Status combinations.</em><br/></p>
<table class='table'>
<thead>
<tr>
<th>Validation Status</th>
<th>Mutation Status</th>
<th>Explanation</th>
</tr>
</thead>
<tbody>
<tr>
<td>Valid</td>
<td>Unknown</td>
<td>a valid variant with unknown somatic status due to lack of data from matched normal tissue.</td>
</tr>
<tr>
<td>Invalid</td>
<td>None</td>
<td>validation attempted, tumor and normal are homozygous reference (formerly described as Wildtype)</td>
</tr>
<tr>
<td>Inconclusive</td>
<td>Unknown</td>
<td>validation failed, neither the genotype nor its somatic status is certain due to lack of data from matched normal tissue</td>
</tr>
<tr>
<td>Inconclusive</td>
<td>None</td>
<td>validation failed, tumor genotype appears to be homozygous reference</td>
</tr>
</tbody>
</table>
<pre><code>Important Criteria

**Index column indicates the order in which the columns are expected**. **All
headers are case sensitive.** The Case Sensitive column specifies which values
are case sensitive. The Null column indicates which MAF columns are allowed to
have null values. The Enumerated column indicates which MAF columns have
specified values: an Enumerated value of "No" indicates that there are no
specified values for that column; other values indicate the specific values
listed allowed; a value of "Set" indicates that the MAF column values come from
a specified set of known values (*e.g.*HUGO gene symbols).
</code></pre>
<h1><i class="fas fa-book"></i>   MAF file checks</h1><hr>
<p>The DCC Archive Validator checks the integrity of a MAF file. Validation will
fail if any of the below are not true for a MAF file:</p>
<ol>
<li>
<p>Column header text (including case) and order must match specification
    (Table 1) exactly</p>
</li>
<li>
<p>Values under column headers listed in the specification (Table 1) as not
    null must have values</p>
</li>
<li>
<p>Values that are specified in Table 1 as Case Sensitive must be.</p>
</li>
<li>
<p>If column headers are listed in the specification as having <em>enumerated</em>
    values (<em>i.e.</em> a "Yes" in the "Enumerated" column), then the values under
    those column must come from the enumerated values listed under "Enumerated".</p>
</li>
<li>
<p>If column headers are listed in the specification as having <em>set</em> values
    (<em>i.e.</em> a "Set" in the "Enumerated" column), then the values under those
    column must come from the enumerated values of that domain (<em>e.g.</em> HUGO gene
    symbols).</p>
</li>
<li>
<p>All Allele-based columns must contain- (deletion), or a string composed of
    the following capitalized letters: A, T, G, C.</p>
</li>
<li>
<p>IfValidation_Status== "Untested"
    thenTumor_Validation_Allele1,Tumor_Validation_Allele2,Match_Norm_Validation_Allele1,Match_Norm_Validation_Allele2can
    be null (depending onValidation_Status).</p>
<ol>
<li>IfValidation_Status== "Inconclusive"
    thenTumor_Validation_Allele1,Tumor_Validation_Allele2,Match_Norm_Validation_Allele1,Match_Norm_Validation_Allele2can
    be null (depending onValidation_Status)<strong>.</strong></li>
</ol>
</li>
<li>
<p>If Validation_Status == Valid, then Validated_Tumor_Allele1 and
    Validated_Tumor_Allele2must be populated (one of A, C, G, T, and -)</p>
<ol>
<li>
<p>If Validation_Status == "Valid" then Tumor_Validation_Allele1,
    Tumor_Validation_Allele2, Match_Norm_Validation_Allele1,
    Match_Norm_Validation_Allele2 cannot be null</p>
</li>
<li>
<p>IfValidation_Status== "Invalid"
    thenTumor_Validation_Allele1,Tumor_Validation_Allele2,Match_Norm_Validation_Allele1,Match_Norm_Validation_Allele2cannot
    be null AND Tumor_Validation_Allelle1 ==
    Match_Norm_Validation_Allele1AND Tumor_Validation_Allelle2 ==
    Match_Norm_Validation_Allele2 (Added as a replacement for 8a as a
    result of breakdown)</p>
</li>
</ol>
</li>
<li>
<p>Check allele values against Mutation_Status:<br />
    Check allele values against Validation_status:</p>
<ol>
<li>
<p>If Mutation_Status == "Germline" and Validation_Status == "Valid", then
    Tumor_Validation_Allele1 == Match_Norm_Validation_Allele1 and
    Tumor_Validation_Allele2 == Match_Norm_Validation_Allele2.</p>
</li>
<li>
<p>If Mutation_Status == "Somatic" and Validation_Status == "Valid", then
    Match_Norm_Validation_Allele1 == Match_Norm_Validation_Allele2 ==
    Reference_Allele and (Tumor_Validation_Allele1 or
    Tumor_Validation_Allele2) != Reference_Allele</p>
</li>
<li>
<p>If Mutation_Status == "LOH" and Validation_Status=="Valid", then
    Tumor_Validation_Allele1 == Tumor_Validation_Allele2 and
    Match_Norm_Validation_Allele1 != Match_Norm_Validation_Allele2 and
    Tumor_Validation_Allele1 == (Match_Norm_Validation_Allele1 or
    Match_Norm_Validation_Allele2).</p>
</li>
</ol>
</li>
<li>
<p>Check that Start_position \&lt;= End_position</p>
</li>
<li>
<p>Check for the Start_position and End_position against Variant_Type:</p>
<ol>
<li>
<p>If Variant_Type == "INS", then (End_position - Start_position + 1 ==
    length (Reference_Allele) or End_position - Start_position == 1) and
    length(Reference_Allele) \&lt;= length(Tumor_Seq_Allele1 and
    Tumor_Seq_Allele2)</p>
</li>
<li>
<p>If Variant_Type == "DEL", then End_position - Start_position + 1 ==
    length (Reference_Allele), then length(Reference_Allele) >=
    length(Tumor_Seq_Allele1 and Tumor_Seq_Allele2)</p>
</li>
<li>
<p>If Variant_Type == "SNP", then length(Reference_Allele and
    Tumor_Seq_Allele1 and Tumor_Seq_Allele2) == 1 and (Reference_Allele and
    Tumor_Seq_Allele1 and Tumor_Seq_Allele2) != "-"</p>
</li>
<li>
<p>If Variant_Type == "DNP", then length(Reference_Allele and
    Tumor_Seq_Allele1 and Tumor_Seq_Allele2) == 2 and (Reference_Allele and
    Tumor_Seq_Allele1 and Tumor_Seq_Allele2) !contain "-"</p>
</li>
<li>
<p>If Variant_Type == "TNP", then length(Reference_Allele and
    Tumor_Seq_Allele1 and Tumor_Seq_Allele2) == 3 and (Reference_Allele and
    Tumor_Seq_Allele1 and Tumor_Seq_Allele2) !contain "-"</p>
</li>
<li>
<p>If Variant_Type == "ONP", then length(Reference_Allele) ==
    length(Tumor_Seq_Allele1) == length(Tumor_Seq_Allele2) > 3 and
    (Reference_Allele and Tumor_Seq_Allele1 and Tumor_Seq_Allele2) !contain
    "-"</p>
</li>
</ol>
</li>
<li>
<p>Validation for UUID-based files:</p>
<ol>
<li>
<p>Column #33 must be Tumor_Sample_UUID containing UUID of the BCR aliquot
    for tumor sample</p>
</li>
<li>
<p>Column #34 must be Matched_Norm_Sample_UUID containing UUID of the BCR
    aliquot for matched normal sample</p>
</li>
<li>
<p>Metadata represented by Tumor_Sample_Barcode and
    Matched_Norm_Sample_Barcode should correspond to the UUIDs assigned to
    Tumor_Sample_UUID and Matched_Norm_Sample_UUID respectively</p>
</li>
</ol>
</li>
<li>
<p>If Validation_Status == "Valid" or "Invalid", then Validation_Method !=
    "none" (case insensitive) .</p>
</li>
</ol>
<h1><i class="fas fa-book"></i>   MAF naming convention</h1><hr>
<p>In archives uploaded to the DCC, the MAF file name should relate to the
containing archive name in the following way:</p>
<p>If the archive has the name</p>
<pre><code>\&lt;domain\&gt;_\&lt;disease_abbrev\&gt;.\&lt;platform\&gt;.Level_2.\&lt;serial_index\&gt;.\&lt;revision\&gt;.0.tar.gz
</code></pre>
<p>then a somatic MAF file with the archive should be named according to</p>
<pre><code>\&lt;domain\&gt;_\&lt;disease_abbrev\&gt;.\&lt;platform\&gt;.Level_2.\&lt;serial_index\&gt;[.\&lt;optional_tag\&gt;].somatic.maf
</code></pre>
<p>and a protected MAF with the archive should be named according to</p>
<pre><code>\&lt;domain\&gt;_\&lt;disease_abbrev\&gt;.\&lt;platform\&gt;.Level_2.\&lt;serial_index\&gt;[.\&lt;optional_tag\&gt;].protected.maf
</code></pre>
<p>The \&lt;optional_tag> may consist of alphanumeric characters, dash, and
underscore; no spaces or periods; or it may be left out altogether. The purpose
of the optional tag is to impart some brief annotation.</p>
<p><em>Example</em></p>
<p>For the archive</p>
<pre><code>genome.wustl.edu_OV.IlluminaGA_DNASeq.Level_2.7.6.0.tar.gz
</code></pre>
<p>the following are examples of valid maf names</p>
<pre><code>genome.wustl.edu_OV.IlluminaGA_DNASeq.Level_2.7.somatic.maf
genome.wustl.edu_OV.IlluminaGA_DNASeq.Level_2.7.protected.maf
</code></pre>
        </div>
      </div>
    </div>

      <footer id="docs-footer">
        <div class="container-fluid footer-container">

      <div>
        <a href="https://portal.gdc.cancer.gov" target="_blank">Site Home</a>
        | <a href="http://www.cancer.gov/global/web/policies" target="_blank">Policies</a>
        | <a href="http://www.cancer.gov/global/web/policies/accessibility" target="_blank">Accessibility</a>
        | <a href="http://www.cancer.gov/global/web/policies/foia" target="_blank">FOIA</a>
      </div>
      <div>
        <a href="http://www.hhs.gov" target="_blank">U.S. Department of Health and Human Services</a>
        | <a href="http://www.nih.gov" target="_blank">National Institutes of Health</a>
        | <a href="http://www.cancer.gov" target="_blank">National Cancer Institute</a>
        | <a href="http://www.usa.gov" target="_blank">USA.gov</a>
      </div>
      <div>
        NIH... Turning Discovery Into Health Â®
      </div>

      <div class="version-caption">
        GDC Docs Version 2.0
      </div>
    </div>
    </footer>
    



    
<script>
/* When the user clicks on the button, 
toggle between hiding and showing the dropdown content */

// Close the dropdown if the user clicks outside of it

var toggler = document.getElementById("myToggle");
toggler.onclick= function(){
    var myDropdown = document.getElementById("myDropdown");
    myDropdown.classList.toggle('show');
}
window.onclick = function(e) {
    if (!e.target.matches('.dropbtn')) {
        var myDropdown = document.getElementById("myDropdown");
    if (myDropdown.classList.contains('show')) {
        myDropdown.classList.remove('show');
    }
  }
}

</script>

    <!-- JavaScript Bundle with Popper -->
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>      
</body>
</html>