<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Release Notes</title>
    <link href="/vendor/fontawesome/css/all.min.css" rel="stylesheet" />
    <!-- CSS only -->
    <link
      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
      rel="stylesheet"
      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
      crossorigin="anonymous"
    />
    <link
      rel="stylesheet"
      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"
      integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
      crossorigin="anonymous"
    />
    <link rel="preconnect" href="https://fonts.googleapis.com" />
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
    <link
      href="https://fonts.googleapis.com/css2?family=EB+Garamond:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&family=Source+Sans+Pro:ital,wght@0,300;0,400;0,600;0,700;1,300;1,400;1,600;1,700&display=swap"
      rel="stylesheet"
    />

    <!-- Get locas CSS files -->
    
    <link rel="stylesheet" type="text/css" href="/home/jacob/Work/Docsite-Revamp/css/styles.css" />
    
    <link rel="stylesheet" type="text/css" href="/home/jacob/Work/Docsite-Revamp/css/docs.css" />
    
    <link rel="stylesheet" type="text/css" href="/home/jacob/Work/Docsite-Revamp/css/encyclopedia.css" />
    
  </head>
  <body>
    <!---->
    <nav class="navbar navbar-expand-md navbar-light bg-light">
      <div class="container-fluid">
        <button
          class="navbar-toggler"
          type="button"
          data-bs-toggle="collapse"
          data-bs-target="#navbarNavDropdown"
          aria-controls="navbarNavDropdown"
          aria-expanded="false"
          aria-label="Toggle navigation"
        >
          <span class="navbar-toggler-icon"></span>
        </button>
        <!-- Collapsed navigation -->
        <div class="my-container">
          <!-- Main title -->
          <a class="navbar-brand float-start" href="/.">
            <img
              class="brand-image"
              src="/home/jacob/Work/Docsite-Revamp/img/nih-header-logo-color.png"
              alt="NIH National Cancer
            Institute GDC Documentation"
            />
            <span class="hidden"
              >NIH National Cancer Institute GDC Documentation</span
            >
          </a>
          <!-- Expander button -->
          <button
            type="button"
            class="navbar-toggle"
            data-toggle="collapse"
            data-target=".navbar-collapse"
          >
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>

          <!-- GDC Apps -->
          <div class="pull-right">
            <div class="gdc-app-list dropbtn">
              <div class="dropbtn">
                <a id="myToggle" class="dropbtn dropdown-toggle">
                  <span class="dropbtn"
                    ><i class="fas fa-th"></i> GDC Apps</span
                  >
                </a>
                <div id="myDropdown" class="dropdown-content container">
                  <div class="row gdc-app-row">
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                  </div>
                  <div class="row gdc-app-row">
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                  </div>
                  <div class="row gdc-app-row">
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                    <div class="col-sm gdc-app-list">
                      <i class="fas fa-lg fa-heartbeat"></i>
                    </div>
                  </div>
                </div>
              </div>
              <div
                data-dropdown-menu=""
                class="dropdown-menu gdc-apps-menu-container"
                role="menu"
              >
                <div class="gdc-apps-menu-grid">
                  <div class="menu-column">
                    <a
                      href="https://portal.gdc.cancer.gov/"
                      title="Data Portal"
                      target="_blank"
                      class="menu-item"
                    >
                      <img
                        width="21px"
                        src="/img/GDC-App-data-portal-blue.svg"
                        class="icon icon-gdc-portal home"
                        alt="GDC Data Portal"
                      />
                      <div>Data Portal</div>
                    </a>
                    <a
                      href="https://gdc.cancer.gov/developers/gdc-application-programming-interface-api"
                      title="GDC API"
                      target="_blank"
                      class="menu-item"
                    >
                      <span class="icon icon-gdc-portal-api">
                        <span class="path1"></span><span class="path2"></span
                        ><span class="path3"></span><span class="path4"></span
                        ><span class="path5"></span><span class="path6"></span
                        ><span class="path7"></span><span class="path8"></span
                        ><span class="path9"></span><span class="path10"></span>
                      </span>
                      <div>API</div>
                    </a>
                    <a
                      href="https://docs.gdc.cancer.gov/"
                      title="GDC Docs"
                      target="_blank"
                      class="active menu-item"
                    >
                      <span class="icon icon-gdc-docs">
                        <span class="path1"></span><span class="path2"></span
                        ><span class="path3"></span><span class="path4"></span
                        ><span class="path5"></span><span class="path6"></span
                        ><span class="path7"></span><span class="path8"></span
                        ><span class="path9"></span><span class="path10"></span
                        ><span class="path11"></span><span class="path12"></span
                        ><span class="path13"></span
                        ><span class="path14"></span>
                      </span>
                      <div>Documentation</div>
                    </a>
                    <a
                      href="https://portal.gdc.cancer.gov/legacy-archive"
                      title="GDC Legacy Archive"
                      target="_blank"
                      class="menu-item"
                    >
                      <span class="icon icon-gdc-legacy-archive">
                        <span class="path1"></span><span class="path2"></span
                        ><span class="path3"></span><span class="path4"></span
                        ><span class="path5"></span><span class="path6"></span
                        ><span class="path7"></span><span class="path8"></span
                        ><span class="path9"></span><span class="path10"></span>
                      </span>
                      <div>Legacy Archive</div>
                    </a>
                  </div>
                  <div class="menu-column">
                    <a
                      href="https://gdc.cancer.gov/"
                      title="GDC Website"
                      target="_blank"
                      class="menu-item"
                    >
                      <img
                        width="21px"
                        src="/img/GDC-App-website-blue.svg"
                        class="icon icon-gdc-portal home"
                        alt="GDC Website"
                      />
                      <div>Website</div>
                    </a>
                    <a
                      href="https://gdc.cancer.gov/access-data/gdc-data-transfer-tool"
                      class="menu-item"
                      title="GDC Data Transfer Tool"
                      target="_blank"
                    >
                      <span class="icon icon-gdc-data-transer-tool">
                        <span class="path1"></span><span class="path2"></span
                        ><span class="path3"></span><span class="path4"></span
                        ><span class="path5"></span><span class="path6"></span
                        ><span class="path7"></span><span class="path8"></span>
                      </span>
                      <div>Data Transfer Tool</div>
                    </a>
                    <a
                      href="https://portal.gdc.cancer.gov/submission/"
                      title="GDC Submission Portal"
                      target="_blank"
                      class="menu-item"
                    >
                      <span class="icon icon-gdc-submission-portal">
                        <span class="path1"></span><span class="path2"></span
                        ><span class="path3"></span><span class="path4"></span
                        ><span class="path5"></span><span class="path6"></span
                        ><span class="path7"></span><span class="path8"></span
                        ><span class="path9"></span><span class="path10"></span>
                      </span>
                      <div>Data Submission Portal</div>
                    </a>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div class="container-fluid">
            <div class="collapse navbar-collapse" id="navbarNavDropdown">
              <ul class="navbar-nav navbar-center me-auto">
                <li class="nav-item">
                  <a class="nav-link" href="/">Home</a>
                </li>
                <li class="nav-item dropdown">
                  <a
                    aria-current="page"
                    href="/output/dist/api-getting-started/"
                    class="nav-link dropdown-toggle"
                    id="navbarDropdownMenuLink"
                    role="button"
                    data-bs-toggle="dropdown"
                    aria-expanded="false"
                    >API
                  </a>

                  <ul
                    class="dropdown-menu"
                    aria-labelledby="navbarDropdownMenuLink"
                  >
                     
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Getting_Started.html">Getting Started</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Search_and_Retrieval.html">Search and Retrieval</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Downloading_Files.html">Downloading Files</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Data_Analysis.html">Data Analysis</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/BAM_Slicing.html">BAM Slicing</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Submission.html">Submission</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Python_Examples.html">Python Examples</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/GraphQL_Examples.html">GraphQL Examples</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/System_Information.html">System Information</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Additional_Examples.html">Additional Examples</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Appendix_A_Available_Fields.html">Appendix A: Available Fields</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Appendix_B_Key_Terms.html">Appendix B: Key Terms</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Users_Guide/Appendix_C_Format_of_Submission_Requests_and_Responses.html">Appendix C: Format of Submission Queries and Responses</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/Release_Notes/API_Release_Notes.html">Release Notes</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/API/PDF/index.html">fa-file-text Download PDF /API/PDF/API_UG.pdf</a>
                    </li>
                     
                  </ul>
                </li>
                <li class="nav-item dropdown">
                  <a
                    class="nav-link dropdown-toggle"
                    id="navbarDropdownMenuLink"
                    role="button"
                    data-bs-toggle="dropdown"
                    aria-expanded="false"
                    href="/output/dist/data-submission-portal-data-submission-process/"
                  >
                    Data Submission
                  </a>

                  <ul
                    class="dropdown-menu"
                    aria-labelledby="navbarDropdownMenuLink"
                  >
                     
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Checklist.html">Before Submitting Data to the GDC Portal</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Data_Submission_Overview.html">Data Submission Overview</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Data_Submission_Process.html">Data Submission Portal</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Data_Submission_Walkthrough.html">Data Upload Walkthrough</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Pre_Release_QC.html">Pre-Release Data Portal</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Users_Guide/Best_Practices.html">Submission Best Practices</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/Release_Notes/Data_Submission_Portal_Release_Notes.html">Release Notes</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Submission_Portal/PDF/index.html">fa-file-text Download PDF /Data_Submission_Portal/PDF/Data_Submission_Portal_UG.pdf</a>
                    </li>
                     
                  </ul>
                </li>
                <li class="nav-item dropdown">
                  <a
                    class="nav-link dropdown-toggle"
                    href="/output/dist/data-transfer-tool-index/"
                    id="navbarDropdownMenuLink"
                    role="button"
                    data-bs-toggle="dropdown"
                    aria-expanded="false"
                  >
                    Data Transfer Tool
                  </a>

                  <ul
                    class="dropdown-menu"
                    aria-labelledby="navbarDropdownMenuLink"
                  >
                     
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Users_Guide/Getting_Started.html">Getting Started</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Users_Guide/Preparing_for_Data_Download_and_Upload.html">Preparing for Data Download and Upload</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Users_Guide/Data_Download_and_Upload.html">Data Transfer Tool Command Line Documentation</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Release_Notes/DTT_Release_Notes.html">Release Notes - Command Line</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Users_Guide/Data_Download_DTT_UI.html">Data Transfer Tool UI Documentation</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Release_Notes/DTT_UI_Release_Notes.html">Release Notes - UI</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/Users_Guide/Appendix_B_TroubleShooting.html">Troubleshooting Guide</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Transfer_Tool/PDF/index.html">fa-file-text Download PDF /Data_Transfer_Tool/PDF/Data_Transfer_Tool_UG.pdf</a>
                    </li>
                     
                  </ul>
                </li>
                <li class="nav-item dropdown">
                  <a
                    href="/output/dist/data-dictionary-index/"
                    class="nav-link dropdown-toggle"
                    id="navbarDropdownMenuLink"
                    role="button"
                    data-bs-toggle="dropdown"
                    aria-expanded="false"
                  >
                    Data Dictionary
                  </a>

                  <ul
                    class="dropdown-menu"
                    aria-labelledby="navbarDropdownMenuLink"
                  >
                     
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Dictionary/index.html">About</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Dictionary/viewer.html">Viewer</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Dictionary/gdcmvs.html">Search</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data_Dictionary/Release_Notes/Data_Dictionary_Release_Notes.html">Release Notes</a>
                    </li>
                     
                  </ul>
                </li>
                <li class="nav-item dropdown">
                  <a
                    class="nav-link dropdown-toggle"
                    id="navbarDropdownMenuLink"
                    role="button"
                    data-bs-toggle="dropdown"
                    aria-expanded="false"
                    href="/output/dist/data-getting-started/"
                    >Data</a
                  >

                  <ul
                    class="dropdown-menu"
                    aria-labelledby="navbarDropdownMenuLink"
                  >
                     
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Introduction.html">Introduction</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Data_Model/GDC_Data_Model.html">GDC Data Model</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Data_Security/Data_Security.html">Data Security</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/File_Formats/MAF_Format.html">File Format: MAF</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/File_Formats/VCF_Format.html">File Format: VCF</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline.html">Bioinformatics Pipeline: DNA-Seq Analysis</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline.html">Bioinformatics Pipeline: mRNA Analysis</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/miRNA_Pipeline.html">Bioinformatics Pipeline: miRNA Analysis</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/CNV_Pipeline.html">Bioinformatics Pipeline: Copy Number Variation Analysis</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/Methylation_Pipeline.html">Bioinformatics Pipeline: Methylation Liftover Pipeline</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/RPPA_intro.html">Bioinformatics Pipeline: Protein Expression</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/Aligned_reads_summary_metrics.html">Aligned Reads Summary Metrics</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/Release_Notes/Data_Release_Notes.html">Release Notes</a>
                    </li>
                      
                    <li>
                      <a class="dropdown-item" href="/home/jacob/Work/Docsite-Revamp/output/Data/PDF/index.html">fa-file-text Download PDF /Data/PDF/Data_UG.pdf</a>
                    </li>
                     
                  </ul>
                </li>
                <li class="nav-item">
                  <a class="nav-link" href="/home/jacob/Work/Docsite-Revamp/output/Encyclopedia/index.html">
                    Encyclopedia
                  </a>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </div>
    </nav>

    <div class="row">
      <div class="col-md-3">
        <div class="container-fluid sidebar-container">
          <div class="sidebar">
            <ul class="main">
              <li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Introduction.html">Introduction</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Data_Model/GDC_Data_Model.html">GDC Data Model</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Data_Security/Data_Security.html">Data Security</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/File_Formats/MAF_Format.html">File Format: MAF</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/File_Formats/VCF_Format.html">File Format: VCF</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline.html">Bioinformatics Pipeline: DNA-Seq Analysis</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline.html">Bioinformatics Pipeline: mRNA Analysis</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/miRNA_Pipeline.html">Bioinformatics Pipeline: miRNA Analysis</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/CNV_Pipeline.html">Bioinformatics Pipeline: Copy Number Variation Analysis</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/Methylation_Pipeline.html">Bioinformatics Pipeline: Methylation Liftover Pipeline</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/RPPA_intro.html">Bioinformatics Pipeline: Protein Expression</a></li><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Bioinformatics_Pipelines/Aligned_reads_summary_metrics.html">Aligned Reads Summary Metrics</a></li><li class="main"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/Release_Notes/Data_Release_Notes.html">Release Notes</a></li><ul><li><a href='#Data Release 33.1'>Data Release 33.1</a></li><li><a href='#Data Release 33.0'>Data Release 33.0</a></li><li><a href='#Data Release 32.0'>Data Release 32.0</a></li><li><a href='#Data Release 31.0'>Data Release 31.0</a></li><li><a href='#Data Release 30.0'>Data Release 30.0</a></li><li><a href='#Data Release 29.0'>Data Release 29.0</a></li><li><a href='#Data Release 28.0'>Data Release 28.0</a></li><li><a href='#Data Release 27.0 Bug Fix'>Data Release 27.0 Bug Fix</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#Data Release 19.1'>Data Release 19.1</a></li><li><a href='#None'>None</a></li><li><a href='#Data Release 18.0'>Data Release 18.0</a></li><li><a href='#Data Release 17.1'>Data Release 17.1</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#Data Release 13.0'>Data Release 13.0</a></li><li><a href='#Data Release 12.0'>Data Release 12.0</a></li><li><a href='#Data Release 11.0'>Data Release 11.0</a></li><li><a href='#Data Release 10.1'>Data Release 10.1</a></li><li><a href='#None'>None</a></li><li><a href='#None'>None</a></li><li><a href='#Data Release 8.0'>Data Release 8.0</a></li><li><a href='#Data Release 7.0'>Data Release 7.0</a></li><li><a href='#Data Release 6.0'>Data Release 6.0</a></li><li><a href='#Data Release 5.0'>Data Release 5.0</a></li><li><a href='#Data Release 4.0'>Data Release 4.0</a></li><li><a href='#Data Release 3.0'>Data Release 3.0</a></li><li><a href='#Data Release 2.0'>Data Release 2.0</a></li><li><a href='#Initial Data Release (1.0)'>Initial Data Release (1.0)</a></li></ul><li class="inactive"><a href="/home/jacob/Work/Docsite-Revamp/output/Data/PDF/index.html">fa-file-text Download PDF /Data/PDF/Data_UG.pdf</a></li>
            </ul>
          </div>
        </div>
      </div>
      <div class="col-md-9">
        <div class="container-fluid doc-container"><h1><i class="fas fa-book"></i>   Data Release Notes</h1><hr/>
<table class="table">
<thead>
<tr>
<th>Version</th>
<th>Date</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="Data_Release_Notes.md#data-release-331">v33.1</a></td>
<td>May 31, 2022</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-330">v33.0</a></td>
<td>May 3, 2022</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-320">v32.0</a></td>
<td>March 29, 2022</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-310">v31.0</a></td>
<td>October 29, 2021</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-300">v30.0</a></td>
<td>September 23, 2021</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-290">v29.0</a></td>
<td>March 31, 2021</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-280">v28.0</a></td>
<td>February 2, 2021</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-270-bug-fix">v27.0-fix</a></td>
<td>November 9, 2020</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-270">v27.0</a></td>
<td>October 29, 2020</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-260">v26.0</a></td>
<td>September 8, 2020</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-250">v25.0</a></td>
<td>July 22, 2020</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-240">v24.0</a></td>
<td>May 7, 2020</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-230">v23.0</a></td>
<td>April 7, 2020</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-220">v22.0</a></td>
<td>January 16, 2020</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-210">v21.0</a></td>
<td>December 10, 2019</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-200">v20.0</a></td>
<td>November 11, 2019</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-191">v19.1</a></td>
<td>November 6, 2019</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-190">v19.0</a></td>
<td>September 17, 2019</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-180">v18.0</a></td>
<td>July 8, 2019</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-171">v17.1</a></td>
<td>June 12, 2019</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-170">v17.0</a></td>
<td>June 5, 2019</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-160">v16.0</a></td>
<td>March 26, 2019</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-150">v15.0</a></td>
<td>February 20, 2019</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-140">v14.0</a></td>
<td>December 18, 2018</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-130">v13.0</a></td>
<td>September 27, 2018</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-120">v12.0</a></td>
<td>June 13, 2018</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-110">v11.0</a></td>
<td>May 21, 2018</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-101">v10.1</a></td>
<td>February 15, 2018</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-100">v10.0</a></td>
<td>December 21, 2017</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-90">v9.0</a></td>
<td>October 24, 2017</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-80">v8.0</a></td>
<td>August 22, 2017</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-70">v7.0</a></td>
<td>June 29, 2017</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-60">v6.0</a></td>
<td>May 9, 2017</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-50">v5.0</a></td>
<td>March 16, 2017</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-40">v4.0</a></td>
<td>October 31, 2016</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-30">v3.0</a></td>
<td>September 16, 2016</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#data-release-20">v2.0</a></td>
<td>August 9, 2016</td>
</tr>
<tr>
<td><a href="Data_Release_Notes.md#initial-data-release-10">v1.0</a></td>
<td>June 6, 2016</td>
</tr>
</tbody>
</table>
<h2 id="Data Release 33.1">Data Release 33.1</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: May 31, 2022</li>
</ul>
<h3>New updates</h3>
<ul>
<li>None, see "Bugs Fixed Since Last Release" section below.</li>
</ul>
<p>A complete list of files for this release are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20220531_data_release_33.1_active.tsv.gz">gdc_manifest_20220531_data_release_33.1_active.tsv.gz</a></li>
</ul>
<h3>Unavailable Files</h3>
<ul>
<li>None</li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>32 cases from the EXCEPTIONAL_RESPONDERS-ER project were released as they were missing from the previous release.</li>
<li>All mutations from EXCEPTIONAL_RESPONDERS-ER in the exploration portal come from WXS data, whereas they were previously a mixture of WXS and Targeted Sequencing.</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Pathology reports do not have any associated case/biospecimen information in the portal. This information can be found in the reports themselves. <!--SV-2118--> </li>
<li>397 alignments from the TCGA program were found to have contamination values over 0.04 (<a href="Contaminated_Alignments.dr32.tsv">alignment list</a>). The ensemble MAFs produced by these alignments were removed from the Data Portal.</li>
<li>One methylation aliquot from the TCGA-COAD project, TCGA-D5-6930-01A-11D-1926-05, was not added to the portal and will be added in a future release.</li>
<li>The clinical supplement for TARGET-ALL-P1 is not currently available. It will be made available in a future release.</li>
<li>Copy number variations currently do not appear in the Exploration page.  This will be restored in a future release.</li>
<li>Mutations from SomaticSniper were erroneously labelled as LOH (loss of heterozygosity). This affects the VCF files, MAF files, and may cause SomaticSniper mutations to be absent from ensemble MAFs.</li>
<li>The slide image viewer does not display properly for 14 slides, which are identified <a href="missing_tiling.txt">here</a>.  The full slide image can be downloaded as an SVS file.</li>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release. <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="Data Release 33.0">Data Release 33.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: May 3, 2022</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New Project: NCI Exceptional Responders Initiative (EXCEPTIONAL_RESPONDERS-ER, phs001145)<ul>
<li>RNA-Seq - 45 Cases</li>
<li>WXS - 50 Cases</li>
<li>Targeted Sequencing - 41 Cases</li>
<li>Mutations from WXS and Targeted Sequencing are present in the exploration page.</li>
</ul>
</li>
<li>New Project: Molecular Profiling to Predict Response to Treatment - Wilms Tumor (MP2PRT-WT, phs001965)<ul>
<li>WGS - 52 Cases</li>
<li>RNA-Seq - 52 Cases</li>
<li>miRNA-Seq - 52 Cases</li>
</ul>
</li>
<li>Methylation files from the SeSAMe pipeline are now available for CGCI-HTMCP-CC and the TARGET projects.</li>
</ol>
<p>A complete list of files for this release are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20220503_data_release_33.0_active.tsv.gz">gdc_manifest_20220503_data_release_33.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20220503_data_release_33.0_legacy.tsv.gz">gdc_manifest_20220503_data_release_33.0_legacy.tsv.gz</a></li>
</ul>
<h3>Unavailable Files</h3>
<ul>
<li>The Arriba pipeline failed for one aliquot from EXCEPTIONAL-RESPONDERS-ER and is documented <a href="Removed_Aliquots.tsv">here</a>.</li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>Gene-level copy number files from TCGA-THCA and TCGA-UCEC were set as controlled-access files.  These have been corrected to be available as open-access files.</li>
<li>Due to a problem with the columns generated by the pipeline, all scRNA-Seq files have been replaced with a new version.</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>397 alignments from the TCGA program were found to have contamination values over 0.04 (<a href="Contaminated_Alignments.dr32.tsv">alignment list</a>). The ensemble MAFs produced by these alignments were removed from the Data Portal.</li>
<li>One methylation aliquot from the TCGA-COAD project, TCGA-D5-6930-01A-11D-1926-05, was not added to the portal and will be added in a future release.</li>
<li>The clinical supplement for TARGET-ALL-P1 is not currently available. It will be made available in a future release.</li>
<li>Copy number variations currently do not appear in the Exploration page.  This will be restored in a future release.</li>
<li>Mutations from SomaticSniper were erroneously labelled as LOH (loss of heterozygosity). This affects the VCF files, MAF files, and may cause SomaticSniper mutations to be absent from ensemble MAFs.</li>
<li>The slide image viewer does not display properly for 14 slides, which are identified <a href="missing_tiling.txt">here</a>.  The full slide image can be downloaded as an SVS file.</li>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release. <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="Data Release 32.0">Data Release 32.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data - GENCODE v36 Release</li>
<li><strong>Release Date</strong>: March 29, 2022</li>
</ul>
<h3>New updates</h3>
<h4>New data files</h4>
<ol>
<li>The following data types have been replaced with new GENCODE v36 versions<ul>
<li>RNA-Seq: all files, including alignments, gene expression files, and transcript fusion files.</li>
<li>WXS and Targeted Sequencing: annotated VCFs, single-caller MAFs, Ensemble MAFs.</li>
<li>WGS: BEDPE-format structural variants and gene-level copy number variants.</li>
<li>GENIE Targeted Sequencing files.</li>
<li>FM-AD Targeted Sequencing files.<ul>
<li>The primary-site-level FM-AD MAF files have been replaced with aliquot-level MAF files.</li>
</ul>
</li>
</ul>
</li>
<li>RNA-Seq STAR-Counts files now contain additional normalized counts such as FPKM, FPKM-UQ, and TPM.</li>
<li>All WXS files for TCGA have been replaced with new versions. Alignments will contain QC metrics and variants were produced using the same pipelines as all other GDC projects.</li>
<li>TCGA RNA-Seq has been changed to contain three alignments (genomic, transcriptome, and chimeric), STAR-counts files, and transcript fusion files for each aliquot.</li>
<li>The project-level MAFs in TCGA and FM-AD have been replaced with aliquot-level MAFs.</li>
<li>GENCODE v22 derived files (not BAM) that no longer appear in the portal will be downloadable as previous versions of v36 files.  </li>
<li>Methylation data produced from the SeSAMe pipeline is now available for all TCGA projects.</li>
<li>Note that miRNA-Seq data remains unchanged. The miRNA-Seq pipeline uses the miRBase database, which is not affected by the GENCODE version change.</li>
<li>A set of manifests were generated at the project-level that map each v22 file to its corresponding v36 file.  These can be used to help users transition from v22 to v36 and can be downloaded <a href="https://github.com/NCI-GDC/gdc-docs/tree/develop/docs/Data/Release_Notes/GCv36_Manifests">here</a>.</li>
</ol>
<h4>Removed data files and pipelines</h4>
<ol>
<li>Files from the HTSeq pipeline are no longer supported and will no longer appear in the portal. Normalized counts can now be found in the STAR-Counts files.  </li>
<li>Files that originated from the methylation liftover pipeline are no longer supported and will no longer appear in the portal.</li>
<li>GENCODE v22 BAM files that no longer appear in the portal will be available for six months past this release. They may not be available after that.</li>
<li>New variant calling tumor-normal pairing was implemented in TCGA, which results in certain aliquots no longer being available as a v36 version (see the aliquots labeled "Unpaired Aliquots" <a href="Removed_Aliquots.tsv">here</a>).</li>
<li>Some aliquots failed harmonization when the new v36 gene model was used, which results in some new versions no longer being available (see the aliquots labeled "Failed Harmonization" <a href="Removed_Aliquots.tsv">here</a>).</li>
<li>Some aliquots were found to contain a cross-patient contamination level of over 0.04 as measured by GATK4 CalculateContamination (see the aliquots labeled "Contamination" <a href="Removed_Aliquots.DR32.tsv">here</a>).  </li>
</ol>
<h4>Data Portal Exploration Data</h4>
<ol>
<li>The Data Portal Exploration Page is now populated based on open-access mutations from analyses that used GENCODE v36.</li>
<li>Mutations from SomaticSniper will not appear on the Exploration page.</li>
<li>Due to the copy number variation pipeline transition from GISTIC to ASCAT, the CNV data was not included in the GDC Exploration page. This will be replaced in a future release once visualization of the new pipeline is fully assessed.</li>
<li>The TCGA program mutations have been processed using the same pipeline as all other projects, which resulted in a 26% reduction in the number of open-access mutations. Some points on this change are listed below with TCGA-BRCA as the benchmark project:<ul>
<li>97% of the previously released open-access mutations are still discoverable in the new GDC controlled-access MAFs. This number increases to 99.95% when focusing only on mutations that were also called by <a href="https://gdc.cancer.gov/about-data/publications/pancanatlas">MC3</a>.</li>
<li>Somatic mutations will now be removed from the Data Portal Exploration Page unless they are detected by more than one variant calling software. This accounts for 40% of the total reduction.</li>
<li>Somatic mutations will now be removed from the Data Portal Exploration Page if they are detected outside of the target capture region, while previously out-of-target mutations detected from the TCGA Gene Annotation File (GAF) regions were allowed. This accounts for 36% of the total reduction.  </li>
<li>Some TCGA-specific variant-rescue steps have been removed in favor of a more robust and uniform filtering pipeline.</li>
<li>Some other minor changes due to updates in the gene model or other databases (e.g., the ExAC germline variant database was replaced with gnomAD in DR32).</li>
</ul>
</li>
</ol>
<p>A complete list of files for this release are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20220316_data_release_32.0_active.tsv.gz">gdc_manifest_20220316_data_release_32.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20220316_data_release_32.0_legacy.tsv.gz">gdc_manifest_20220316_data_release_32.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>397 alignments from the TCGA program were found to have contamination values over 0.04 (<a href="Contaminated_Alignments.dr32.tsv">alignment list</a>). The ensemble MAFs produced by these alignments were removed from the Data Portal.</li>
<li>One methylation aliquot from the TCGA-COAD project, TCGA-D5-6930-01A-11D-1926-05, was not added to the portal and will be added in a future release.</li>
<li>The clinical supplement for TARGET-ALL-P1 is not currently available. It will be made available in a future release.</li>
<li>Copy number variations currently do not appear in the Exploration page.  This will be restored in a future release.</li>
<li>Mutations from SomaticSniper were erroneously labelled as LOH (loss of heterozygosity). This affects the VCF files, MAF files, and may cause SomaticSniper mutations to be absent from ensemble MAFs.</li>
<li>The slide image viewer does not display properly for 14 slides, which are identified <a href="missing_tiling.txt">here</a>.  The full slide image can be downloaded as an SVS file.</li>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release. <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="Data Release 31.0">Data Release 31.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: October 29, 2021</li>
</ul>
<h3>New updates</h3>
<ol>
<li>TCGA Slide Images:<ul>
<li>All TCGA slide images that were removed earlier this year have been restored.</li>
<li>Note that the UUIDs for most TCGA slide images have changed.  Older manifest files may not work when downloading slide images.</li>
</ul>
</li>
<li>CPTAC-3 clinical data has been refreshed and includes new follow up entities.</li>
<li>REBC-THYR<ul>
<li>The clinical and biospecimen XML files were removed as they were not intended for release in DR 30.</li>
<li>The case REBC-ADL5 was added, which includes one WGS pair.</li>
</ul>
</li>
</ol>
<p>A complete list of files for this release are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20211029_data_release_31.0_active.tsv.gz">gdc_manifest_20211029_data_release_31.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20211029_data_release_31.0_legacy.tsv.gz">gdc_manifest_20211029_data_release_31.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>One file from a previous version of the methylation pipeline appeared in the data portal (bd2f864a-3f00-47b5-815d-bd01ca21ef61; CPTAC-3).  This file should no longer appear in the data portal. <!--SV-2012--></li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The slide image viewer does not display properly for 14 slides, which are identified <a href="missing_tiling.txt">here</a>.  The full slide image can be downloaded as an SVS file.</li>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="Data Release 30.0">Data Release 30.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: September 23, 2021</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New Projects:<ul>
<li>TRIO-CRU (phs001163) - Ukrainian National Research Center for Radiation Medicine Trio Study<ul>
<li>WGS Alignments</li>
</ul>
</li>
<li>REBC-THYR (phs001134) - Comprehensive genomic characterization of radiation-related papillary thyroid cancer in the Ukraine<ul>
<li>miRNA-Seq</li>
<li>RNA-Seq</li>
<li>WGS</li>
</ul>
</li>
</ul>
</li>
<li>CPTAC Program<ul>
<li>CPTAC-3 methylation data produced from the SeSAMe pipeline is now available.</li>
<li>CPTAC-2 miRNA-Seq files have been replaced with better quality data.</li>
</ul>
</li>
<li>HCMI-CMDC<ul>
<li>31 New cases have been released to the GDC Data Portal.</li>
<li>Methylation data produced from the SeSAMe pipeline is now available.</li>
</ul>
</li>
<li>TCGA<ul>
<li>Protein expression data (RPPA) is now available for 32 projects.</li>
<li>RNA-Seq data for TCGA-TGCT was replaced with files from an updated pipeline.</li>
</ul>
</li>
<li>TARGET-AML - New RNA-Seq and miRNA-Seq aliquots have been released.</li>
</ol>
<p>A complete list of files for this release are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20210923_data_release_30.0_active.tsv.gz">gdc_manifest_20210923_data_release_30.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20210923_data_release_30.0_legacy.tsv.gz">gdc_manifest_20210923_data_release_30.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>One file from a previous version of the methylation pipeline appears in the data portal (bd2f864a-3f00-47b5-815d-bd01ca21ef61; CPTAC-3).  This file cannot be downloaded, but may cause bulk downloads to fail.  Remove this file from any manifest or cart you plan on downloading. <!--SV-2012--></li>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="Data Release 29.0">Data Release 29.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: March 31, 2021</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Count Me In Program<ul>
<li>Aliquot-level MAFs are now available for projects CMI-ASC, CMI-MBC, and CMI-MPC.</li>
<li>Somatic mutation are now explorable for projects CMI-ASC, CMI-MBC, and CMI-MPC</li>
</ul>
</li>
<li>CPTAC Program<ul>
<li>CPTAC-2 open-access somatic mutations are now browsable through the GDC Exploration Portal.</li>
<li>MSI data is now browsable through the faceted search for CPTAC-2 and CPTAC-3.</li>
</ul>
</li>
<li>HCMI-CMDC - Data files and explorable mutations for 18 new cases are now available.</li>
</ol>
<p>A complete list of files for this release are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20210331_data_release_29.0_active.tsv.gz">gdc_manifest_20210331_data_release_29.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20210331_data_release_29.0_legacy.tsv.gz">gdc_manifest_20210331_data_release_29.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>The aggregated and masked MAF files that were missing for seven pancreatic cases in CPTAC-3 have been restored to the data portal.</li>
<li>The missing RNA-Seq data files for the seven normal pancreatic cases in CPTAC-3 have been restored to the data portal.</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>
<p>Issues in the Legacy Archive</p>
<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
<h2 id="Data Release 28.0">Data Release 28.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: February 2, 2021</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New Project: CMI-MPC - Count Me In - The Metastatic Prostate Cancer Project<ul>
<li>WXS alignments and variant calls (VCFs) are available.</li>
</ul>
</li>
<li>New Data Type: Single nuclei (snRNA-Seq) data is now available for 18 CPTAC-3 cases. See the <a href="https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/#scrna-seq-pipeline">RNA-Seq</a> documentation for details.</li>
<li>CPTAC-3<ul>
<li>Data files for 147 new cases from the pancreatic cohort are now available.</li>
<li>CPTAC-3 open-access somatic mutations are now browsable through the GDC Exploration Portal.</li>
<li>RNA-Seq transcript fusion files are now available.</li>
<li>Targeted Sequencing alignments and raw tumor-only variant calls (VCF) are now available.</li>
</ul>
</li>
<li>HCMI-CMDC<ul>
<li>Data files for 22 new cases are now available.</li>
<li>The HCMI-CMDC open-access somatic mutations have been refreshed on the GDC Exploration Portal to reflect all newly released cases.</li>
</ul>
</li>
</ol>
<p>A complete list of files for this release are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20210202_data_release_28.0_active.tsv.gz">gdc_manifest_20210202_data_release_28.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20210202_data_release_28.0_legacy.tsv.gz">gdc_manifest_20210202_data_release_28.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The aggregated and masked MAF files for seven pancreatic cases in CPTAC-3 do not appear in the Data Portal. See below for download instructions.<ul>
<li><a href="CPTAC-3_7Cases-WXS-MAFs_GDC-Manifest.txt">This manifest</a> can be used to download the files.  </li>
<li>To download the raw aggregated MAF files, dbGaP access to CPTAC-3 (phs001287) is required.  The masked MAF files are open-access.</li>
<li>The seven cases are as follows: C3L-04027, C3L-04080, C3N-02585, C3N-02768, C3N-02971, C3N-03754, and C3N-03839. The case the each file is associated with is denoted in the manifest.</li>
</ul>
</li>
<li>The RNA-Seq data files for the seven normal pancreatic cases in CPTAC-3 do not appear in the Data Portal. See below for download instructions.<ul>
<li><a href="CPTAC-3_7CasesRNASeq_GDC-Manifest.txt">This manifest</a> can be used to download the files.  </li>
<li>To download the alignments or splice-junction files, dbGaP access to CPTAC-3 (phs001287) is required.  The other gene expression files are open-access.</li>
<li>The seven cases are as follows: C3L-03513, C3L-07032, C3L-07033, C3L-07034, C3L-07035, C3L-07036, C3L-07037. The case the each file is associated with is denoted in the manifest.</li>
</ul>
</li>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="Data Release 27.0 Bug Fix">Data Release 27.0 Bug Fix</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: November 9, 2020</li>
</ul>
<h3>New updates</h3>
<ol>
<li>None, see bug fix section below.</li>
</ol>
<p>A complete list of files for this release are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20201109_data_release_27.0_active.tsv.gz">gdc_manifest_20201109_data_release_27.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20201109_data_release_27.0_legacy.tsv.gz">gdc_manifest_20201109_data_release_27.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>Some files in projects CGCI-BLGSP, CGCI-HTMCP-CC, and HCMI-CMDC were marked on the portal as controlled-access, when they were supposed to be open-access. These are now downloadable as open-access files.</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="None">Data Release 27.0 <!--REQ-413--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: October 29, 2020</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Initial release for the WGS variant calling pipeline. See the <a href="https://docs.gdc.cancer.gov/Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline/#whole-genome-sequencing-variant-calling">documentation</a> on WGS variant calling for more details on the available files.  This includes data from the following projects:<ul>
<li>CGCI-BLGSP <!--SPT7-75--></li>
<li>CGCI-HTMCP-CC <!--SPT7-27--></li>
<li>HCMI-CMDC <!--DAT-2948--></li>
</ul>
</li>
<li>RNA-Seq transcript fusion files are available for the following projects: <!--DAT-2947--><ul>
<li>CGCI-BLGSP</li>
<li>CGCI-HTMCP-CC</li>
<li>HCMI-CMDC</li>
</ul>
</li>
<li>Aliquot level MAFs were released for CGCI-HTMCP-CC Targeted Sequencing variants. Open access MAFs are included. <!--SPT7-27--></li>
<li>17 new cases were released for the HCMI-CMDC project.  This includes WGS, WXS, and RNA-Seq data. <!--DAT-2949--></li>
<li>WGS alignments were released for 99 TCGA-LUAD cases (196 files). <!--DAT-1831--></li>
<li>Therapeutic agents (treatment) and tumor stage (diagnosis) properties were migrated to remove deprecated values and better adhere to a standardized set of values. <!--DAT-2831--> <!--DAT-2880--></li>
</ol>
<p>A complete list of files for DR27.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20201029_data_release_27.0_active.tsv.gz">gdc_manifest_20201029_data_release_27.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20201029_data_release_27.0_legacy.tsv.gz">gdc_manifest_20201029_data_release_27.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Some files in projects CGCI-BLGSP, CGCI-HTMCP-CC, and HCMI-CMDC are marked on the portal as controlled-access. These files are publicly downloadable using the Data Transfer Tool or API. All files from the following data types should be open-access within the previously specified projects: Biospecimen Supplement, Clinical Supplement, Gene Expression Quantification, Masked Somatic Mutation</li>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="None">Data Release 26.0 <!--REQ-409--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: September 8, 2020</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New program released:<ul>
<li>Count Me In (CMI)<ul>
<li>CMI-ASC - The Angiosarcoma Project<ul>
<li>RNA-Seq</li>
<li>WXS</li>
</ul>
</li>
<li>CMI-MBC - The Metastatic Breast Cancer Project<ul>
<li>RNA-Seq</li>
<li>WXS</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
<li>Somatic mutations are now available on the exploration portal for the following projects:<ul>
<li>MMRF-COMMPASS</li>
<li>TARGET-ALL-P3</li>
<li>TARGET-AML</li>
<li>TARGET-NBL</li>
<li>TARGET-WT</li>
</ul>
</li>
<li>Primary sites and disease types were updated for multiple projects to correspond to GDC Dictionary updates.</li>
</ol>
<p>A complete list of files for DR26.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20200908_data_release_26.0_active.tsv.gz">gdc_manifest_20200908_data_release_26.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20200908_data_release_26.0_legacy.tsv.gz">gdc_manifest_20200908_data_release_26.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>The CPTAC-3 head and neck cohort can now be queried by choosing the head and neck anatomic site on the GDC home page.</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>
<p>Issues in the Legacy Archive</p>
<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
<h2 id="None">Data Release 25.0 <!--REQ-403--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: July 22, 2020</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New data types released:<ul>
<li>RNA-Seq Transcript Fusion files were released for the following projects:<ul>
<li>TARGET-ALL-P1</li>
<li>TARGET-ALL-P2</li>
<li>TARGET-ALL-P3</li>
<li>TARGET-CCSK</li>
<li>TARGET-NBL</li>
<li>TARGET-OS</li>
<li>TARGET-RT</li>
<li>TARGET-WT</li>
</ul>
</li>
<li>The msi_status and msi_score properties can be queried on the GDC Portal for the CPTAC-3 project.  <ul>
<li>To query for these fields: go to the <a href="https://portal.gdc.cancer.gov/repository">GDC Repository</a>, click on "Add a File Filter" at the top left of the screen, type msi_score or msi_status in the field, and click on "msi_score" or "msi_status".  This should bring up the corresponding filters to use on the portal.</li>
</ul>
</li>
</ul>
</li>
<li>108 cases from the CPTAC-3 LSCC Cohort were released. Includes the following data types:<ul>
<li>WXS</li>
<li>WGS</li>
<li>RNA-Seq</li>
<li>miRNA-Seq</li>
</ul>
</li>
<li>Aliquot level MAFs were released for MMRF-COMMPASS WXS variants. Open access MAFs are included.</li>
<li>HCMI-CMDC open-access somatic mutations were released to the <a href="https://portal.gdc.cancer.gov/exploration">Exploration Portal</a>.  </li>
</ol>
<p>A complete list of files for DR25.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20200722_data_release_25.0_active.tsv.gz">gdc_manifest_20200722_data_release_25.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20200722_data_release_25.0_legacy.tsv.gz">gdc_manifest_20200722_data_release_25.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>A few supplements from CGCI-BLGSP are now associated with their correct versions. <!--SV-1709--></li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Currently the CPTAC-3 HNSCC cohort does not appear when the "Head and Neck" primary site is selected from the GDC home page. This cohort can be queried by clicking <a href="https://portal.gdc.cancer.gov/repository?facetTab=cases&amp;filters=%7B%22op%22%3A%22and%22%2C%22content%22%3A%5B%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22cases.primary_site%22%2C%22value%22%3A%5B%22other%20and%20ill-defined%20sites%22%5D%7D%7D%2C%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22cases.project.project_id%22%2C%22value%22%3A%5B%22CPTAC-3%22%5D%7D%7D%5D%7D">here</a></li>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="None">Data Release 24.0 <!--REQ-401--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: May 7, 2020</li>
</ul>
<h3>New updates</h3>
<ol>
<li>
<p>New project released: CGCI-HTMCP-CC - HIV+ Tumor Molecular Characterization Project - Cervical Cancer</p>
<ul>
<li>RNA-Seq: Alignments and gene expression levels</li>
<li>miRNA-Seq: Alignments and miRNA expression levels</li>
<li>WGS: Alignments</li>
<li>Targeted Sequencing: Alignments</li>
</ul>
</li>
<li>
<p>110 new cases were released from the HNSCC cohort of CPTAC-3.  This includes WXS, WGS, RNA-Seq and miRNA-Seq data.</p>
</li>
<li>
<p>Aliquot-level WXS MAFs are now available from the following projects:</p>
<ul>
<li>CPTAC-2</li>
<li>CPTAC-3</li>
</ul>
</li>
</ol>
<p>A complete list of files for DR24.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20200507_data_release_24.0_active.tsv.gz">gdc_manifest_20200507_data_release_24.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20200507_data_release_24.0_legacy.tsv.gz">gdc_manifest_20200507_data_release_24.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Currently the CPTAC-3 HNSCC cohort does not appear when the "Head and Neck" primary site is selected from the GDC home page. This cohort can be queried by clicking <a href="https://portal.gdc.cancer.gov/repository?facetTab=cases&amp;filters=%7B%22op%22%3A%22and%22%2C%22content%22%3A%5B%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22cases.primary_site%22%2C%22value%22%3A%5B%22other%20and%20ill-defined%20sites%22%5D%7D%7D%2C%7B%22op%22%3A%22in%22%2C%22content%22%3A%7B%22field%22%3A%22cases.project.project_id%22%2C%22value%22%3A%5B%22CPTAC-3%22%5D%7D%7D%5D%7D">here</a></li>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="None">Data Release 23.0 <!--REQ-397--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: April 7, 2020</li>
</ul>
<h3>New updates</h3>
<ol>
<li>
<p>New data types released:</p>
<ul>
<li>Aliquot-level MAFs: MAF Files with mutations derived from one tumor/normal pair <!--DAT-2732--><ul>
<li>HCMI-CMDC</li>
<li>TARGET-ALL-P2</li>
<li>TARGET-ALL-P3</li>
<li>TARGET-AML</li>
<li>TARGET-NBL</li>
<li>TARGET-OS</li>
<li>TARGET-WT</li>
<li>Note: Previously released TARGET project level MAFs can be downloaded with the following manifest: <a href="TARGET_Project-Level-MAF_GDC-Manifest.txt">TARGET_Project-Level-MAF_GDC-Manifest.txt</a></li>
</ul>
</li>
<li>Copy number segment and estimate files from SNP6 ASCAT <!--BINF-293--><ul>
<li>All TCGA Projects</li>
<li>TARGET-ALL-P2</li>
<li>TARGET-AML</li>
</ul>
</li>
</ul>
</li>
<li>
<p>To accommodate users who prefer to use project-level MAFs, a MAF aggregation tool was developed by the GDC: <!--DEV-89--></p>
<ul>
<li><a href="https://github.com/NCI-GDC/gdc-maf-tool/releases">Github Release</a></li>
</ul>
</li>
<li>
<p>New RNA-Seq data was released from HCMI-CMDC for nine additional cases.  </p>
</li>
<li>
<p>Clinical updates were performed for the following projects</p>
<ul>
<li>CGCI-BLGSP</li>
<li>HCMI-CMDC <!--SV-1660--></li>
<li>WCDT-MCRPC <!--DAT-2782--></li>
</ul>
</li>
</ol>
<p>A complete list of files for DR23.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20200407_data_release_23.0_active.tsv.gz">gdc_manifest_20200407_data_release_23.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20200407_data_release_23.0_legacy.tsv.gz">gdc_manifest_20200407_data_release_23.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>The 6 HCMI-CMDC cases without clinical data now have clinical data. <!--SV-1660--></li>
<li>Most of the "associated_entities" fields in CGCI-BLGSP were not populated correct, this has been resolved.  <!--SV-1701--></li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="None">Data Release 22.0 <!--REQ-396--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: January 16, 2020</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New projects released:<ul>
<li>WCDT-MCRPC - Genomic Characterization of Metastatic Castration Resistant Prostate Cancer (phs001648)<ul>
<li>RNA-Seq; WGS Data <!--SPT7-4--> <!--SPT7-38--></li>
</ul>
</li>
</ul>
</li>
<li>New data from HCMI-CMDC <!--DAT-2758--> <!--SPT7-33--><ul>
<li>16 New Cases</li>
<li>Includes WXS, WGS, and RNA-Seq data</li>
</ul>
</li>
<li>New data from CPTAC-3 <!--DAT-2723--><ul>
<li>108 New Cases</li>
<li>Includes WXS, WGS, and RNA-Seq data</li>
<li>miRNA-Seq data for currently released cases</li>
</ul>
</li>
</ol>
<p>A complete list of files for DR22.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20200116_data_release_22.0_active.tsv.gz">gdc_manifest_20200116_data_release_22.0_active.tsv.gz</a></li>
<li><a href="gdc_manifest_20200116_data_release_22.0_legacy.tsv.gz">gdc_manifest_20200116_data_release_22.0_legacy.tsv.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>6 of the HCMI-CMDC cases are missing clinical nodes <!--SV-1660--><ul>
<li>HCM-CSHL-0060-C18</li>
<li>HCM-CSHL-0089-C25</li>
<li>HCM-CSHL-0090-C25</li>
<li>HCM-CSHL-0092-C25</li>
<li>HCM-CSHL-0091-C25</li>
<li>HCM-CSHL-0057-C18</li>
</ul>
</li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="None">Data Release 21.0 <!--REQ-396--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: December 10, 2019</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New projects released:<ul>
<li>GENIE - AACR Project Genomics Evidence Neoplasia Information Exchange (phs001337) <!--SPT-173--><ul>
<li>Includes Targeted Sequencing, Transcript Fusion, Copy Number Estimate from GENIE 5.0</li>
</ul>
</li>
<li>AACR Project GENIE is divided by sequencing center:<ul>
<li>GENIE-MSK</li>
<li>GENIE-DFCI</li>
<li>GENIE-MDA</li>
<li>GENIE-JHU</li>
<li>GENIE-UHN</li>
<li>GENIE-VICC</li>
<li>GENIE-GRCC</li>
<li>GENIE-NKI</li>
</ul>
</li>
</ul>
</li>
</ol>
<p>A complete list of files for DR21.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20191210_data_release_21.0_active.txt.gz">gdc_manifest_20191210_data_release_21.0_active.txt.gz</a></li>
<li><a href="gdc_manifest_20191210_data_release_21.0_legacy.txt.gz">gdc_manifest_20191210_data_release_21.0_legacy.txt.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The Copy Number Estimate files in GENIE are labeled on the portal as TXT while the files are actually in TSV format.  <!--DAT-2728--></li>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="None">Data Release 20.0 <!--REQ-392--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: November 11, 2019</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New projects released:<ul>
<li>CPTAC-2 - CPTAC Proteogenomic Confirmatory Study (phs000892) <!--SPT-825--><ul>
<li>Includes WXS, RNA-Seq, and miRNA-Seq</li>
</ul>
</li>
<li>OHSU-CNL - Genomic landscape of Neutrophilic Leukemias of Ambiguous Diagnosis (phs001799) <!--SPT-755--><ul>
<li>Includes WXS and RNA-Seq</li>
<li>No VCF files will be included at this time.  They will follow in a later release.</li>
</ul>
</li>
</ul>
</li>
<li>New TARGET data released<ul>
<li>TARGET-OS: WGS, WXS <!--SPT-394, SPT-226--></li>
<li>TARGET-NBL: WGS <!--SPT-412--></li>
<li>TARGET-AML: miRNA <!--SV-1518--></li>
</ul>
</li>
<li>CGCI-BLGSP miRNA-Seq released <!-- SPT-441--></li>
</ol>
<p>A complete list of files for DR20.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20191111_data_release_20.0_active.txt.gz">gdc_manifest_20191111_data_release_20.0_active.txt.gz</a></li>
<li><a href="gdc_manifest_20191111_data_release_20.0_legacy.txt.gz">gdc_manifest_20191111_data_release_20.0_legacy.txt.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="Data Release 19.1">Data Release 19.1</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: November 6, 2019</li>
</ul>
<h3>New updates</h3>
<ul>
<li>The following cases are no longer available in the GDC Data Portal.  They had no data files associated with them in DR 19 so there are no changes in file availability in this release.<ul>
<li>TARGET-00-NAAENF</li>
<li>TARGET-00-NAAENG</li>
<li>TARGET-00-NAAENH</li>
<li>TARGET-00-NAAENI</li>
<li>TARGET-00-NAAENJ</li>
<li>TARGET-00-NAAENK</li>
<li>TARGET-00-NAAENL</li>
<li>TARGET-00-NAAENM</li>
<li>TARGET-00-NAAENN</li>
<li>TARGET-00-NAAENP</li>
<li>TARGET-00-NAAENR</li>
<li>TARGET-00-NAAEPE</li>
</ul>
</li>
</ul>
<p>A complete list of files for DR19.1 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20190917_data_release_19.0_active.txt.gz">gdc_manifest_20190917_data_release_19.0_active.txt.gz</a></li>
<li><a href="gdc_manifest_20190917_data_release_19.0_legacy.txt.gz">gdc_manifest_20190917_data_release_19.0_legacy.txt.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="None">Data Release 19.0 <!--REQ-389--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: September 17, 2019</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New projects released:<ul>
<li>BEATAML1.0-COHORT - Functional Genomic Landscape of Acute Myeloid Leukemia (phs001657)<ul>
<li>Includes WXS and RNA-Seq</li>
</ul>
</li>
</ul>
</li>
<li>New TARGET data released<ul>
<li>TARGET-ALL-P1 RNA-Seq</li>
<li>TARGET-ALL-P2 RNA-Seq, WXS, and miRNA-Seq</li>
<li>TARGET-ALL-P3 miRNA-Seq</li>
<li>TARGET-AML WXS, WGS, and miRNA-Seq</li>
<li>TARGET-NBL WXS and RNA-Seq</li>
<li>TARGET-RT WGS and RNA-Seq</li>
<li>TARGET-WT WGS, WXS, and RNA-Seq</li>
</ul>
</li>
<li>Additional CGCI-BLGSP WGS data released</li>
<li>Pindel VCFs released for TARGET-ALL-P2, TARGET-ALL-P3, TARGET-AML, TARGET-NBL, TARGET-WT, MMRF-COMMPASS, HCMI-CMDC, and CPTAC-3</li>
<li>Disease-specific staging properties for many projects were released <!--DAT-2480,  DAT-2503--></li>
</ol>
<p>A complete list of files for DR19.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20190917_data_release_19.0_active.txt.gz">gdc_manifest_20190917_data_release_19.0_active.txt.gz</a></li>
<li><a href="gdc_manifest_20190917_data_release_19.0_legacy.txt.gz">gdc_manifest_20190917_data_release_19.0_legacy.txt.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="Data Release 18.0">Data Release 18.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: July 8, 2019</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New Projects released<ul>
<li>MMRF-COMMPASS - Multiple Myeloma CoMMpass Study (phs000748)<ul>
<li>Includes WGS, WXS, and RNA-Seq</li>
</ul>
</li>
<li>ORGANOID-PANCREATIC - Pancreas Cancer Organoid Profiling (phs001611)<ul>
<li>Includes WGS, WXS, and RNA-Seq</li>
</ul>
</li>
<li>TARGET-ALL-P1 - Acute Lymphoblastic Leukemia - Phase I (phs000218)<ul>
<li>Includes WGS</li>
</ul>
</li>
<li>TARGET-ALL-P2 - Acute Lymphoblastic Leukemia - Phase II (phs000218)<ul>
<li>Includes WGS</li>
</ul>
</li>
<li>CGCI-BLGSP - Burkitt Lymphoma Genome Sequencing Project (phs000235)<ul>
<li>Includes WGS and RNA-Seq</li>
</ul>
</li>
</ul>
</li>
<li>New versions of RNA-Seq data for TARGET-ALL-P3     </li>
<li>New RNA-Seq data for TARGET-CCSK</li>
<li>New RNA-Seq data for TARGET-OS</li>
</ol>
<p>A complete list of files for DR18.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20190708_data_release_18.0_active.txt.gz">gdc_manifest_20190708_data_release_18.0_active.txt.gz</a></li>
<li><a href="gdc_manifest_20190708_data_release_18.0_legacy.txt.gz">gdc_manifest_20190708_data_release_18.0_legacy.txt.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>New versions of RNA-Seq data for TARGET-ALL-P3 resolve issue with missing reads from BAM files.</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Some tumor-only annotated VCFs (not raw VCFs) could have a small proportion of variants that appear twice.  Tumor-only annotated VCFs can be identified by searching for workflow "GATK4 MuTect2 Annotation" <!--SV-1425--></li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.* Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="Data Release 17.1">Data Release 17.1</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: June 12, 2019</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Rebuilt indices for NCICCR-DLBCL and CTSP-DLBCL1.  Fewer files viewable in GDC Data Portal or API.</li>
</ol>
<p>A complete list of files for DR17.1 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20190612_data_release_17.1_active.txt.gz">gdc_manifest_20190612_data_release_17.1_active.txt.gz</a></li>
<li><a href="gdc_manifest_20190612_data_release_17.1_legacy.txt.gz">gdc_manifest_20190612_data_release_17.1_legacy.txt.gz</a></li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET ALL-P3 RNA-Seq results from DR14 are missing ~18% of reads.  Downsampling appears to be completely random and count files have a very high correlation (&gt;99.99%) with complete data. New versions of these files will be created that include the entire set of reads.</li>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="None">Data Release 17.0 <!--REQ-383--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: June 5, 2019</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New Projects released<ul>
<li>HCMI-CMDC - NCI Cancer Model Development for the Human Cancer Model Initiative (HCMI) (phs001486)</li>
<li>BEATAML1.0-CRENOLANIB - Clinical Resistance to Crenolanib in Acute Myeloid Leukemia Due to Diverse Molecular Mechanisms (phs001628)</li>
</ul>
</li>
<li>RNA-Seq data for NCICCR-DLBCL and CTSP-DLBCL1 are released</li>
<li>ATAC-Seq data for TCGA projects are released</li>
<li>CPTAC-3 RNA-Seq data are released</li>
<li>Clinical data updates for TCGA - to see parser code updates review <a href="https://docs.gdc.cancer.gov/API/Release_Notes/API_Release_Notes/#v1200">API v1.20 release notes</a></li>
<li>Clinical data updates for other projects to accommodate migration of vital_status, days_to_birth, and days_to_death from the Diagnosis to the Demographic node</li>
</ol>
<p>A complete list of files for DR17.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20190605_data_release_17.0_active.txt.gz">gdc_manifest_20190605_data_release_17.0_active.txt.gz</a></li>
<li><a href="gdc_manifest_20190605_data_release_17.0_legacy.txt.gz">gdc_manifest_20190605_data_release_17.0_legacy.txt.gz</a>.</li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>TCGA Projects<ul>
<li>Incorrect information about treatment may be included for patients within TCGA-HNSC and TCGA-LGG.  Please refer to the clinical XML for accurate information on treatment <!--DAT-2264, DAT-2265--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.<!--TT-602, DAT-1489--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
</li>
<li>TARGET projects<ul>
<li>TARGET ALL-P3 RNA-Seq results from DR14 are missing ~18% of reads.  Downsampling appears to be completely random and count files have a very high correlation (&gt;99.99%) with complete data. New versions of these files will be created that include the entire set of reads.</li>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>No data from TARGET-MDLS is available.</li>
</ul>
</li>
<li>Issues in the Legacy Archive<ul>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
</ul>
</li>
</ul>
<h2 id="None">Data Release 16.0 <!--REQ-385--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: March 26, 2019</li>
</ul>
<h3>New updates</h3>
<ol>
<li>The CPTAC-3 project (phs001287) is released with WXS and WGS data.  RNA-Seq will be released at a later date.  Additional project details can be found at on the <a href="https://gdc.cancer.gov/about-gdc/contributed-genomic-data-cancer-research/clinical-proteomic-tumor-analysis-consortium-cptac">CPTAC Data Source page</a>.</li>
<li>TARGET-ALL-P3 (phs000218) WGS BAM files are released.</li>
<li>VAREPOP-APOLLO (phs001374) VCF files are released.</li>
</ol>
<p>A complete list of files for DR16.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20190326_data_release_16.0_active.txt.gz">gdc_manifest_20190326_data_release_16.0_active.txt.gz</a></li>
<li><a href="gdc_manifest_20190326_data_release_16.0_legacy.txt.gz">gdc_manifest_20190326_data_release_16.0_legacy.txt.gz</a>.</li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>TARGET ALL-P3 RNA-Seq results from DR14 are missing ~18% of reads.  Downsampling appears to be completely random and count files have a very high correlation (&gt;99.99%) with complete data. New versions of these files will be created that include the entire set of reads.</li>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.</li>
</ul>
<!--TT-602, DAT-1489-->
<ul>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MDLS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="None">Data Release 15.0 <!--REQ-381--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: February 20, 2019</li>
</ul>
<h3>New updates</h3>
<ol>
<li>TARGET-ALL-P3 is now available and includes RNA-Seq and WXS data.</li>
<li>New RNA-Seq workflow is now being utilized for new projects.  More details can be found in the <a href="../../Data/Bioinformatics_Pipelines/Expression_mRNA_Pipeline/#rna-seq-alignment-workflow">RNA-Seq pipeline documentation</a>.</li>
<li>New tumor only variant calling pipeline is now being utilized for new projects.  More details can be found in the <a href="../../Data/Bioinformatics_Pipelines/DNA_Seq_Variant_Calling_Pipeline/#tumor-only-variant-calling-workflow">Tumor only pipeline documentation</a>.</li>
</ol>
<p>A complete list of files for DR15.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20190220_data_release_15.0_active.txt.gz">gdc_manifest_20190220_data_release_15.0_active.txt.gz</a></li>
<li><a href="gdc_manifest_20190220_data_release_15.0_legacy.txt.gz">gdc_manifest_20190220_data_release_15.0_legacy.txt.gz</a>.</li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The read alignment end coordinates in the x.isoform.quantification.txt files produced by the miRNA pipeline are exclusive (i.e. offset by 1) for all TCGA miRNA legacy (GRCh37/hg19) and current harmonized (GRCh38/hg38) miRNA data.  This error has no impact on miRNA alignment or quantification - only the coordinates reported in the quantification file.</li>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.</li>
</ul>
<!--TT-602, DAT-1489-->
<ul>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MDLS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="None">Data Release 14.0 <!--REQ-335--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: December 18, 2018</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Copy Number Variation (CNV) data derived from GISTIC2 results are now available for download for TCGA projects <!--DAT-1681--></li>
<li>New miRNA data available for 181 aliquots for TARGET and TCGA <!--DAT-1823--></li>
<li>Released two SNP6 files (6cd4ef5e-324a-4ace-8779-7a33bd559c83, dfa89ee9-6ee5-460b-bd58-b5ca0e9cb7ac) <!--DAT-1805--></li>
<li>New versions of TCGA biospecimen supplements are available <!--DAT-1776--></li>
<li>Updated primary site for <code>TCGA-AG-3881</code> to <code>Unknown</code> <!--DAT-1853--></li>
<li>8 New Harmonized WGS BAM files for TARGET-WT, TARGET-NBL, TARGET-AML added to the portal <!--DAT-1839--></li>
</ol>
<p>A complete list of files for DR14.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20181218_data_release_14.0_active.txt.gz">gdc_manifest_20181218_data_release_14.0_active.txt.gz</a></li>
<li><a href="gdc_manifest_20181218_data_release_14.0_legacy.txt.gz">gdc_manifest_20181218_data_release_14.0_legacy.txt.gz</a>.</li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>FM-AD clinial and biospecimen supplements are now correctly labeled as TSV rather than XLSX <!--DAT-1681--></li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.</li>
</ul>
<!--TT-602, DAT-1489-->
<ul>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MDLS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="Data Release 13.0">Data Release 13.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: September 27, 2018</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Three new projects are released to the GDC (VAREPOP-APOLLO (phs001374), CTSP-DLBCL1 (phs001184), NCICCR-DLBCL (phs001444) <!--SPT-136,SPT-145,SPT-84--></li>
<li>TARGET WGS alignments are released. VCFs will be provided in a later release <!--DAT-1699--></li>
<li>Clinical data was harmonized with ICD-O-3 terminology for TCGA properties case.primary_site, case.disease_type, diagnosis.primary_diagnosis, diagnosis.site_of_resection_or_biopsy, diagnosis.tissue_or_organ_of_origin <!--DAT-1512--></li>
<li>Redaction annotations applied to 11 aliquots in TCGA-DLBC <!--DAT-1169, Data-871--></li>
<li>Redaction annotations applied to incorrectly trimmed miRNA file in the Legacy Achive <!--SV-653--></li>
</ol>
<p>A complete list of files for DR13.0 are listed for the GDC Data Portal and the GDC Legacy Archive are found below:</p>
<ul>
<li><a href="gdc_manifest_20180927_data_release_13.0_active.txt.gz">gdc_manifest_20180927_data_release_13.0_active.txt.gz</a></li>
<li><a href="gdc_manifest_20180927_data_release_13.0_legacy.txt.gz">gdc_manifest_20180927_data_release_13.0_legacy.txt.gz</a>.</li>
</ul>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>253 files Copy Number Segment and Masked Copy Number Segment files were released.  These were skipped in DR 12.0 <!--DAT-1404--></li>
<li>36 Diagnostic TCGA slides were released.  They were skipped in DR 12.0  <!--SV-1109--></li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>506 Copy Number Segment and 36 Slide Image files are designated as controlled-access on the GDC Data Portal.  These files are actually open-access and will be downloadable without a token using <a href="gdc_manifest_20181003_data_release_13.0_cnv_slides.txt">this manifest</a>.  <!--DAT-1804--></li>
<li>2 Copy Number Segment files from TCGA-TGCT do not appear on the GDC Portal. They can be downloaded using the Data Transfer Tool using the following UUIDs. <!--DAT-1805--><ul>
<li>6cd4ef5e-324a-4ace-8779-7a33bd559c83 - RAMPS_p_TCGA_Batch_430_NSP_GenomeWideSNP_6_E07_1538238.nocnv_grch38.seg.v2.txt</li>
<li>dfa89ee9-6ee5-460b-bd58-b5ca0e9cb7ac - RAMPS_p_TCGA_Batch_430_NSP_GenomeWideSNP_6_E07_1538238.grch38.seg.v2.txt</li>
</ul>
</li>
<li>TARGET CGI BAMs in the Legacy Archive for the following aliquots should not be used because they were not repaired and concatenated into their original composite BAM files by CGHub.<ul>
<li>TARGET-20-PASJGZ-04A-02D</li>
<li>TARGET-30-PAPTLY-01A-01D</li>
<li>TARGET-20-PAEIKD-09A-01D</li>
<li>TARGET-20-PASMYS-14A-02D</li>
<li>TARGET-20-PAMYAS-14A-02D</li>
<li>TARGET-10-PAPZST-09A-01D</li>
</ul>
</li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.</li>
</ul>
<!--TT-602, DAT-1489-->
<ul>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>FM-AD clinical and biospecimen supplement files have incorrect data format.  They are listed as XLSX, but are in fact TSV files. <!--DAT-1123--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MDLS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="Data Release 12.0">Data Release 12.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: June 13, 2018</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Updated clinical and biospecimen XML files for TCGA cases are available in the GDC Data Portal. Equivalent Legacy Archive files may no longer be up to date. <!--TT-328--></li>
<li>All biospecimen and clinical supplement files for TCGA projects formerly only found in the Legacy Archive have been updated and transferred to the GDC Data Portal. Equivalent Legacy Archive files and metadata retrieved from the API may no longer be up to date. <!--TT-327--></li>
<li>Diagnostic slides from TCGA are now available in the GDC Data Portal and Slide Image Viewer.  They were formerly only available in the Legacy Archive. <!--DAT-1316--></li>
<li>Updated Copy Number Segment and Masked Copy Number Segment files are now available.  These were generated using an improved mapping of hg38 coordinates for the Affymetrix SNP6.0 probe set. <!--DAT-1303--></li>
<li>VCF files containing SNVs produced from TARGET WGS CGI data are available.  The variant calls were initially produced by CGI and lifted over to hg38. <!--DAT-1281--></li>
</ol>
<p>Updated files for this release are listed <a href="DR12.0_files_swap.txt.gz">here</a>.
A complete list of files for DR12.0 are listed for the GDC Data Portal <a href="gdc_manifest_20180613_data_release_12.0_active.txt.gz">here</a> and the GDC Legacy Archive <a href="gdc_manifest_20180613_data_release_12.0_legacy.txt.gz">here</a>.</p>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>TARGET NBL RNA-Seq data is now associated with the correct aliquot. <!--SV-1097--></li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Some Copy Number Segment and Masked Copy Number Segment were not replaced in DR 12.0.  253 files remain to be swapped in a later release <!--DAT-1404--></li>
<li>Some miRNA files with QC failed reads were not swapped in DR11.0.  361 aliquots remain to be swapped in a later release <!--DAT-1589--></li>
<li>74 Diagnostic TCGA slides are attached to a portion rather than a sample like the rest of the diagnostic slides. The reflects how these original samples were handled. <!--SV-1111--></li>
<li>36 Diagnostic TCGA slides are not yet available in the active GDC Portal. They are still available in the GDC Legacy Archive.  <!--SV-1109--></li>
<li>11 bam files for TARGET-NBL RNA-Seq are not available in the GDC Data portal <!--DAT-1476--></li>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> are not available. These VCFs files will be replaced in a later release.</li>
</ul>
<!--TT-602, DAT-1489-->
<ul>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>FM-AD clinical and biospecimen supplement files have incorrect data format.  They are listed as XLSX, but are in fact TSV files. <!--DAT-1123--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>There are 11 cases in project TCGA-DLBC that are known to have incorrect WXS data in the GDC Data Portal.  Impacted cases are listed below.  This affects the BAMs and VCFs associated with these cases in the GDC Data Portal.  Corrected BAMs can be found in the GDC Legacy Archive.  Variants from affected aliquots appear in the protected MAFs with GDC_FILTER=ContEst to indicate a sample contamination problem, but are removed during the generation of the Somatic MAF file.  In a later release we will supply corrected BAM, VCF, and MAF files for these cases.  In the mean time, we advise you not to use any of the WXS files associated with these cases in the GDC Data Portal.  A list of these files can be found here. <a href="DLBC_Affected_Files.txt">Download list of affected files</a>. <!-- Data-871--><ul>
<li>TCGA-FF-8062</li>
<li>TCGA-FM-8000</li>
<li>TCGA-G8-6324</li>
<li>TCGA-G8-6325</li>
<li>TCGA-G8-6326</li>
<li>TCGA-G8-6906</li>
<li>TCGA-G8-6907</li>
<li>TCGA-G8-6909</li>
<li>TCGA-G8-6914</li>
<li>TCGA-GR-7351</li>
<li>TCGA-GR-7353</li>
</ul>
</li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MDLS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="Data Release 11.0">Data Release 11.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: May 21, 2018</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Updated miRNA files to remove QCFail reads.  This included all BAM and downstream count files. <!--DAT-1246--></li>
<li>TCGA Tissue slide images now available in GDC Data Portal.  Previously these were found only in the Legacy Archive <!--DAT-1251--></li>
</ol>
<p>Updated files for this release are listed <a href="DR11.0_files_swap.txt.gz">here</a>.
A complete list of files for DR11.0 are listed for the GDC Data Portal <a href="gdc_manifest_20180521_data_release_11.0_active.txt.gz">here</a> and the GDC Legacy Archive <a href="gdc_manifest_20180521_data_release_11.0_legacy.txt.gz">here</a>.</p>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>N/A</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Two tissue slide images are unavailable for download from GDC Data Portal <!--DAT-1439--></li>
<li>RNA-Seq files for TARGET-NBL are attached to the incorrect aliquot.  The BAM files contain the correct information in their header but the connection in the GDC to read groups and aliquots is incorrect.  The linked file below contains a mapping between aliquots where file are currently associated and the aliquot where they should instead be associated <a href="correct_aliquot_mappings.tsv">(mapping file)</a>. <!--SV-1097--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> were not replaced in DR10.0 and thus do not contain indels.  However, the indels from this aliquot can be found in the MAF files and are displayed in the Exploration section in the Data Portal.  These VCFs files will be replaced in a later release.</li>
</ul>
<!--SV-950-->
<ul>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>miRNA alignments include QC failed reads.  </li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>FM-AD clinical and biospecimen supplement files have incorrect data format.  They are listed as XLSX, but are in fact TSV files. <!--DAT-1123--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>There are 11 cases in project TCGA-DLBC that are known to have incorrect WXS data in the GDC Data Portal.  Impacted cases are listed below.  This affects the BAMs and VCFs associated with these cases in the GDC Data Portal.  Corrected BAMs can be found in the GDC Legacy Archive.  Variants from affected aliquots appear in the protected MAFs with GDC_FILTER=ContEst to indicate a sample contamination problem, but are removed during the generation of the Somatic MAF file.  In a later release we will supply corrected BAM, VCF, and MAF files for these cases.  In the mean time, we advise you not to use any of the WXS files associated with these cases in the GDC Data Portal.  A list of these files can be found here. <a href="DLBC_Affected_Files.txt">Download list of affected files</a>. <!-- Data-871--><ul>
<li>TCGA-FF-8062</li>
<li>TCGA-FM-8000</li>
<li>TCGA-G8-6324</li>
<li>TCGA-G8-6325</li>
<li>TCGA-G8-6326</li>
<li>TCGA-G8-6906</li>
<li>TCGA-G8-6907</li>
<li>TCGA-G8-6909</li>
<li>TCGA-G8-6914</li>
<li>TCGA-GR-7351</li>
<li>TCGA-GR-7353</li>
</ul>
</li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MDLS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="Data Release 10.1">Data Release 10.1</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: February 15, 2018</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Updated FM-AD clinical data to conform with Data Dictionary release v1.11</li>
</ol>
<h3>Bugs Fixed Since Last Release</h3>
<p>None</p>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>RNA-Seq files for TARGET-NBL are attached to the incorrect aliquot.  The BAM files contain the correct information in their header but the connection in the GDC to read groups and aliquots is incorrect.  The linked file below contains a mapping between aliquots where file are currently associated and the aliquot where they should instead be associated <a href="correct_aliquot_mappings.tsv">(mapping file)</a>. <!--SV-1097--></li>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> were not replaced in DR10.0 and thus do not contain indels.  However, the indels from this aliquot can be found in the MAF files and are displayed in the Exploration section in the Data Portal.  These VCFs files will be replaced in a later release.</li>
</ul>
<!--SV-950-->
<ul>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>miRNA alignments include QC failed reads.  </li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>FM-AD clinical and biospecimen supplement files have incorrect data format.  They are listed as XLSX, but are in fact TSV files. <!--DAT-1123--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>There are 11 cases in project TCGA-DLBC that are known to have incorrect WXS data in the GDC Data Portal.  Impacted cases are listed below.  This affects the BAMs and VCFs associated with these cases in the GDC Data Portal.  Corrected BAMs can be found in the GDC Legacy Archive.  Variants from affected aliquots appear in the protected MAFs with GDC_FILTER=ContEst to indicate a sample contamination problem, but are removed during the generation of the Somatic MAF file.  In a later release we will supply corrected BAM, VCF, and MAF files for these cases.  In the mean time, we advise you not to use any of the WXS files associated with these cases in the GDC Data Portal.  A list of these files can be found here. <a href="DLBC_Affected_Files.txt">Download list of affected files</a>. <!-- Data-871--><ul>
<li>TCGA-FF-8062</li>
<li>TCGA-FM-8000</li>
<li>TCGA-G8-6324</li>
<li>TCGA-G8-6325</li>
<li>TCGA-G8-6326</li>
<li>TCGA-G8-6906</li>
<li>TCGA-G8-6907</li>
<li>TCGA-G8-6909</li>
<li>TCGA-G8-6914</li>
<li>TCGA-GR-7351</li>
<li>TCGA-GR-7353</li>
</ul>
</li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MDLS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="None">Data Release 10.0 <!--REQ-318--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: December 21, 2017</li>
</ul>
<h3>New updates</h3>
<ol>
<li>New TARGET files for all projects <!--DAT-1135--></li>
<li>TARGET updates for clinical and biospecimen data <!--DAT-1125, DAT-1153--></li>
<li>Replace corrupted .bai files <!--TT-109--></li>
<li>Update TCGA and TARGET MAF files to include VarScan2 indels and more information in all_effects column <!--DAT-987--></li>
<li>Update VarScan VCF files <!--DAT-1020--></li>
</ol>
<p>Updated files for this release are listed <a href="DR10.0_files_swap.txt.gz">here</a>.
A complete list of files for DR10.0 are listed for the GDC Data Portal <a href="gdc_manifest_20171221_data_release_10.0_active.txt.gz">here</a> and the GDC Legacy Archive <a href="gdc_manifest_20171221_data_release_10.0_legacy.txt.gz">here</a>.</p>
<h3>Bugs Fixed Since Last Release</h3>
<p>None</p>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>The raw and annotated VarScan VCF files for aliquot <code>TCGA-VR-A8ET-01A-11D-A403-09</code> were not replaced in DR10.0 and thus do not contain indels.  However, the indels from this aliquot can be found in the MAF files and are displayed in the Exploration section in the Data Portal.  These VCFs files will be replaced in a later release.</li>
</ul>
<!--SV-950-->
<ul>
<li>There are 5051 TARGET files for which <code>experimental_strategy</code>, <code>data_format</code>, <code>platform</code>, and <code>data_subtype</code> are blank <!--SV-944--></li>
<li>There are two cases with identical submitter_id <code>TARGET-10-PARUYU</code> <!--SV-940--></li>
<li>TARGET-MDLS cases do not have disease_type or primary_site populated <!--SV-939--></li>
<li>Some TARGET cases are missing <code>days_to_last_follow_up</code> <!--SV-934--></li>
<li>Some TARGET cases are missing <code>age_at_diagnosis</code> <!--SV-933--></li>
<li>Some TARGET files are not connected to all related aliquots <!--SV-929--></li>
<li>miRNA alignments include QC failed reads.  </li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>FM-AD clinical and biospecimen supplement files have incorrect data format.  They are listed as XLSX, but are in fact TSV files. <!--DAT-1123--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>There are 11 cases in project TCGA-DLBC that are known to have incorrect WXS data in the GDC Data Portal.  Impacted cases are listed below.  This affects the BAMs and VCFs associated with these cases in the GDC Data Portal.  Corrected BAMs can be found in the GDC Legacy Archive.  Variants from affected aliquots appear in the protected MAFs with GDC_FILTER=ContEst to indicate a sample contamination problem, but are removed during the generation of the Somatic MAF file.  In a later release we will supply corrected BAM, VCF, and MAF files for these cases.  In the mean time, we advise you not to use any of the WXS files associated with these cases in the GDC Data Portal.  A list of these files can be found here. <a href="DLBC_Affected_Files.txt">Download list of affected files</a>. <!-- Data-871--><ul>
<li>TCGA-FF-8062</li>
<li>TCGA-FM-8000</li>
<li>TCGA-G8-6324</li>
<li>TCGA-G8-6325</li>
<li>TCGA-G8-6326</li>
<li>TCGA-G8-6906</li>
<li>TCGA-G8-6907</li>
<li>TCGA-G8-6909</li>
<li>TCGA-G8-6914</li>
<li>TCGA-GR-7351</li>
<li>TCGA-GR-7353</li>
</ul>
</li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MDLS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="None">Data Release 9.0 <!--REQ-317--></h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: October 24, 2017</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Foundation Medicine Data Release</li>
<li>This includes controlled-access VCF and MAF files as well as clinical and biospecimen supplements and metadata.</li>
<li>Original Foundation Medicine supplied data can be found on the <a href="https://gdc.cancer.gov/about-gdc/contributed-genomic-data-cancer-research/foundation-medicine/foundation-medicine">Foundation Medicine Project Page</a>.</li>
<li>Updated RNA-Seq data for TARGET NBL  </li>
<li>Includes new BAM and count files</li>
</ol>
<p>Updated files for this release are listed <a href="DR9.0_files_swap.txt.gz">here</a>.
A complete list of files for DR9.0 are listed <a href="gdc_manifest_20171024_data_release_9.0_active.txt.gz">here</a>.</p>
<h3>Bugs Fixed Since Last Release</h3>
<p>None</p>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>miRNA alignments include QC failed reads.  </li>
<li>Samples of TARGET sample_type <code>Recurrent Blood Derived Cancer - Bone Marrow</code> are mislabeled as <code>Recurrent Blood Derived Cancer - Peripheral Blood</code>.  A workaround is to look at the sample barcode, which is -04 for <code>Recurrent Blood Derived Cancer - Bone Marrow</code>. (e.g. <code>TARGET-20-PAMYAS-04A-03R</code>) <!--SV-918--></li>
<li>FM-AD clinical and biospecimen supplement files have incorrect data format.  They are listed as XLSX, but are in fact TSV files. <!--DAT-1123--></li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>There are 11 cases in project TCGA-DLBC that are known to have incorrect WXS data in the GDC Data Portal.  Impacted cases are listed below.  This affects the BAMs and VCFs associated with these cases in the GDC Data Portal.  Corrected BAMs can be found in the GDC Legacy Archive.  Variants from affected aliquots appear in the protected MAFs with GDC_FILTER=ContEst to indicate a sample contamination problem, but are removed during the generation of the Somatic MAF file.  In a later release we will supply corrected BAM, VCF, and MAF files for these cases.  In the mean time, we advise you not to use any of the WXS files associated with these cases in the GDC Data Portal.  A list of these files can be found here. <a href="DLBC_Affected_Files.txt">Download list of affected files</a>. <!-- Data-871--><ul>
<li>TCGA-FF-8062</li>
<li>TCGA-FM-8000</li>
<li>TCGA-G8-6324</li>
<li>TCGA-G8-6325</li>
<li>TCGA-G8-6326</li>
<li>TCGA-G8-6906</li>
<li>TCGA-G8-6907</li>
<li>TCGA-G8-6909</li>
<li>TCGA-G8-6914</li>
<li>TCGA-GR-7351</li>
<li>TCGA-GR-7353</li>
</ul>
</li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MDLS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="Data Release 8.0">Data Release 8.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: August 22, 2017</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Released updated miRNA quantification files to address double counting of some normalized counts described in DR7.0 release notes. <!--DAT-1, DAT-988--></li>
</ol>
<p>Updated files for this release are listed <a href="DR8.0_files_swap.txt">here</a>.
A Complete list of files for DR8.0 are listed <a href="gdc_manifest_20170822_data_release_8.0_active.txt.gz">here</a>.</p>
<h3>Bugs Fixed Since Last Release</h3>
<p>None</p>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>TARGET-NBL RNA-Seq files were run as single ended even though they are derived from paired-end data.  These files will be rerun through the GDC RNA-Seq pipelines in a later release.  Impacted files can be found <a href="https://portal.gdc.cancer.gov/repository?facetTab=files&amp;files_offset=20&amp;filters=~%28op~%27and~content~%28~%28op~%27in~content~%28field~%27cases.project.program.name~value~%28~%27TARGET%29%29%29~%28op~%27in~content~%28field~%27cases.project.project_id~value~%28~%27TARGET-NBL%29%29%29~%28op~%27in~content~%28field~%27files.data_format~value~%28~%27BAM%29%29%29~%28op~%27in~content~%28field~%27files.experimental_strategy~value~%28~%27RNA-Seq%29%29%29%29%29&amp;searchTableTab=files">here</a>.  Downstream count files are also affected.  Users may access original FASTQ files in the GDC Legacy Archive, which are not impacted by this issue.</li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>There are 11 cases in project TCGA-DLBC that are known to have incorrect WXS data in the GDC Data Portal.  Impacted cases are listed below.  This affects the BAMs and VCFs associated with these cases in the GDC Data Portal.  Corrected BAMs can be found in the GDC Legacy Archive.  Variants from affected aliquots appear in the protected MAFs with GDC_FILTER=ContEst to indicate a sample contamination problem, but are removed during the generation of the Somatic MAF file.  In a later release we will supply corrected BAM, VCF, and MAF files for these cases.  In the mean time, we advise you not to use any of the WXS files associated with these cases in the GDC Data Portal.  A list of these files can be found here. <a href="DLBC_Affected_Files.txt">Download list of affected files</a>. <!-- Data-871--><ul>
<li>TCGA-FF-8062</li>
<li>TCGA-FM-8000</li>
<li>TCGA-G8-6324</li>
<li>TCGA-G8-6325</li>
<li>TCGA-G8-6326</li>
<li>TCGA-G8-6906</li>
<li>TCGA-G8-6907</li>
<li>TCGA-G8-6909</li>
<li>TCGA-G8-6914</li>
<li>TCGA-GR-7351</li>
<li>TCGA-GR-7353</li>
</ul>
</li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MDLS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="Data Release 7.0">Data Release 7.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: June 29, 2017</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Updated public Mutation Annotation Format (MAF) files are now available. Updates include filtering to remove variants impacted by OxoG artifacts and those impacted by strand bias. <!--AP-29--></li>
<li>Protected MAF files are updated to include flags for OxoG and strand bias. <!--AP-29--></li>
<li>Annotated VCFs are updated to include flags for OxoG artifacts and strand bias. <!--DAT-874--></li>
</ol>
<p>Updated files for this release are listed <a href="DR7.0_files_swap.txt">here</a>.
A Complete list of files for DR7.0 are listed <a href="gdc_manifest_20170629_data_release_7.0.txt.gz">here</a></p>
<h3>Bugs Fixed Since Last Release</h3>
<p>None</p>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>TARGET-NBL RNA-Seq files were run as single ended even though they are derived from paired-end data.  These files will be rerun through the GDC RNA-Seq pipelines in a later release.  Impacted files can be found <a href="https://portal.gdc.cancer.gov/repository?facetTab=files&amp;files_offset=20&amp;filters=~%28op~%27and~content~%28~%28op~%27in~content~%28field~%27cases.project.program.name~value~%28~%27TARGET%29%29%29~%28op~%27in~content~%28field~%27cases.project.project_id~value~%28~%27TARGET-NBL%29%29%29~%28op~%27in~content~%28field~%27files.data_format~value~%28~%27BAM%29%29%29~%28op~%27in~content~%28field~%27files.experimental_strategy~value~%28~%27RNA-Seq%29%29%29%29%29&amp;searchTableTab=files">here</a>.  Downstream count files are also affected.  Users may access original FASTQ files in the GDC Legacy Archive, which are not impacted by this issue.</li>
<li>Reads that are mapped to multiple genomic locations are double counted in some of the GDC miRNA results.  The GDC will release updated files correcting the issue in an upcoming release.<!--SV-745-->  The specific impacts are described further below:<ul>
<li>Isoform Expression Quantification files<ul>
<li>Raw reads counts are accurate</li>
<li>Normalized counts are proportionally skewed (r^2=1.0)</li>
</ul>
</li>
<li>miRNA Expression Quantification files<ul>
<li>A small proportion of miRNA counts are overestimated (mean r^2=0.9999)</li>
<li>Normalized counts are proportionally skewed (mean r^2=0.9999)</li>
</ul>
</li>
<li>miRNA BAM files<ul>
<li>no impact</li>
</ul>
</li>
</ul>
</li>
<li>Mutation frequency may be underestimated when using MAF files for genes that overlap other genes.  This is because MAF files only record one gene per variant.</li>
<li>Most intronic mutations are removed for MAF generation.  However, validated variants may rescue these in some cases.  Therefore intronic mutations in MAF files are not representative of those called by mutation callers.</li>
<li>The latest TARGET data is not yet available at the GDC.  For the complete and latest data, please see the <a href="https://ocg.cancer.gov/programs/target/data-matrix">TARGET Data Matrix</a>.  Data that is not present or is not the most up to date includes:<ul>
<li>All microarray data and metadata</li>
<li>All sequencing analyzed data and metadata</li>
<li>1180 of 12063 sequencing runs of raw data</li>
</ul>
</li>
<li>Demographic information for some TARGET patients is incorrect.  The correct information can be found in the associated clinical supplement file.  Impacted patients are TARGET-50-PAJNUS. <!--SV-710--></li>
<li>There are 11 cases in project TCGA-DLBC that are known to have incorrect WXS data in the GDC Data Portal.  Impacted cases are listed below.  This affects the BAMs and VCFs associated with these cases in the GDC Data Portal.  Corrected BAMs can be found in the GDC Legacy Archive.  Variants from affected aliquots appear in the protected MAFs with GDC_FILTER=ContEst to indicate a sample contamination problem, but are removed during the generation of the Somatic MAF file.  In a later release we will supply corrected BAM, VCF, and MAF files for these cases.  In the mean time, we advise you not to use any of the WXS files associated with these cases in the GDC Data Portal.  A list of these files can be found here. <a href="DLBC_Affected_Files.txt">Download list of affected files</a>. <!-- Data-871--><ul>
<li>TCGA-FF-8062</li>
<li>TCGA-FM-8000</li>
<li>TCGA-G8-6324</li>
<li>TCGA-G8-6325</li>
<li>TCGA-G8-6326</li>
<li>TCGA-G8-6906</li>
<li>TCGA-G8-6907</li>
<li>TCGA-G8-6909</li>
<li>TCGA-G8-6914</li>
<li>TCGA-GR-7351</li>
<li>TCGA-GR-7353</li>
</ul>
</li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MLDS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<h2 id="Data Release 6.0">Data Release 6.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: May 9, 2017</li>
</ul>
<h3>New updates</h3>
<ol>
<li>GDC updated public Mutation Annotation Format (MAF) files are now available. Updates include leveraging the MC3 variant filtering strategy, which results in more variants being recovered relative to the previous version. A detailed description of the new format can be found <a href="../File_Formats/MAF_Format/">here</a>. <!--DAT-572--></li>
<li>Protected MAFs are updated to include additional variant annotation information <!--DAT-572--></li>
<li>Some MuTect2 VCFs updated to include dbSNP and COSMIC annotations found in other VCFs <!--TT-21--></li>
</ol>
<p>Updated files for this release are listed <a href="DR6.0_files_swap.txt">here</a>.</p>
<h3>Bugs Fixed Since Last Release</h3>
<p>None</p>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>There are 11 cases in project TCGA-DLBC that are known to have incorrect WXS data in the GDC Data Portal.  Impacted cases are listed below.  This affects the BAMs and VCFs associated with these cases in the GDC Data Portal.  Corrected BAMs can be found in the GDC Legacy Archive.  Variants from affected aliquots appear in the protected MAFs with GDC_FILTER=ContEst to indicate a sample contamination problem, but are removed during the generation of the Somatic MAF file.  In a later release we will supply corrected BAM, VCF, and MAF files for these cases.  In the mean time, we advise you not to use any of the WXS files associated with these cases in the GDC Data Portal.  A list of these files can be found here. <a href="DLBC_Affected_Files.txt">Download list of affected files</a>. <!-- Data-871--><ul>
<li>TCGA-FF-8062</li>
<li>TCGA-FM-8000</li>
<li>TCGA-G8-6324</li>
<li>TCGA-G8-6325</li>
<li>TCGA-G8-6326</li>
<li>TCGA-G8-6906</li>
<li>TCGA-G8-6907</li>
<li>TCGA-G8-6909</li>
<li>TCGA-G8-6914</li>
<li>TCGA-GR-7351</li>
<li>TCGA-GR-7353</li>
</ul>
</li>
<li>Variants found in VCF and MAF files may contain OxoG artifacts, which are produced during library preparation and may result in the apparent substitutions of C to A or G to T in certain sequence contexts.  In the future we will plan to label potential oxoG artifacts in the MAF files.</li>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Some validated somatic mutations may not be present in open-access MAF files.  Please review the protected MAF files in the GDC Data Portal if you are unable to find your mutation in the open-access files.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>No data from TARGET-MLDS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<p>Details are provided in <a href="Manifests/GDC_Data_v3_release_notes_manifest.txt">Data Release Manifest</a>
<br/></p>
<h2 id="Data Release 5.0">Data Release 5.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: March 16, 2017</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Additional annotations from TCGA DCC are available <!--DAT-52--><ul>
<li>Complete list of updated TCGA files is found <a href="DR5.0_CHANGES_TCGA.xlsx">here</a></li>
</ul>
</li>
<li>Clinical data added for TARGET ALL P1 and P2 <!--DAT-197--></li>
<li>Pathology reports now have submitter IDs as assigned by the BCR <!--DAT-81--></li>
<li>TARGET Data refresh<ul>
<li>Most recent biospecimen and clinical information from the TARGET DCC. New imported files are listed <a href="DR5.0_changes_TARGET.xlsx">here</a></li>
<li>Updated indexed biospecimen and clinical metadata</li>
<li>Updated SRA XMLs files</li>
<li>Does not include updates to TARGET NBL <!--SV-585--></li>
</ul>
</li>
</ol>
<h3>Bugs Fixed Since Last Release</h3>
<ol>
<li>Missing cases from TCGA-LAML were added to Legacy Archive <!--DAT-272,DAT-324--></li>
<li>Biotab files are now linked to Projects and Cases in Legacy Archive <!--SV-303,DAT-28--></li>
</ol>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Some TCGA annotations are unavailable in the Legacy Archive or Data Portal<!--DAT-52-->. These annotations can be found <a href="tcga-annotations-unavailable-20170315.json">here</a>.</li>
<li>Some validated somatic mutations may not be present in open-access MAF files.  When creating open-access MAF files from the protected versions we are extremely conservative in removing potential germline variants.  Our approach is to remove all mutations that are present in dbSNP.  In a subsequent release we will provide updated open-access MAF files, which preserve variants found in MC3 or a TCGA validation study.  Please review the protected MAF files in the GDC Data Portal if you are unable to find your mutation in the open-access files.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>MAF Column #109 "FILTER" entries are separated by both commas and semi-colons. <!-- PGDC-2589 --></li>
<li>TARGET-AML is undergoing reorganization.  Pending reorganization, cases from this projects may not contain many clinical, biospecimen, or genomic data files.</li>
<li>No data from TARGET-MLDS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>Two biotab files are not linked to Project or Case in the Legacy Archive <!--SV-535, DAT-493--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
<li>Tumor grade property is not populated <!--SV-585--></li>
<li>Progression_or_recurrence property is not populated <!--SV-584--></li>
</ul>
<p>Details are provided in <a href="Manifests/GDC_Data_v3_release_notes_manifest.txt">Data Release Manifest</a>
<br/></p>
<h2 id="Data Release 4.0">Data Release 4.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: October 31, 2016</li>
</ul>
<h3>New updates</h3>
<ol>
<li>TARGET ALL P1 and P2 biospecimen and molecular data are now available in the Legacy Archive.  Clinical data will be available in a later release. <!-- Dat-185, Dat-194--></li>
<li>Methylation data from 27k/450k Arrays has been lifted over to hg38 and is now available in the GDC Data Portal <!-- Dat-109 --></li>
<li>Public MAF files are now available for VarScan2, MuSE, and SomaticSniper.  MuTect2 MAFs were made available in a previous release. <!--DAT-235--></li>
<li>Updated VCFs and MAF files are available for MuTect2 pipeline to compensate for WGA-related false positive indels.  See additional information on that change <a href="https://gdc.cancer.gov/content/mutect2-insertion-artifacts">here</a>. A listing of replaced files is provided <a href="GDC_Data_v4_mapping_of_replaced_Mutect2_MAF_and_VCF_files.zip">here</a>. <!-- Dat-145, Dat-260 --></li>
<li>Added submitter_id for Pathology Reports in Legacy Archive <!--DAT-81--></li>
</ol>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>None</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Some validated somatic mutations may not be present in open-access MAF files.  When creating open-access MAF files from the protected versions we are extremely conservative in removing potential germline variants.  Our approach is to remove all mutations that are present in dbSNP.  In a subsequent release we will provide updated open-access MAF files, which preserve variants found in COSMIC or a TCGA validation study.  Please review the protected MAF files in the GDC Data Portal if you are unable to find your mutation in the open-access files.</li>
<li>Public MAF files for different variant calling pipelines but the same project may contain different numbers of samples.  Samples are omitted from the public MAF files if they have no PASS variants, which can lead to this apparent discrepancy.</li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>MAF Column #109 "FILTER" entries are separated by both commas and semi-colons. <!-- PGDC-2589 --></li>
<li>TARGET-AML is undergoing reorganization.  Pending reorganization, cases from this projects may not contain many clinical, biospecimen, or genomic data files.</li>
<li>No data from TARGET-MLDS is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>There are 200 cases from TCGA-LAML that do not appear in the Legacy Archive <!--SV-327--></li>
<li>Biotab files are not linked to Project or Case in the Legacy Archive <!--SV-303--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
</ul>
<p>Details are provided in <a href="Manifests/GDC_Data_v5_release_notes_manifest.txt">Data Release Manifest</a>
<br/></p>
<h2 id="Data Release 3.0">Data Release 3.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: September 16, 2016</li>
</ul>
<h3>New updates</h3>
<ol>
<li>CCLE data now available (in the Legacy Archive only)</li>
<li>BMI calculation is corrected</li>
<li>Slide is now categorized as a Biospecimen entity</li>
</ol>
<h3>Bugs Fixed Since Last Release</h3>
<ul>
<li>BMI calculation is corrected</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Insertions called for tumor samples that underwent whole genome amplification may be of lower quality.  Whether a sample underwent this process can be found in the analyte_type property within analyte and aliquot. TCGA analyte type can be also identified in the 20th character of TCGA barcode, at which "W" corresponds to WGA.<!--BINF-6--></li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Public MAFs (those with germline variants removed) are only available for MuTect2 pipeline.  MAFs for other pipelines are forthcoming.  </li>
<li>MAF Column #109 "FILTER" entries are separated by both commas and semi-colons. <!-- PGDC-2589 --></li>
<li>TARGET-AML and TARGET-ALL projects are undergoing reorganization.  Pending reorganization, cases from these projects may not contain many clinical, biospecimen, or genomic data files.</li>
<li>No data from TARGET-PPTP is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>There are 200 cases from TCGA-LAML that do not appear in the Legacy Archive <!--SV-327--></li>
<li>Biotab files are not linked to Project or Case in the Legacy Archive <!--SV-303--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
</ul>
<p>Details are provided in <a href="Manifests/GDC_Data_v3_release_notes_manifest.txt">Data Release Manifest</a>
<br/></p>
<h2 id="Data Release 2.0">Data Release 2.0</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: August 9, 2016</li>
</ul>
<h3>New updates</h3>
<ol>
<li>Additional data, previously available via CGHub and the TCGA DCC, is now available in the GDC</li>
<li>Better linking between files and their associated projects and cases in the Legacy Archive</li>
<li>MAF files are now available in the GDC Data Portal</li>
</ol>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>Insertions called for tumor samples that underwent whole genome amplification may be of lower quality.  These are present in VCF and MAF files produced by the MuTect2 variant calling pipeline.  This information can be found in the analyte_type property within analyte and aliquot. TCGA analyte type can be also identified in the 20th character of TCGA barcode, at which "W" corresponds to WGA.<!--BINF-6--></li>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>Public MAFs (those with germline variants removed) are only available for MuTect2 pipeline.  MAFs for other pipelines are forthcoming.  </li>
<li>MAF Column #109 "FILTER" entries are separated by both commas and semi-colons. <!-- PGDC-2589 --></li>
<li>TARGET-AML and TARGET-ALL projects are undergoing reorganization.  Pending reorganization, cases from these projects may not contain many clinical, biospecimen, or genomic data files.</li>
<li>No data from TARGET-PPTP is available.</li>
<li>Slide barcodes (<code>submitter_id</code> values for Slide entities in the Legacy Archive) are not available <!-- DAT-10 --></li>
<li>SDF Files are not linked to Project or Case in the Legacy Archive <!--SV-332--></li>
<li>There are 200 cases from TCGA-LAML that do not appear in the Legacy Archive <!--SV-327--></li>
<li>Biotab files are not linked to Project or Case in the Legacy Archive <!--SV-303--></li>
<li>SDRF files are not linked to Project or Case in the Legacy Archive <!--SV-288--></li>
<li>Portion "weight" property is incorrectly described in the Data Dictionary as the weight of the patient in kg, should be described as the weight of the portion in mg <!--SV-391--></li>
</ul>
<p>Details are provided in <a href="Manifests/GDC_Data_v2_release_notes_manifest.txt">Data Release Manifest</a>
<br/></p>
<h2 id="Initial Data Release (1.0)">Initial Data Release (1.0)</h2>
<ul>
<li><strong>GDC Product</strong>: Data</li>
<li><strong>Release Date</strong>: June 6, 2016</li>
</ul>
<h3>Available Program Data</h3>
<ul>
<li>The Cancer Genome Atlas (TCGA)</li>
<li>Therapeutically Applicable Research To Generate Effective Treatments (TARGET)</li>
</ul>
<h3>Available Harmonized Data</h3>
<ul>
<li>WXS<ul>
<li>Co-cleaned BAM files aligned to GRCh38 using BWA</li>
</ul>
</li>
<li>mRNA-Seq<ul>
<li>BAM files aligned to GRCh38 using STAR 2-pass strategy</li>
<li>Expression quantification using HTSeq</li>
</ul>
</li>
<li>miRNA-Seq<ul>
<li>BAM files aligned to GRCh38 using BWA aln</li>
<li>Expression quantification using BCCA miRNA Profiling Pipeline*</li>
</ul>
</li>
<li>Genotyping Array<ul>
<li>CNV segmentation data</li>
</ul>
</li>
</ul>
<h3>Known Issues and Workarounds</h3>
<ul>
<li>BAM files produced by the GDC RNA-Seq Alignment workflow will currently fail validation using the Picard ValidateSamFiles tool.  This is caused by STAR2 not recording mate mapping information for unmapped reads, which are retained in our BAM files.  Importantly, all affected BAM files are known to behave normally in downstream workflows including expression quantification.</li>
<li>All legacy files for TCGA are available in the GDC Legacy Archive, but not always linked back to cases depending on available metadata.</li>
<li>Public MAFs (those with germline variants removed) are only available for MuTect2 pipeline.  MAFs for other pipelines are forthcoming.  </li>
<li>TARGET-AML and TARGET-ALL projects are undergoing reorganization.  Pending reorganization, cases from these projects may not contain many clinical, biospecimen, or genomic data files.</li>
<li>No data from TARGET-PPTP is available.</li>
<li>Legacy data not available in harmonized form:<ul>
<li>Annotated VCF files from TARGET, anticipated in future data release</li>
<li>TCGA data that failed harmonization or QC or have been newly updated in CGHub: ~1.0% of WXS aliquots, ~1.6% of RNA-Seq aliquots</li>
<li>TARGET data that failed harmonization or QC, have been newly updated in CGHub, or whose project names are undergoing reorganization: ~76% of WXS aliquots, ~49% of RNA-Seq aliquots, ~57% of miRNA-Seq.</li>
</ul>
</li>
<li>MAF Column #109 "FILTER" entries are separated by both commas and semi-colons. <!-- PGDC-2589 --></li>
<li>MAFs are not yet available for query or search in the GDC Data Portal or API.  You may download these files using the following manifests, which can be passed directly to the Data Transfer Tool.  Links for the open-access TCGA MAFs are provided below for downloading individual files.<ul>
<li><a href="Manifests/GDC_open_MAFs_manifest.txt">Open-access MAFs manifest</a></li>
<li><a href="Manifests/GDC_controlled_MAFs_manifest.txt">Controlled-access MAFs manifest</a></li>
</ul>
</li>
</ul>
<p>Details are provided in <a href="Manifests/GDC_Data_v1_release_notes_manifest.txt">Data Release Manifest</a></p>
<h3>Download Open-access MAF files</h3>
<p><a href="https://api.gdc.cancer.gov/data/abbe72a5-cb39-48e4-8df5-5fd2349f2bb2">TCGA.ACC.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/cb97ef6e-7a13-4d42-81b4-1510c00d6373">TCGA.BLCA.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/91ae5ca9-55c2-4c9c-929e-8638444dc7b5">TCGA.BRCA.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/fedc0e90-5070-41e4-993d-603b92f4ecfd">TCGA.CESC.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/f33fd38b-287c-4978-a1bb-a95bbfd4351a">TCGA.CHOL.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/bf1c53dc-79bb-43ae-88e4-23758853e5c6">TCGA.COAD.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/4835f959-6ab4-4ee8-901f-c92aaad4592d">TCGA.DLBC.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/150c6a9f-71cd-4710-9617-cd150498202e">TCGA.ESCA.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/4b176a7b-a5c3-457e-af95-992018b6f3d7">TCGA.GBM.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/64683606-b957-4478-a7d5-673de68b0341">TCGA.HNSC.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/a8dc2dd2-74b3-4035-9551-c0ae3f76293e">TCGA.KICH.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/0cf0f121-4f10-436e-bfc3-8fcfd5f78d0d">TCGA.KIRC.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/b1c13b93-7ad6-4d09-b613-84a7c55e61d9">TCGA.KIRP.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/bac5a617-5b1d-4c33-b9ac-b48bd7e4947a">TCGA.LAML.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/42ff7a98-5a9a-48ad-ad9d-d3a23c245296">TCGA.LGG.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/dc3f239a-ffc6-4e60-b5f5-9f365daaf60a">TCGA.LIHC.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/76b0eec4-bbb6-4340-972d-05a5aace63a4">TCGA.LUAD.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/846c4788-cda0-4f11-b240-a7ad977e032f">TCGA.LUSC.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/34a32349-bf87-4e96-86a5-ca23f0db475e">TCGA.MESO.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/bc5ee1aa-969d-472d-920b-0e654cc585fa">TCGA.OV.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/6f72cd49-6c0e-4409-8e94-26ea5d421bc8">TCGA.PAAD.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/e087cf64-b514-4e92-af9d-2b18341098d5">TCGA.PCPG.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/0d708001-437e-4b78-9ffa-bfafdfc10a28">TCGA.PRAD.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/221f1dec-b539-4345-b687-435659fc21af">TCGA.READ.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/84fad8c0-ac06-4181-92d9-0562392325ba">TCGA.SARC.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/4aee3e32-2802-4e1e-8577-d74b414f30f7">TCGA.SKCM.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/98d14107-5fb5-49bd-ac38-a52178838d6c">TCGA.STAD.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/c3155e81-df07-49c6-8502-ef6ebc60812e">TCGA.TGCT.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/3207f158-6f79-4636-81a1-ce1b30157925">TCGA.THCA.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/ea93d4df-5e76-484a-b7b8-93900ea1d61c">TCGA.THYM.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/a142d8b8-7741-4869-9ca4-0025890eee18">TCGA.UCEC.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/cfc3f9e4-ac0d-4133-b616-a7d2caf28e54">TCGA.UCS.mutect.somatic.maf.gz</a><br/>
<a href="https://api.gdc.cancer.gov/data/a96185ef-5b9b-4f0e-b437-f6a0f4f0892b">TCGA.UVM.mutect.somatic.maf.gz</a><br/></p></div>
      </div>
    </div>

    <footer id="docs-footer">
      <div class="container-fluid footer-container">
        <div>
          <a href="https://portal.gdc.cancer.gov" target="_blank">Site Home</a>
          |
          <a href="http://www.cancer.gov/global/web/policies" target="_blank"
            >Policies</a
          >
          |
          <a
            href="http://www.cancer.gov/global/web/policies/accessibility"
            target="_blank"
            >Accessibility</a
          >
          |
          <a
            href="http://www.cancer.gov/global/web/policies/foia"
            target="_blank"
            >FOIA</a
          >
        </div>
        <div>
          <a href="http://www.hhs.gov" target="_blank"
            >U.S. Department of Health and Human Services</a
          >
          |
          <a href="http://www.nih.gov" target="_blank"
            >National Institutes of Health</a
          >
          |
          <a href="http://www.cancer.gov" target="_blank"
            >National Cancer Institute</a
          >
          | <a href="http://www.usa.gov" target="_blank">USA.gov</a>
        </div>
        <div>NIH... Turning Discovery Into Health </div>

        <div class="version-caption">GDC Docs Version 2.0</div>
      </div>
    </footer>

    <script>
      /* When the user clicks on the button, 
toggle between hiding and showing the dropdown content */

      // Close the dropdown if the user clicks outside of it

      var toggler = document.getElementById("myToggle");
      toggler.onclick = function () {
        var myDropdown = document.getElementById("myDropdown");
        myDropdown.classList.toggle("show");
      };
      window.onclick = function (e) {
        if (!e.target.matches(".dropbtn")) {
          var myDropdown = document.getElementById("myDropdown");
          if (myDropdown.classList.contains("show")) {
            myDropdown.classList.remove("show");
          }
        }
      };
    </script>

    <!-- JavaScript Bundle with Popper -->
    <script
      src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
      integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
      crossorigin="anonymous"
    ></script>
  </body>
</html>